<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="A.DeMartin" />

<meta name="date" content="2024-02-21" />

<title>BMP Tcell</title>

<script src="site_libs/header-attrs-2.27/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">HumanHeartCarTrans</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">BMP Tcell</h1>
<h4 class="author">A.DeMartin</h4>
<h4 class="date">2024-02-21</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-10-03
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>HumanHeartCarTrans/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20240221code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20240221)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20240221code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20240221)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it
difficult for you and others to run your code on a different machine.
Change the absolute path(s) below to the suggested relative path(s) to
make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/Users/immbio/Desktop/Project/Angelina/HumanHeartCarTrans/data/Human_heart_allmerged_seurat.rds
</td>
<td style="text-align:left;">
data/Human_heart_allmerged_seurat.rds
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomangeldemartinHumanHeartCarTranstree2254ff080634e47be879f54e479ff4ecafb376detargetblank2254ff0a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/tree/2254ff080634e47be879f54e479ff4ecafb376de" target="_blank">2254ff0</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomangeldemartinHumanHeartCarTranstree2254ff080634e47be879f54e479ff4ecafb376detargetblank2254ff0a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/tree/2254ff080634e47be879f54e479ff4ecafb376de" target="_blank">2254ff0</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory
    Ignored:    data/seurat files/

Unstaged changes:
    Modified:   analysis/BMPTcell.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/BMPTcell.Rmd</code>) and HTML
(<code>docs/BMPTcell.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/analysis/BMPTcell.Rmd" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
<td>
Oct24
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/angeldemartin/HumanHeartCarTrans/5bf6503e89f0251b3eaf131c688f07153582623f/docs/BMPTcell.html" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
<td>
Oct24
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="load-packages" class="section level2">
<h2>load packages</h2>
</div>
<div id="load-file" class="section level2">
<h2>load file</h2>
<pre class="r"><code>##load merged file 
fileNam &lt;- &quot;/Users/immbio/Desktop/Project/Angelina/HumanHeartCarTrans/data/Human_heart_allmerged_seurat.rds&quot;
seuratM &lt;- readRDS(fileNam)
table(seuratM$dataset)</code></pre>
<pre><code>
        331571_3-5_20231012_Hu_nucseq_USZ_HTx001         331571_4-6_20231012_Hu_nucseq_USZ_HTx002 
                                            2934                                             2059 
    334131_01-1_20231103_Hu_nucseq_USZ_HTx002_RV     334131_02-2_20231103_Hu_nucseq_USZ_HTx003_LV 
                                            3718                                             3905 
    334131_03-3_20231103_Hu_nucseq_USZ_HTx004_RV     334131_04-4_20231106_Hu_nucseq_USZ_HTx003_RV 
                                            3377                                             3377 
    334131_05-5_20231106_Hu_nucseq_USZ_HTx004_LV     334131_06-6_20231106_Hu_nucseq_USZ_HTx005_RV 
                                            4861                                             2694 
    334131_07-7_20231107_Hu_nucseq_USZ_HTx005_LV     334131_08-8_20231107_Hu_nucseq_USZ_HTx006_RV 
                                            1739                                             1672 
    334131_09-9_20231108_Hu_nucseq_USZ_HTx006_LV    334131_10-10_20231108_Hu_nucseq_USZ_HTx001_RV 
                                            3101                                             1901 
    336431_07-7_20231129_Hu_nucseq_USZ_HTx008_RV     336431_08-8_20231129_Hu_nucseq_USZ_EMB001_V1 
                                            3679                                              483 
    336431_09-9_20231129_Hu_nucseq_USZ_HTx008_LV    336431_10-10_20231129_Hu_nucseq_USZ_EMB005_V1 
                                            3082                                              606 
   336431_11-11_20231129_Hu_nucseq_USZ_EMB006_V1    336431_12-12_20231129_Hu_nucseq_USZ_EMB007_V1 
                                             420                                             1326 
   336431_13-13_20231129_Hu_nucseq_USZ_EMB002_V1    336431_14-14_20231129_Hu_nucseq_USZ_EMB003_V1 
                                            1338                                             1362 
   336431_15-15_20231129_Hu_nucseq_USZ_EMB004_V1     340821_01-7_20240118_Hu_nucseq_USZ_EMB007_V2 
                                             986                                             1600 
    340821_02-8_20240118_Hu_nucseq_USZ_EMB008_V2     340821_03-9_20240123_Hu_nucseq_USZ_EMB001_V3 
                                             567                                               68 
   340821_04-10_20240123_Hu_nucseq_USZ_EMB002_V3    340821_05-11_20240123_Hu_nucseq_USZ_EMB003_V3 
                                             349                                              196 
   340821_06-12_20240123_Hu_nucseq_USZ_EMB004_V3    340821_07-13_20240123_Hu_nucseq_USZ_EMB005_V3 
                                             745                                             1896 
   340821_08-14_20240123_Hu_nucseq_USZ_EMB006_V3    340821_09-15_20240123_Hu_nucseq_USZ_EMB007_V3 
                                             230                                              361 
   340821_10-16_20240123_Hu_nucseq_USZ_EMB008_V3    340821_11-17_20240123_Hu_nucseq_USZ_EMB008_V1 
                                             194                                               56 
   340821_12-18_20240123_Hu_nucseq_USZ_HTx007_RV    340821_13-19_20240123_Hu_nucseq_USZ_HTx007_LV 
                                            2352                                             2148 
     340831_1-1_20240118_Hu_nucseq_USZ_EMB001_V2      340831_2-2_20240118_Hu_nucseq_USZ_EMB002_V2 
                                             804                                             2575 
     340831_3-3_20240118_Hu_nucseq_USZ_EMB003_V2      340831_4-4_20240118_Hu_nucseq_USZ_EMB004_V2 
                                             323                                              521 
     340831_5-5_20240118_Hu_nucseq_USZ_EMB005_V2      340831_6-6_20240118_Hu_nucseq_USZ_EMB006_V2 
                                             654                                              770 
     347741_4-4_20240326_Hu_nucseq_USZ_HTx010_RV      347741_5-5_20240326_Hu_nucseq_USZ_HTx010_LV 
                                            2034                                             3004 
    353921_01-1_20240515_Hu_nucseq_USZ_HTx011_RV     353921_02-2_20240515_Hu_nucseq_USZ_HTx011_LV 
                                            1846                                             1351 
    353921_03-3_20240515_Hu_nucseq_USZ_EMB011_V1     353921_04-4_20240515_Hu_nucseq_USZ_EMB011_V2 
                                             738                                              193 
    353921_05-5_20240515_Hu_nucseq_USZ_HTx012_RV     353921_06-6_20240515_Hu_nucseq_USZ_HTx012_LV 
                                            1101                                             2666 
    353921_07-7_20240515_Hu_nucseq_USZ_EMB012_V1     353921_08-8_20240515_Hu_nucseq_USZ_EMB010_V3 
                                             715                                              677 
    353921_09-9_20240515_Hu_nucseq_USZ_HTx013_RV    353921_10-10_20240515_Hu_nucseq_USZ_HTx013_LV 
                                            1715                                             1965 
 353921_11-11_20240515_Hu_nucseq_USZ_EMB010_V1_1  353921_12-12_20240515_Hu_nucseq_USZ_EMB010_V1_2 
                                             996                                              265 
   353921_13-13_20240524_Hu_nucseq_USZ_HTx015_RV    353921_14-14_20240524_Hu_nucseq_USZ_EMB011_V3 
                                            1745                                              623 
   353921_15-15_20240524_Hu_nucseq_USZ_EMB012_V2    353921_16-16_20240524_Hu_nucseq_USZ_EMB012_V3 
                                            1547                                             1846 
   353921_17-17_20240524_Hu_nucseq_USZ_HTx015_LV    353921_18-18_20240524_Hu_nucseq_USZ_EMB013_V1 
                                            1448                                              379 
   353921_19-19_20240524_Hu_nucseq_USZ_EMB013_V2    353921_20-20_20240524_Hu_nucseq_USZ_EMB013_V3 
                                             138                                              327 
   353921_21-21_20240524_Hu_nucseq_USZ_HTx014_RV    353921_22-22_20240524_Hu_nucseq_USZ_HTx014_LV 
                                            2348                                             2337 
   353921_23-23_20240524_Hu_nucseq_USZ_EMB014_V1    353921_24-24_20240524_Hu_nucseq_USZ_EMB014_V2 
                                             866                                              311 
    359861_01-1_20240715_Hu_nucseq_USZ_HTx016_LV     359861_02-2_20240715_Hu_nucseq_USZ_HTx016_RV 
                                            5034                                             7278 
    359861_03-3_20240715_Hu_nucseq_USZ_EMB014_V3     359861_04-4_20240715_Hu_nucseq_USZ_EMB015_V1 
                                             395                                              134 
    359861_05-5_20240715_Hu_nucseq_USZ_EMB015_V2     359861_06-6_20240715_Hu_nucseq_USZ_EMB015_V3 
                                             207                                              449 
    359861_07-7_20240715_Hu_nucseq_USZ_EMB016_V1     359861_08-8_20240715_Hu_nucseq_USZ_EMB016_V2 
                                             547                                               94 
   359861_18-10_20240730_Hu_nucseq_USZ_EMB016_V3   o28576_1_08-8_20220525_Hu_nucseq_Graz_8_HH_GEM 
                                             287                                             3921 
o28576_1_10-10_20220525_Hu_nucseq_Graz_10_HH_GEM o28576_1_11-11_20220525_Hu_nucseq_Graz_11_HH_GEM 
                                            3731                                             3991 
o28576_1_12-12_20220525_Hu_nucseq_Graz_12_HH_GEM     o292731_1-1_20220818_Hu_nucseq_Graz_9_HH_GEM 
                                            3818                                             4908 
   o292731_2-2_20220818_Hu_nucseq_Graz_13_HH_GEM   o294781_01-1_20220912_Hu_nucseq_Graz_21_HH_GEM 
                                            9882                                             1442 
  o294781_02-2_20220912_Hu_nucseq_Graz_22_HH_GEM   o294781_03-3_20220912_Hu_nucseq_Graz_23_HH_GEM 
                                            1998                                              841 
  o294781_04-4_20220912_Hu_nucseq_Graz_24_HH_GEM 
                                            1480 </code></pre>
<pre class="r"><code>table(seuratM$RNA_snn_res.0.25)</code></pre>
<pre><code>
    0     1     2     3     4     5     6     7     8     9    10    11    12    13 
50676 24558 22350 16898 12792  6349  3418  2478  2458  2167  1844  1087   884   688 </code></pre>
<pre class="r"><code>table(seuratM$orig.ident)</code></pre>
<pre><code>
       
148647 </code></pre>
<pre class="r"><code>table(seuratM$clusterName)</code></pre>
<pre><code>
     CM      Fb PerivFb    VSMC     Int    BEC1    BEC2     LEC      NC  AdipoC     Mph    Mph2 
  12792   50676   22350    3418    2167   24558    2458     884    1844     688   16898    2478 
  Tcell  Tcell2 
   6349    1087 </code></pre>
<p>##set color vectors</p>
<pre class="r"><code>colclusterName &lt;- c(&quot;#67001f&quot;, &quot;#D53E4F&quot;, &quot;#f4a582&quot;, &quot;#FEE08B&quot;, &quot;#003c30&quot;,&quot;#01665e&quot;,&quot;#66C2A5&quot;, &quot;#3288BD&quot;,&quot;#BEAED4&quot;, &quot;#c7eae5&quot;,&quot;#355C7D&quot;,&quot;#202547&quot;,&quot;#B45B5C&quot;,&quot;#8c510a&quot;)
names(colclusterName) &lt;- c(&quot;CM&quot;,&quot;Fb&quot;,&quot;PerivFb&quot;,&quot;VSMC&quot;,&quot;Int&quot;,&quot;BEC1&quot;, &quot;BEC2&quot;,&quot;LEC&quot;,&quot;NC&quot;,&quot;AdipoC&quot;,&quot;Mph&quot;,&quot;Mph2&quot;,&quot;Tcell&quot;,&quot;Tcell2&quot;)

coldiseaseCond &lt;- c(&quot;#dfc27d&quot;,&quot;#BE3144&quot;,&quot;#202547&quot;,&quot;#355C7D&quot;,&quot;#779d8d&quot;)
names(coldiseaseCond) &lt;- c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;)</code></pre>
</div>
<div id="bmp-features" class="section level2">
<h2>BMP features</h2>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000125378.BMP4&quot;, pt.size = 0.1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/BMPTcell.Rmd/BMP%20features-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-BMP-features-1">
Past versions of "BMP features-1.png"
</button>
</p>
<div id="fig-no-spaces-BMP-features-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/BMP features-1.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000112175.BMP5&quot;, pt.size = 0.1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/BMPTcell.Rmd/BMP%20features-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-BMP-features-2">
Past versions of "BMP features-2.png"
</button>
</p>
<div id="fig-no-spaces-BMP-features-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/BMP features-2.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000125845.BMP2&quot;, pt.size = 0.1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/BMPTcell.Rmd/BMP%20features-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-BMP-features-3">
Past versions of "BMP features-3.png"
</button>
</p>
<div id="fig-no-spaces-BMP-features-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/BMP features-3.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000153162.BMP6&quot;, pt.size = 0.1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/BMPTcell.Rmd/BMP%20features-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-BMP-features-4">
Past versions of "BMP features-4.png"
</button>
</p>
<div id="fig-no-spaces-BMP-features-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/BMP features-4.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000168487.BMP1&quot;, pt.size = 0.1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/BMPTcell.Rmd/BMP%20features-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-BMP-features-5">
Past versions of "BMP features-5.png"
</button>
</p>
<div id="fig-no-spaces-BMP-features-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/BMP features-5.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000183682.BMP8A&quot;, pt.size = 0.1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/BMPTcell.Rmd/BMP%20features-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-BMP-features-6">
Past versions of "BMP features-6.png"
</button>
</p>
<div id="fig-no-spaces-BMP-features-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/BMP features-6.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000138756.BMP2K&quot;, pt.size = 0.1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/BMPTcell.Rmd/BMP%20features-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-BMP-features-7">
Past versions of "BMP features-7.png"
</button>
</p>
<div id="fig-no-spaces-BMP-features-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/BMP features-7.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000107779.BMPR1A&quot;, pt.size = 0.1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/BMPTcell.Rmd/BMP%20features-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-BMP-features-8">
Past versions of "BMP features-8.png"
</button>
</p>
<div id="fig-no-spaces-BMP-features-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/BMP features-8.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000204217.BMPR2&quot;, pt.size = 0.1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/BMPTcell.Rmd/BMP%20features-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-BMP-features-9">
Past versions of "BMP features-9.png"
</button>
</p>
<div id="fig-no-spaces-BMP-features-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/BMP features-9.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="vlnplot-bmp4-in-fb" class="section level2">
<h2>VlnPlot BMP4 in Fb</h2>
<pre class="r"><code>###violin plot BMP4 on Fb
seuratFb &lt;- subset(seuratM, clusterName %in% c(&quot;Fb&quot;))
table(seuratFb$clusterName)</code></pre>
<pre><code>
     CM      Fb PerivFb    VSMC     Int    BEC1    BEC2     LEC      NC  AdipoC     Mph    Mph2 
      0   50676       0       0       0       0       0       0       0       0       0       0 
  Tcell  Tcell2 
      0       0 </code></pre>
<pre class="r"><code>seuratFb$diseaseCond &lt;- factor(seuratFb$diseaseCond, levels=c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;))
Idents(seuratFb) &lt;- seuratFb$diseaseCond
levels(seuratFb)</code></pre>
<pre><code>[1] &quot;healthy&quot; &quot;explant&quot; &quot;visit1&quot;  &quot;visit2&quot;  &quot;visit3&quot; </code></pre>
<pre class="r"><code>VlnPlot(object=seuratFb, features = &quot;ENSG00000125378.BMP4&quot;, cols = coldiseaseCond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/BMPTcell.Rmd/violin%20BMP4%20Fb-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="vlnplot-bmp5-in-perivfb" class="section level2">
<h2>VlnPlot BMP5 in PerivFb</h2>
<pre class="r"><code>###violin plot BMP5 on PerivFb
seuratPerivFb &lt;- subset(seuratM, clusterName %in% c(&quot;PerivFb&quot;))
table(seuratPerivFb$clusterName)</code></pre>
<pre><code>
     CM      Fb PerivFb    VSMC     Int    BEC1    BEC2     LEC      NC  AdipoC     Mph    Mph2 
      0       0   22350       0       0       0       0       0       0       0       0       0 
  Tcell  Tcell2 
      0       0 </code></pre>
<pre class="r"><code>seuratPerivFb$diseaseCond &lt;- factor(seuratPerivFb$diseaseCond, levels=c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;))
Idents(seuratPerivFb) &lt;- seuratPerivFb$diseaseCond
levels(seuratPerivFb)</code></pre>
<pre><code>[1] &quot;healthy&quot; &quot;explant&quot; &quot;visit1&quot;  &quot;visit2&quot;  &quot;visit3&quot; </code></pre>
<pre class="r"><code>VlnPlot(object=seuratPerivFb, features = &quot;ENSG00000112175.BMP5&quot;, cols = coldiseaseCond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/BMPTcell.Rmd/violin%20BMP5%20PerivFb-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="scatter-plot-bmp4-in-fb-vs-tcells" class="section level2">
<h2>Scatter plot BMP4 in Fb vs Tcells</h2>
<pre class="r"><code>###rel abundance CD8 Tcells
datList &lt;- NULL
for(con in unique(seuratM$patient_diseaseCond)){
  seuratSub &lt;- subset(seuratM, patient_diseaseCond==con)
  print(dim(seuratSub))
  dat_con &lt;- as.data.frame(table(seuratSub$clusterName)) %&gt;%
  mutate(percent=Freq/ncol(seuratSub)) %&gt;% mutate(patient_diseaseCond=con)
  datList[[con]] &lt;- dat_con
}</code></pre>
<pre><code>[1] 39836  4835
[1] 39836  5777
[1] 39836  7282
[1] 39836  8238
[1] 39836  4433
[1] 39836  4773
[1] 39836  6761
[1] 39836   483
[1] 39836   606
[1] 39836   420
[1] 39836  1326
[1] 39836  1338
[1] 39836  1362
[1] 39836   986
[1] 39836  1600
[1] 39836   567
[1] 39836    68
[1] 39836   349
[1] 39836   196
[1] 39836   745
[1] 39836  1896
[1] 39836   230
[1] 39836   361
[1] 39836   194
[1] 39836    56
[1] 39836  4500
[1] 39836   804
[1] 39836  2575
[1] 39836   323
[1] 39836   521
[1] 39836   654
[1] 39836   770
[1] 39836  5038
[1] 39836  3197
[1] 39836   738
[1] 39836   193
[1] 39836  3767
[1] 39836   715
[1] 39836   677
[1] 39836  3680
[1] 39836   996
[1] 39836   265
[1] 39836  3193
[1] 39836   623
[1] 39836  1547
[1] 39836  1846
[1] 39836   379
[1] 39836   138
[1] 39836   327
[1] 39836  4685
[1] 39836   866
[1] 39836   311
[1] 39836 12312
[1] 39836   395
[1] 39836   134
[1] 39836   207
[1] 39836   449
[1] 39836   547
[1] 39836    94
[1] 39836   287
[1] 39836  3921
[1] 39836  3731
[1] 39836  3991
[1] 39836  3818
[1] 39836  4908
[1] 39836  9882
[1] 39836  1442
[1] 39836  1998
[1] 39836   841
[1] 39836  1480</code></pre>
<pre class="r"><code>dat_allclust &lt;- do.call(&quot;rbind&quot;, datList)
datTcell &lt;- dat_allclust %&gt;% filter(Var1 == &quot;Tcell&quot;) 
rownames(datTcell) &lt;- datTcell$patient_diseaseCond

###BMP4 expr fb
Idents(seuratFb) &lt;- seuratFb$patient_diseaseCond
levels(seuratFb)</code></pre>
<pre><code> [1] &quot;CarTransPat1_explant&quot;  &quot;CarTransPat2_explant&quot;  &quot;CarTransPat3_explant&quot;  &quot;CarTransPat4_explant&quot; 
 [5] &quot;CarTransPat5_explant&quot;  &quot;CarTransPat6_explant&quot;  &quot;CarTransPat8_explant&quot;  &quot;CarTransPat1_visit1&quot;  
 [9] &quot;CarTransPat5_visit1&quot;   &quot;CarTransPat6_visit1&quot;   &quot;CarTransPat7_visit1&quot;   &quot;CarTransPat2_visit1&quot;  
[13] &quot;CarTransPat3_visit1&quot;   &quot;CarTransPat4_visit1&quot;   &quot;CarTransPat7_visit2&quot;   &quot;CarTransPat8_visit2&quot;  
[17] &quot;CarTransPat1_visit3&quot;   &quot;CarTransPat2_visit3&quot;   &quot;CarTransPat3_visit3&quot;   &quot;CarTransPat4_visit3&quot;  
[21] &quot;CarTransPat5_visit3&quot;   &quot;CarTransPat6_visit3&quot;   &quot;CarTransPat7_visit3&quot;   &quot;CarTransPat8_visit3&quot;  
[25] &quot;CarTransPat8_visit1&quot;   &quot;CarTransPat7_explant&quot;  &quot;CarTransPat1_visit2&quot;   &quot;CarTransPat2_visit2&quot;  
[29] &quot;CarTransPat3_visit2&quot;   &quot;CarTransPat4_visit2&quot;   &quot;CarTransPat5_visit2&quot;   &quot;CarTransPat6_visit2&quot;  
[33] &quot;CarTransPat10_explant&quot; &quot;CarTransPat11_explant&quot; &quot;CarTransPat11_visit1&quot;  &quot;CarTransPat11_visit2&quot; 
[37] &quot;CarTransPat12_explant&quot; &quot;CarTransPat12_visit1&quot;  &quot;CarTransPat10_visit3&quot;  &quot;CarTransPat13_explant&quot;
[41] &quot;CarTransPat10_visit1&quot;  &quot;CarTransPat10_visit2&quot;  &quot;CarTransPat15_explant&quot; &quot;CarTransPat11_visit3&quot; 
[45] &quot;CarTransPat12_visit2&quot;  &quot;CarTransPat12_visit3&quot;  &quot;CarTransPat13_visit1&quot;  &quot;CarTransPat13_visit2&quot; 
[49] &quot;CarTransPat13_visit3&quot;  &quot;CarTransPat14_explant&quot; &quot;CarTransPat14_visit1&quot;  &quot;CarTransPat14_visit2&quot; 
[53] &quot;CarTransPat16_explant&quot; &quot;CarTransPat14_visit3&quot;  &quot;CarTransPat15_visit1&quot;  &quot;CarTransPat15_visit2&quot; 
[57] &quot;CarTransPat15_visit3&quot;  &quot;CarTransPat16_visit1&quot;  &quot;CarTransPat16_visit2&quot;  &quot;CarTransPat16_visit3&quot; 
[61] &quot;HH1_healthy&quot;           &quot;HH3_healthy&quot;           &quot;HH4_healthy&quot;           &quot;HH5_healthy&quot;          
[65] &quot;HH6_healthy&quot;           &quot;HH7_healthy&quot;           &quot;HH8_healthy&quot;           &quot;HH9_healthy&quot;          
[69] &quot;HH10_healthy&quot;          &quot;HH2_healthy&quot;          </code></pre>
<pre class="r"><code>BMP4 &lt;- &quot;ENSG00000125378.BMP4&quot;

avg.all &lt;- AverageExpression(seuratFb, group.by = &quot;patient_diseaseCond&quot;)
avg.all &lt;- data.frame(log1p(avg.all$RNA)) %&gt;% 
  rownames_to_column(var=&quot;gene&quot;)
avg.BMP4 &lt;- avg.all  %&gt;% filter(gene == BMP4)
avg.BMP4 &lt;- avg.BMP4 %&gt;% tibble::column_to_rownames(var=&quot;gene&quot;)
avg.BMP4t &lt;- data.frame(t(avg.BMP4)) 
avg.BMP4t$patient_diseaseCond &lt;- rownames(avg.BMP4t)
avg.BMP4t &lt;- avg.BMP4t %&gt;% mutate(patient_diseaseCond=gsub(&quot;\\.&quot;, &quot;_&quot;, patient_diseaseCond))

dat_all &lt;- merge(datTcell, avg.BMP4t, by = &quot;patient_diseaseCond&quot;)
dat_all &lt;- dat_all %&gt;% mutate(diseasecond=gsub(&quot;.*_&quot;,&quot;&quot;,patient_diseaseCond)) %&gt;% mutate(patient=gsub(&quot;_.*&quot;,&quot;&quot;,patient_diseaseCond))

# scatterplot
p &lt;- ggscatter(dat_all, x=&quot;percent&quot;, y=&quot;ENSG00000125378.BMP4&quot;,
                     label= &quot;patient_diseaseCond&quot;,
                     color = &quot;diseasecond&quot;,
                     palette = c(&quot;#dfc27d&quot;,&quot;#BE3144&quot;,&quot;#202547&quot;,&quot;#355C7D&quot;,&quot;#779d8d&quot;),
                     repel = T,
                     label.rectangle = F,
                     xlab = &quot;TcellFreq&quot;,
                     ylab = &quot;BMP4&quot;,
                     font.label = c(&quot;0&quot;, &quot;plain&quot;,&quot;black&quot;),
                     size=2,)
p</code></pre>
<p><img src="figure/BMPTcell.Rmd/scatter%20BMP4vsTcells-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-scatter-BMP4vsTcells-1">
Past versions of "scatter BMP4vsTcells-1.png"
</button>
</p>
<div id="fig-no-spaces-scatter-BMP4vsTcells-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/scatter BMP4vsTcells-1.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>## plot indiviual diseasecond
dat_HH &lt;- dat_all %&gt;% filter(diseasecond == &quot;healthy&quot;) 

p &lt;- ggscatter(dat_HH, x=&quot;percent&quot;, y=&quot;ENSG00000125378.BMP4&quot;,
                     label= &quot;patient&quot;,
                     color = &quot;diseasecond&quot;,
                     palette = c(&quot;#dfc27d&quot;),
                     repel = T,
                     label.rectangle = F,
                     xlab = &quot;TcellFreq&quot;,
                     ylab = &quot;BMP4&quot;,
                     font.label = c(&quot;10&quot;, &quot;plain&quot;,&quot;black&quot;),
                     size=2,)
p</code></pre>
<p><img src="figure/BMPTcell.Rmd/scatter%20BMP4vsTcells-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-scatter-BMP4vsTcells-2">
Past versions of "scatter BMP4vsTcells-2.png"
</button>
</p>
<div id="fig-no-spaces-scatter-BMP4vsTcells-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/scatter BMP4vsTcells-2.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>dat_expl &lt;- dat_all %&gt;% filter(diseasecond == &quot;explant&quot;) 

p &lt;- ggscatter(dat_expl, x=&quot;percent&quot;, y=&quot;ENSG00000125378.BMP4&quot;,
                     label= &quot;patient&quot;,
                     color = &quot;diseasecond&quot;,
                     palette = c(&quot;#BE3144&quot;),
                     repel = T,
                     label.rectangle = F,
                     xlab = &quot;TcellFreq&quot;,
                     ylab = &quot;BMP4&quot;,
                     font.label = c(&quot;10&quot;, &quot;plain&quot;,&quot;black&quot;),
                     size=2,)
p</code></pre>
<p><img src="figure/BMPTcell.Rmd/scatter%20BMP4vsTcells-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-scatter-BMP4vsTcells-3">
Past versions of "scatter BMP4vsTcells-3.png"
</button>
</p>
<div id="fig-no-spaces-scatter-BMP4vsTcells-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/scatter BMP4vsTcells-3.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>dat_v1to3 &lt;- dat_all %&gt;% filter(diseasecond %in% c(&quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;))
p &lt;- ggscatter(dat_v1to3, x=&quot;percent&quot;, y=&quot;ENSG00000125378.BMP4&quot;,
                     label= &quot;patient&quot;,
                     color = &quot;diseasecond&quot;,
                     palette = c(&quot;#202547&quot;,&quot;#355C7D&quot;,&quot;#779d8d&quot;),
                     repel = T,
                     label.rectangle = F,
                     xlab = &quot;TcellFreq&quot;,
                     ylab = &quot;BMP4&quot;,
                     font.label = c(&quot;0&quot;, &quot;plain&quot;,&quot;black&quot;),
                     size=2,)
p</code></pre>
<p><img src="figure/BMPTcell.Rmd/scatter%20BMP4vsTcells-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-scatter-BMP4vsTcells-4">
Past versions of "scatter BMP4vsTcells-4.png"
</button>
</p>
<div id="fig-no-spaces-scatter-BMP4vsTcells-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/scatter BMP4vsTcells-4.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>dat_v1 &lt;- dat_all %&gt;% filter(diseasecond %in% c(&quot;visit1&quot;))
p &lt;- ggscatter(dat_v1, x=&quot;percent&quot;, y=&quot;ENSG00000125378.BMP4&quot;,
                     label= &quot;patient&quot;,
                     color = &quot;diseasecond&quot;,
                     palette = c(&quot;#202547&quot;),
                     repel = T,
                     label.rectangle = F,
                     xlab = &quot;TcellFreq&quot;,
                     ylab = &quot;BMP4&quot;,
                     font.label = c(&quot;10&quot;, &quot;plain&quot;,&quot;black&quot;),
                     size=2,)
p</code></pre>
<p><img src="figure/BMPTcell.Rmd/scatter%20BMP4vsTcells-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-scatter-BMP4vsTcells-5">
Past versions of "scatter BMP4vsTcells-5.png"
</button>
</p>
<div id="fig-no-spaces-scatter-BMP4vsTcells-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/scatter BMP4vsTcells-5.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>dat_v2 &lt;- dat_all %&gt;% filter(diseasecond %in% c(&quot;visit2&quot;))
p &lt;- ggscatter(dat_v2, x=&quot;percent&quot;, y=&quot;ENSG00000125378.BMP4&quot;,
                     label= &quot;patient&quot;,
                     color = &quot;diseasecond&quot;,
                     palette = c(&quot;#355C7D&quot;),
                     repel = T,
                     label.rectangle = F,
                     xlab = &quot;TcellFreq&quot;,
                     ylab = &quot;BMP4&quot;,
                     font.label = c(&quot;10&quot;, &quot;plain&quot;,&quot;black&quot;),
                     size=2,)
p</code></pre>
<p><img src="figure/BMPTcell.Rmd/scatter%20BMP4vsTcells-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-scatter-BMP4vsTcells-6">
Past versions of "scatter BMP4vsTcells-6.png"
</button>
</p>
<div id="fig-no-spaces-scatter-BMP4vsTcells-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/scatter BMP4vsTcells-6.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>dat_v3 &lt;- dat_all %&gt;% filter(diseasecond %in% c(&quot;visit3&quot;))
p &lt;- ggscatter(dat_v3, x=&quot;percent&quot;, y=&quot;ENSG00000125378.BMP4&quot;,
                     label= &quot;patient&quot;,
                     color = &quot;diseasecond&quot;,
                     palette = c(&quot;#779d8d&quot;),
                     repel = T,
                     label.rectangle = F,
                     xlab = &quot;TcellFreq&quot;,
                     ylab = &quot;BMP4&quot;,
                     font.label = c(&quot;10&quot;, &quot;plain&quot;,&quot;black&quot;),
                     size=2,)
p</code></pre>
<p><img src="figure/BMPTcell.Rmd/scatter%20BMP4vsTcells-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-scatter-BMP4vsTcells-7">
Past versions of "scatter BMP4vsTcells-7.png"
</button>
</p>
<div id="fig-no-spaces-scatter-BMP4vsTcells-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/BMPTcell.Rmd/scatter BMP4vsTcells-7.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="session-info" class="section level2">
<h2>session info</h2>
<pre class="r"><code>date()</code></pre>
<pre><code>[1] &quot;Thu Oct  3 12:28:50 2024&quot;</code></pre>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.0 (2024-04-24)
Platform: x86_64-apple-darwin20
Running under: macOS Ventura 13.6.9

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: Europe/Zurich
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] NCmisc_1.2.0                VennDiagram_1.7.3           futile.logger_1.4.3        
 [4] ggupset_0.4.0               gridExtra_2.3               DOSE_3.30.1                
 [7] enrichplot_1.24.0           msigdbr_7.5.1               org.Hs.eg.db_3.19.1        
[10] AnnotationDbi_1.66.0        clusterProfiler_4.12.0      multtest_2.60.0            
[13] metap_1.11                  scater_1.32.0               scuttle_1.14.0             
[16] destiny_3.18.0              circlize_0.4.16             muscat_1.18.0              
[19] viridis_0.6.5               viridisLite_0.4.2           lubridate_1.9.3            
[22] forcats_1.0.0               stringr_1.5.1               purrr_1.0.2                
[25] readr_2.1.5                 tidyr_1.3.1                 tibble_3.2.1               
[28] tidyverse_2.0.0             dplyr_1.1.4                 SingleCellExperiment_1.26.0
[31] SummarizedExperiment_1.34.0 Biobase_2.64.0              GenomicRanges_1.56.1       
[34] GenomeInfoDb_1.40.1         IRanges_2.38.1              S4Vectors_0.42.1           
[37] BiocGenerics_0.50.0         MatrixGenerics_1.16.0       matrixStats_1.3.0          
[40] pheatmap_1.0.12             ggpubr_0.6.0                ggplot2_3.5.1              
[43] Seurat_5.1.0                SeuratObject_5.0.2          sp_2.1-4                   
[46] runSeurat3_0.1.0            ExploreSCdataSeurat3_0.1.0 

loaded via a namespace (and not attached):
  [1] igraph_2.0.3              ica_1.0-3                 plotly_4.10.4            
  [4] zlibbioc_1.50.0           tidyselect_1.2.1          bit_4.0.5                
  [7] doParallel_1.0.17         clue_0.3-65               lattice_0.22-6           
 [10] rjson_0.2.21              blob_1.2.4                S4Arrays_1.4.1           
 [13] pbkrtest_0.5.3            parallel_4.4.0            png_0.1-8                
 [16] plotrix_3.8-4             cli_3.6.3                 ggplotify_0.1.2          
 [19] goftest_1.2-3             VIM_6.2.2                 variancePartition_1.34.0 
 [22] BiocNeighbors_1.22.0      uwot_0.2.2                shadowtext_0.1.3         
 [25] curl_5.2.1                mime_0.12                 evaluate_0.24.0          
 [28] tidytree_0.4.6            leiden_0.4.3.1            ComplexHeatmap_2.20.0    
 [31] stringi_1.8.4             backports_1.5.0           lmerTest_3.1-3           
 [34] qqconf_1.3.2              httpuv_1.6.15             magrittr_2.0.3           
 [37] splines_4.4.0             ggraph_2.2.1              sctransform_0.4.1        
 [40] ggbeeswarm_0.7.2          DBI_1.2.3                 jquerylib_0.1.4          
 [43] smoother_1.3              withr_3.0.1               git2r_0.33.0             
 [46] corpcor_1.6.10            class_7.3-22              rprojroot_2.0.4          
 [49] lmtest_0.9-40             tidygraph_1.3.1           formatR_1.14             
 [52] colourpicker_1.3.0        htmlwidgets_1.6.4         fs_1.6.4                 
 [55] ggrepel_0.9.5             labeling_0.4.3            fANCOVA_0.6-1            
 [58] SparseArray_1.4.8         DESeq2_1.44.0             ranger_0.16.0            
 [61] DEoptimR_1.1-3            reticulate_1.38.0         hexbin_1.28.3            
 [64] zoo_1.8-12                XVector_0.44.0            knitr_1.48               
 [67] ggplot.multistats_1.0.0   UCSC.utils_1.0.0          RhpcBLASctl_0.23-42      
 [70] timechange_0.3.0          foreach_1.5.2             fansi_1.0.6              
 [73] patchwork_1.2.0           caTools_1.18.2            data.table_1.16.0        
 [76] ggtree_3.12.0             RSpectra_0.16-1           irlba_2.3.5.1            
 [79] ggrastr_1.0.2             fastDummies_1.7.3         gridGraphics_0.5-1       
 [82] lazyeval_0.2.2            yaml_2.3.9                survival_3.7-0           
 [85] scattermore_1.2           crayon_1.5.3              RcppAnnoy_0.0.22         
 [88] RColorBrewer_1.1-3        progressr_0.14.0          tweenr_2.0.3             
 [91] later_1.3.2               ggridges_0.5.6            codetools_0.2-20         
 [94] GlobalOptions_0.1.2       aod_1.3.3                 KEGGREST_1.44.1          
 [97] Rtsne_0.17                shape_1.4.6.1             limma_3.60.3             
[100] pkgconfig_2.0.3           TMB_1.9.14                mathjaxr_1.6-0           
[103] EnvStats_2.8.1            aplot_0.2.3               scatterplot3d_0.3-44     
[106] spatstat.sparse_3.1-0     ape_5.8                   xtable_1.8-4             
[109] car_3.1-2                 highr_0.11                plyr_1.8.9               
[112] httr_1.4.7                rbibutils_2.2.16          tools_4.4.0              
[115] globals_0.16.3            beeswarm_0.4.0            broom_1.0.6              
[118] nlme_3.1-165              lambda.r_1.2.4            HDO.db_0.99.1            
[121] lme4_1.1-35.5             digest_0.6.36             numDeriv_2016.8-1.1      
[124] Matrix_1.7-0              farver_2.1.2              tzdb_0.4.0               
[127] remaCor_0.0.18            reshape2_1.4.4            yulab.utils_0.1.4        
[130] glue_1.7.0                cachem_1.1.0              polyclip_1.10-6          
[133] generics_0.1.3            Biostrings_2.72.1         mvtnorm_1.2-5            
[136] parallelly_1.37.1         mnormt_2.1.1              statmod_1.5.0            
[139] RcppHNSW_0.6.0            ScaledMatrix_1.12.0       carData_3.0-5            
[142] minqa_1.2.7               pbapply_1.7-2             spam_2.10-0              
[145] gson_0.1.0                utf8_1.2.4                graphlayouts_1.1.1       
[148] gtools_3.9.5              ggsignif_0.6.4            RcppEigen_0.3.4.0.0      
[151] shiny_1.8.1.1             GenomeInfoDbData_1.2.12   glmmTMB_1.1.9            
[154] memoise_2.0.1             rmarkdown_2.27            scales_1.3.0             
[157] future_1.33.2             RANN_2.6.1                Cairo_1.6-2              
[160] spatstat.data_3.1-2       rstudioapi_0.16.0         cluster_2.1.6            
[163] whisker_0.4.1             mutoss_0.1-13             spatstat.utils_3.0-5     
[166] hms_1.1.3                 fitdistrplus_1.1-11       munsell_0.5.1            
[169] cowplot_1.1.3             colorspace_2.1-0          rlang_1.1.4              
[172] DelayedMatrixStats_1.26.0 sparseMatrixStats_1.16.0  xts_0.14.0               
[175] dotCall64_1.1-1           shinydashboard_0.7.2      ggforce_0.4.2            
[178] laeken_0.5.3              mgcv_1.9-1                xfun_0.45                
[181] e1071_1.7-14              TH.data_1.1-2             iterators_1.0.14         
[184] abind_1.4-5               GOSemSim_2.30.0           treeio_1.28.0            
[187] futile.options_1.0.1      bitops_1.0-7              Rdpack_2.6               
[190] promises_1.3.0            scatterpie_0.2.3          RSQLite_2.3.7            
[193] qvalue_2.36.0             sandwich_3.1-0            fgsea_1.30.0             
[196] DelayedArray_0.30.1       proxy_0.4-27              GO.db_3.19.1             
[199] compiler_4.4.0            prettyunits_1.2.0         boot_1.3-30              
[202] beachmat_2.20.0           listenv_0.9.1             Rcpp_1.0.13              
[205] edgeR_4.2.0               workflowr_1.7.1           BiocSingular_1.20.0      
[208] tensor_1.5                MASS_7.3-61               progress_1.2.3           
[211] BiocParallel_1.38.0       babelgene_22.9            spatstat.random_3.2-3    
[214] R6_2.5.1                  fastmap_1.2.0             multcomp_1.4-25          
[217] fastmatch_1.1-4           rstatix_0.7.2             vipor_0.4.7              
[220] TTR_0.24.4                ROCR_1.0-11               TFisher_0.2.0            
[223] rsvd_1.0.5                vcd_1.4-12                nnet_7.3-19              
[226] gtable_0.3.5              KernSmooth_2.23-24        miniUI_0.1.1.1           
[229] deldir_2.0-4              htmltools_0.5.8.1         ggthemes_5.1.0           
[232] bit64_4.0.5               spatstat.explore_3.2-7    lifecycle_1.0.4          
[235] blme_1.0-5                nloptr_2.1.1              sass_0.4.9               
[238] vctrs_0.6.5               robustbase_0.99-3         spatstat.geom_3.2-9      
[241] sn_2.1.1                  ggfun_0.1.5               future.apply_1.11.2      
[244] bslib_0.7.0               pillar_1.9.0              gplots_3.1.3.1           
[247] pcaMethods_1.96.0         locfit_1.5-9.10           jsonlite_1.8.8           
[250] GetoptLong_1.0.5         </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.0 (2024-04-24)
Platform: x86_64-apple-darwin20
Running under: macOS Ventura 13.6.9

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: Europe/Zurich
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] NCmisc_1.2.0                VennDiagram_1.7.3           futile.logger_1.4.3        
 [4] ggupset_0.4.0               gridExtra_2.3               DOSE_3.30.1                
 [7] enrichplot_1.24.0           msigdbr_7.5.1               org.Hs.eg.db_3.19.1        
[10] AnnotationDbi_1.66.0        clusterProfiler_4.12.0      multtest_2.60.0            
[13] metap_1.11                  scater_1.32.0               scuttle_1.14.0             
[16] destiny_3.18.0              circlize_0.4.16             muscat_1.18.0              
[19] viridis_0.6.5               viridisLite_0.4.2           lubridate_1.9.3            
[22] forcats_1.0.0               stringr_1.5.1               purrr_1.0.2                
[25] readr_2.1.5                 tidyr_1.3.1                 tibble_3.2.1               
[28] tidyverse_2.0.0             dplyr_1.1.4                 SingleCellExperiment_1.26.0
[31] SummarizedExperiment_1.34.0 Biobase_2.64.0              GenomicRanges_1.56.1       
[34] GenomeInfoDb_1.40.1         IRanges_2.38.1              S4Vectors_0.42.1           
[37] BiocGenerics_0.50.0         MatrixGenerics_1.16.0       matrixStats_1.3.0          
[40] pheatmap_1.0.12             ggpubr_0.6.0                ggplot2_3.5.1              
[43] Seurat_5.1.0                SeuratObject_5.0.2          sp_2.1-4                   
[46] runSeurat3_0.1.0            ExploreSCdataSeurat3_0.1.0 

loaded via a namespace (and not attached):
  [1] igraph_2.0.3              ica_1.0-3                 plotly_4.10.4            
  [4] zlibbioc_1.50.0           tidyselect_1.2.1          bit_4.0.5                
  [7] doParallel_1.0.17         clue_0.3-65               lattice_0.22-6           
 [10] rjson_0.2.21              blob_1.2.4                S4Arrays_1.4.1           
 [13] pbkrtest_0.5.3            parallel_4.4.0            png_0.1-8                
 [16] plotrix_3.8-4             cli_3.6.3                 ggplotify_0.1.2          
 [19] goftest_1.2-3             VIM_6.2.2                 variancePartition_1.34.0 
 [22] BiocNeighbors_1.22.0      uwot_0.2.2                shadowtext_0.1.3         
 [25] curl_5.2.1                mime_0.12                 evaluate_0.24.0          
 [28] tidytree_0.4.6            leiden_0.4.3.1            ComplexHeatmap_2.20.0    
 [31] stringi_1.8.4             backports_1.5.0           lmerTest_3.1-3           
 [34] qqconf_1.3.2              httpuv_1.6.15             magrittr_2.0.3           
 [37] splines_4.4.0             ggraph_2.2.1              sctransform_0.4.1        
 [40] ggbeeswarm_0.7.2          DBI_1.2.3                 jquerylib_0.1.4          
 [43] smoother_1.3              withr_3.0.1               git2r_0.33.0             
 [46] corpcor_1.6.10            class_7.3-22              rprojroot_2.0.4          
 [49] lmtest_0.9-40             tidygraph_1.3.1           formatR_1.14             
 [52] colourpicker_1.3.0        htmlwidgets_1.6.4         fs_1.6.4                 
 [55] ggrepel_0.9.5             labeling_0.4.3            fANCOVA_0.6-1            
 [58] SparseArray_1.4.8         DESeq2_1.44.0             ranger_0.16.0            
 [61] DEoptimR_1.1-3            reticulate_1.38.0         hexbin_1.28.3            
 [64] zoo_1.8-12                XVector_0.44.0            knitr_1.48               
 [67] ggplot.multistats_1.0.0   UCSC.utils_1.0.0          RhpcBLASctl_0.23-42      
 [70] timechange_0.3.0          foreach_1.5.2             fansi_1.0.6              
 [73] patchwork_1.2.0           caTools_1.18.2            data.table_1.16.0        
 [76] ggtree_3.12.0             RSpectra_0.16-1           irlba_2.3.5.1            
 [79] ggrastr_1.0.2             fastDummies_1.7.3         gridGraphics_0.5-1       
 [82] lazyeval_0.2.2            yaml_2.3.9                survival_3.7-0           
 [85] scattermore_1.2           crayon_1.5.3              RcppAnnoy_0.0.22         
 [88] RColorBrewer_1.1-3        progressr_0.14.0          tweenr_2.0.3             
 [91] later_1.3.2               ggridges_0.5.6            codetools_0.2-20         
 [94] GlobalOptions_0.1.2       aod_1.3.3                 KEGGREST_1.44.1          
 [97] Rtsne_0.17                shape_1.4.6.1             limma_3.60.3             
[100] pkgconfig_2.0.3           TMB_1.9.14                mathjaxr_1.6-0           
[103] EnvStats_2.8.1            aplot_0.2.3               scatterplot3d_0.3-44     
[106] spatstat.sparse_3.1-0     ape_5.8                   xtable_1.8-4             
[109] car_3.1-2                 highr_0.11                plyr_1.8.9               
[112] httr_1.4.7                rbibutils_2.2.16          tools_4.4.0              
[115] globals_0.16.3            beeswarm_0.4.0            broom_1.0.6              
[118] nlme_3.1-165              lambda.r_1.2.4            HDO.db_0.99.1            
[121] lme4_1.1-35.5             digest_0.6.36             numDeriv_2016.8-1.1      
[124] Matrix_1.7-0              farver_2.1.2              tzdb_0.4.0               
[127] remaCor_0.0.18            reshape2_1.4.4            yulab.utils_0.1.4        
[130] glue_1.7.0                cachem_1.1.0              polyclip_1.10-6          
[133] generics_0.1.3            Biostrings_2.72.1         mvtnorm_1.2-5            
[136] parallelly_1.37.1         mnormt_2.1.1              statmod_1.5.0            
[139] RcppHNSW_0.6.0            ScaledMatrix_1.12.0       carData_3.0-5            
[142] minqa_1.2.7               pbapply_1.7-2             spam_2.10-0              
[145] gson_0.1.0                utf8_1.2.4                graphlayouts_1.1.1       
[148] gtools_3.9.5              ggsignif_0.6.4            RcppEigen_0.3.4.0.0      
[151] shiny_1.8.1.1             GenomeInfoDbData_1.2.12   glmmTMB_1.1.9            
[154] memoise_2.0.1             rmarkdown_2.27            scales_1.3.0             
[157] future_1.33.2             RANN_2.6.1                Cairo_1.6-2              
[160] spatstat.data_3.1-2       rstudioapi_0.16.0         cluster_2.1.6            
[163] whisker_0.4.1             mutoss_0.1-13             spatstat.utils_3.0-5     
[166] hms_1.1.3                 fitdistrplus_1.1-11       munsell_0.5.1            
[169] cowplot_1.1.3             colorspace_2.1-0          rlang_1.1.4              
[172] DelayedMatrixStats_1.26.0 sparseMatrixStats_1.16.0  xts_0.14.0               
[175] dotCall64_1.1-1           shinydashboard_0.7.2      ggforce_0.4.2            
[178] laeken_0.5.3              mgcv_1.9-1                xfun_0.45                
[181] e1071_1.7-14              TH.data_1.1-2             iterators_1.0.14         
[184] abind_1.4-5               GOSemSim_2.30.0           treeio_1.28.0            
[187] futile.options_1.0.1      bitops_1.0-7              Rdpack_2.6               
[190] promises_1.3.0            scatterpie_0.2.3          RSQLite_2.3.7            
[193] qvalue_2.36.0             sandwich_3.1-0            fgsea_1.30.0             
[196] DelayedArray_0.30.1       proxy_0.4-27              GO.db_3.19.1             
[199] compiler_4.4.0            prettyunits_1.2.0         boot_1.3-30              
[202] beachmat_2.20.0           listenv_0.9.1             Rcpp_1.0.13              
[205] edgeR_4.2.0               workflowr_1.7.1           BiocSingular_1.20.0      
[208] tensor_1.5                MASS_7.3-61               progress_1.2.3           
[211] BiocParallel_1.38.0       babelgene_22.9            spatstat.random_3.2-3    
[214] R6_2.5.1                  fastmap_1.2.0             multcomp_1.4-25          
[217] fastmatch_1.1-4           rstatix_0.7.2             vipor_0.4.7              
[220] TTR_0.24.4                ROCR_1.0-11               TFisher_0.2.0            
[223] rsvd_1.0.5                vcd_1.4-12                nnet_7.3-19              
[226] gtable_0.3.5              KernSmooth_2.23-24        miniUI_0.1.1.1           
[229] deldir_2.0-4              htmltools_0.5.8.1         ggthemes_5.1.0           
[232] bit64_4.0.5               spatstat.explore_3.2-7    lifecycle_1.0.4          
[235] blme_1.0-5                nloptr_2.1.1              sass_0.4.9               
[238] vctrs_0.6.5               robustbase_0.99-3         spatstat.geom_3.2-9      
[241] sn_2.1.1                  ggfun_0.1.5               future.apply_1.11.2      
[244] bslib_0.7.0               pillar_1.9.0              gplots_3.1.3.1           
[247] pcaMethods_1.96.0         locfit_1.5-9.10           jsonlite_1.8.8           
[250] GetoptLong_1.0.5         </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
