<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="A.DeMartin" />

<meta name="date" content="2024-02-21" />

<title>FbHighres</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">HumanHeartCarTrans</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">FbHighres</h1>
<h4 class="author">A.DeMartin</h4>
<h4 class="date">2024-02-21</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-11-17
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>HumanHeartCarTrans/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20240221code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20240221)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20240221code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20240221)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it
difficult for you and others to run your code on a different machine.
Change the absolute path(s) below to the suggested relative path(s) to
make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/Users/immbio/Desktop/Project/Angelina/HumanHeartCarTrans/data/Human_heart_allmerged_seurat.rds
</td>
<td style="text-align:left;">
data/Human_heart_allmerged_seurat.rds
</td>
</tr>
<tr>
<td style="text-align:left;">
/Users/immbio/Desktop/Project/Angelina/HumanHeartCarTrans/data/Human_heart_AllFb.rds
</td>
<td style="text-align:left;">
data/Human_heart_AllFb.rds
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomangeldemartinHumanHeartCarTranstreef7bd86bcfe8b24c088b7827e1091a1822af78b91targetblankf7bd86ba">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/tree/f7bd86bcfe8b24c088b7827e1091a1822af78b91" target="_blank">f7bd86b</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomangeldemartinHumanHeartCarTranstreef7bd86bcfe8b24c088b7827e1091a1822af78b91targetblankf7bd86ba"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/tree/f7bd86bcfe8b24c088b7827e1091a1822af78b91" target="_blank">f7bd86b</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory
    Ignored:    data/seurat files/

Unstaged changes:
    Modified:   analysis/FbHighres.Rmd
    Modified:   analysis/RelativeAbundances.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/FbHighres.Rmd</code>) and HTML
(<code>docs/FbHighres.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/analysis/FbHighres.Rmd" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
<td>
Oct18-3
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/angeldemartin/HumanHeartCarTrans/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/FbHighres.html" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
<td>
Oct18-3
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/analysis/FbHighres.Rmd" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
<td>
Oct24
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/angeldemartin/HumanHeartCarTrans/5bf6503e89f0251b3eaf131c688f07153582623f/docs/FbHighres.html" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
<td>
Oct24
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/analysis/FbHighres.Rmd" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
<td>
Sept18-24
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/angeldemartin/HumanHeartCarTrans/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/FbHighres.html" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
<td>
Sept18-24
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="load-packages" class="section level2">
<h2>load packages</h2>
</div>
<div id="load-file" class="section level2">
<h2>load file</h2>
<pre class="r"><code>##load merged file 
fileNam &lt;- &quot;/Users/immbio/Desktop/Project/Angelina/HumanHeartCarTrans/data/Human_heart_allmerged_seurat.rds&quot;
seuratM &lt;- readRDS(fileNam)
table(seuratM$orig.ident)</code></pre>
<p>##set color vectors</p>
<pre class="r"><code>colclusterName &lt;- c(&quot;#67001f&quot;, &quot;#D53E4F&quot;, &quot;#f4a582&quot;, &quot;#FEE08B&quot;, &quot;#003c30&quot;,&quot;#01665e&quot;,&quot;#66C2A5&quot;, &quot;#3288BD&quot;,&quot;#BEAED4&quot;, &quot;#c7eae5&quot;,&quot;#355C7D&quot;,&quot;#202547&quot;,&quot;#B45B5C&quot;,&quot;#8c510a&quot;)
names(colclusterName) &lt;- c(&quot;CM&quot;,&quot;Fb&quot;,&quot;PerivFb&quot;,&quot;VSMC&quot;,&quot;Int&quot;,&quot;BEC1&quot;, &quot;BEC2&quot;,&quot;LEC&quot;,&quot;NC&quot;,&quot;AdipoC&quot;,&quot;Mph&quot;,&quot;Mph2&quot;,&quot;Tcell&quot;,&quot;Tcell2&quot;)

coldiseaseCond &lt;- c(&quot;#dfc27d&quot;,&quot;#BE3144&quot;,&quot;#202547&quot;,&quot;#355C7D&quot;,&quot;#779d8d&quot;)
names(coldiseaseCond) &lt;- c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;)</code></pre>
</div>
<div id="subset-fb-perivfb-vsmc" class="section level2">
<h2>subset Fb, PerivFb, VSMC</h2>
<pre class="r"><code>Idents(seuratM) &lt;- seuratM$clusterName
seuratAllFb &lt;- subset(seuratM, clusterName %in% c(&quot;Fb&quot;, &quot;PerivFb&quot;, &quot;VSMC&quot;))
levels(seuratAllFb)
DimPlot(seuratAllFb, reduction = &quot;umap&quot;, cols = colclusterName, pt.size = 2)</code></pre>
</div>
<div id="rerun-seurat" class="section level2">
<h2>rerun seurat</h2>
<pre class="r"><code>seuratAllFb &lt;- NormalizeData (object = seuratAllFb)
seuratAllFb &lt;- FindVariableFeatures(object = seuratAllFb)
seuratAllFb &lt;- ScaleData(object = seuratAllFb, verbose = TRUE)
seuratAllFb &lt;- RunPCA(object=seuratAllFb, npcs = 30, verbose = FALSE)
##seuratAllFb &lt;- RunTSNE(object=seuratAllFb, reduction=&quot;pca&quot;, dims = 1:20)
seuratAllFb &lt;- RunUMAP(object=seuratAllFb, reduction=&quot;pca&quot;, dims = 1:20)
seuratAllFb &lt;- FindNeighbors(object = seuratAllFb, reduction = &quot;pca&quot;, dims= 1:20)

res &lt;- c(0.25, 0.6, 0.8, 0.4)
for (i in 1:length(res)) {
  seuratAllFb &lt;- FindClusters(object = seuratAllFb, resolution = res[i], random.seed = 1234)
}

table(seuratAllFb$orig.ident)
table(seuratAllFb$RNA_snn_res.0.25)</code></pre>
<pre class="r"><code>### save seurat object
saveRDS(seuratAllFb, file=&quot;/Users/immbio/Desktop/Project/Angelina/HumanHeartCarTrans/data/Human_heart_AllFb.rds&quot;)</code></pre>
<p>##############################end
pre-processing##############################</p>
</div>
<div id="load-file-allfb" class="section level2">
<h2>load file AllFb</h2>
<pre class="r"><code>fileNam &lt;- &quot;/Users/immbio/Desktop/Project/Angelina/HumanHeartCarTrans/data/Human_heart_AllFb.rds&quot;
seuratAllFb &lt;- readRDS(fileNam)
table(seuratAllFb$dataset)</code></pre>
<pre><code>
        331571_3-5_20231012_Hu_nucseq_USZ_HTx001         331571_4-6_20231012_Hu_nucseq_USZ_HTx002 
                                            1663                                             1029 
    334131_01-1_20231103_Hu_nucseq_USZ_HTx002_RV     334131_02-2_20231103_Hu_nucseq_USZ_HTx003_LV 
                                            2048                                             1671 
    334131_03-3_20231103_Hu_nucseq_USZ_HTx004_RV     334131_04-4_20231106_Hu_nucseq_USZ_HTx003_RV 
                                            1700                                             1281 
    334131_05-5_20231106_Hu_nucseq_USZ_HTx004_LV     334131_06-6_20231106_Hu_nucseq_USZ_HTx005_RV 
                                            2630                                             1343 
    334131_07-7_20231107_Hu_nucseq_USZ_HTx005_LV     334131_08-8_20231107_Hu_nucseq_USZ_HTx006_RV 
                                             738                                              894 
    334131_09-9_20231108_Hu_nucseq_USZ_HTx006_LV    334131_10-10_20231108_Hu_nucseq_USZ_HTx001_RV 
                                            2093                                             1287 
    336431_07-7_20231129_Hu_nucseq_USZ_HTx008_RV     336431_08-8_20231129_Hu_nucseq_USZ_EMB001_V1 
                                            1807                                              259 
    336431_09-9_20231129_Hu_nucseq_USZ_HTx008_LV    336431_10-10_20231129_Hu_nucseq_USZ_EMB005_V1 
                                            1841                                              288 
   336431_11-11_20231129_Hu_nucseq_USZ_EMB006_V1    336431_12-12_20231129_Hu_nucseq_USZ_EMB007_V1 
                                             257                                              622 
   336431_13-13_20231129_Hu_nucseq_USZ_EMB002_V1    336431_14-14_20231129_Hu_nucseq_USZ_EMB003_V1 
                                             356                                              679 
   336431_15-15_20231129_Hu_nucseq_USZ_EMB004_V1     340821_01-7_20240118_Hu_nucseq_USZ_EMB007_V2 
                                             294                                              641 
    340821_02-8_20240118_Hu_nucseq_USZ_EMB008_V2     340821_03-9_20240123_Hu_nucseq_USZ_EMB001_V3 
                                             322                                               37 
   340821_04-10_20240123_Hu_nucseq_USZ_EMB002_V3    340821_05-11_20240123_Hu_nucseq_USZ_EMB003_V3 
                                             239                                               91 
   340821_06-12_20240123_Hu_nucseq_USZ_EMB004_V3    340821_07-13_20240123_Hu_nucseq_USZ_EMB005_V3 
                                             269                                             1129 
   340821_08-14_20240123_Hu_nucseq_USZ_EMB006_V3    340821_09-15_20240123_Hu_nucseq_USZ_EMB007_V3 
                                             129                                              189 
   340821_10-16_20240123_Hu_nucseq_USZ_EMB008_V3    340821_11-17_20240123_Hu_nucseq_USZ_EMB008_V1 
                                             107                                               28 
   340821_12-18_20240123_Hu_nucseq_USZ_HTx007_RV    340821_13-19_20240123_Hu_nucseq_USZ_HTx007_LV 
                                            1667                                             1349 
     340831_1-1_20240118_Hu_nucseq_USZ_EMB001_V2      340831_2-2_20240118_Hu_nucseq_USZ_EMB002_V2 
                                             316                                             1096 
     340831_3-3_20240118_Hu_nucseq_USZ_EMB003_V2      340831_4-4_20240118_Hu_nucseq_USZ_EMB004_V2 
                                             108                                              249 
     340831_5-5_20240118_Hu_nucseq_USZ_EMB005_V2      340831_6-6_20240118_Hu_nucseq_USZ_EMB006_V2 
                                             216                                              458 
     347741_4-4_20240326_Hu_nucseq_USZ_HTx010_RV      347741_5-5_20240326_Hu_nucseq_USZ_HTx010_LV 
                                             890                                             1867 
    353921_01-1_20240515_Hu_nucseq_USZ_HTx011_RV     353921_02-2_20240515_Hu_nucseq_USZ_HTx011_LV 
                                             884                                              573 
    353921_03-3_20240515_Hu_nucseq_USZ_EMB011_V1     353921_04-4_20240515_Hu_nucseq_USZ_EMB011_V2 
                                             330                                               92 
    353921_05-5_20240515_Hu_nucseq_USZ_HTx012_RV     353921_06-6_20240515_Hu_nucseq_USZ_HTx012_LV 
                                             447                                             1212 
    353921_07-7_20240515_Hu_nucseq_USZ_EMB012_V1     353921_08-8_20240515_Hu_nucseq_USZ_EMB010_V3 
                                             282                                              216 
    353921_09-9_20240515_Hu_nucseq_USZ_HTx013_RV    353921_10-10_20240515_Hu_nucseq_USZ_HTx013_LV 
                                             601                                              579 
 353921_11-11_20240515_Hu_nucseq_USZ_EMB010_V1_1  353921_12-12_20240515_Hu_nucseq_USZ_EMB010_V1_2 
                                             399                                              107 
   353921_13-13_20240524_Hu_nucseq_USZ_HTx015_RV    353921_14-14_20240524_Hu_nucseq_USZ_EMB011_V3 
                                             905                                              290 
   353921_15-15_20240524_Hu_nucseq_USZ_EMB012_V2    353921_16-16_20240524_Hu_nucseq_USZ_EMB012_V3 
                                             696                                              884 
   353921_17-17_20240524_Hu_nucseq_USZ_HTx015_LV    353921_18-18_20240524_Hu_nucseq_USZ_EMB013_V1 
                                            1031                                              131 
   353921_19-19_20240524_Hu_nucseq_USZ_EMB013_V2    353921_20-20_20240524_Hu_nucseq_USZ_EMB013_V3 
                                              68                                              100 
   353921_21-21_20240524_Hu_nucseq_USZ_HTx014_RV    353921_22-22_20240524_Hu_nucseq_USZ_HTx014_LV 
                                            1704                                             1571 
   353921_23-23_20240524_Hu_nucseq_USZ_EMB014_V1    353921_24-24_20240524_Hu_nucseq_USZ_EMB014_V2 
                                             338                                              163 
    359861_01-1_20240715_Hu_nucseq_USZ_HTx016_LV     359861_02-2_20240715_Hu_nucseq_USZ_HTx016_RV 
                                            3570                                             4814 
    359861_03-3_20240715_Hu_nucseq_USZ_EMB014_V3     359861_04-4_20240715_Hu_nucseq_USZ_EMB015_V1 
                                             193                                               36 
    359861_05-5_20240715_Hu_nucseq_USZ_EMB015_V2     359861_06-6_20240715_Hu_nucseq_USZ_EMB015_V3 
                                              86                                              251 
    359861_07-7_20240715_Hu_nucseq_USZ_EMB016_V1     359861_08-8_20240715_Hu_nucseq_USZ_EMB016_V2 
                                             276                                               32 
   359861_18-10_20240730_Hu_nucseq_USZ_EMB016_V3   o28576_1_08-8_20220525_Hu_nucseq_Graz_8_HH_GEM 
                                             135                                             1884 
o28576_1_10-10_20220525_Hu_nucseq_Graz_10_HH_GEM o28576_1_11-11_20220525_Hu_nucseq_Graz_11_HH_GEM 
                                            1494                                             2014 
o28576_1_12-12_20220525_Hu_nucseq_Graz_12_HH_GEM     o292731_1-1_20220818_Hu_nucseq_Graz_9_HH_GEM 
                                            2194                                             2440 
   o292731_2-2_20220818_Hu_nucseq_Graz_13_HH_GEM   o294781_01-1_20220912_Hu_nucseq_Graz_21_HH_GEM 
                                            3976                                              901 
  o294781_02-2_20220912_Hu_nucseq_Graz_22_HH_GEM   o294781_03-3_20220912_Hu_nucseq_Graz_23_HH_GEM 
                                            1040                                              404 
  o294781_04-4_20220912_Hu_nucseq_Graz_24_HH_GEM 
                                             924 </code></pre>
<pre class="r"><code>table(seuratAllFb$RNA_snn_res.0.25)</code></pre>
<pre><code>
    0     1     2     3     4     5     6     7 
24378 21315 13725 10481  3358  2109   737   341 </code></pre>
<pre class="r"><code>table(seuratAllFb$orig.ident)</code></pre>
<pre><code>
      
76444 </code></pre>
<pre class="r"><code>#### cluster_name_Fb
seuratAllFb$clusterNameAllFb &lt;- &quot;clusterNameAllFb&quot;
seuratAllFb$clusterNameAllFb[which(seuratAllFb$RNA_snn_res.0.25 %in% &quot;0&quot; )] &lt;- &quot;Fb1&quot;
seuratAllFb$clusterNameAllFb[which(seuratAllFb$RNA_snn_res.0.25 %in% &quot;1&quot; )] &lt;- &quot;PerivFb1&quot;
seuratAllFb$clusterNameAllFb[which(seuratAllFb$RNA_snn_res.0.25 %in% &quot;2&quot; )] &lt;- &quot;Fb2&quot;
seuratAllFb$clusterNameAllFb[which(seuratAllFb$RNA_snn_res.0.25 %in% &quot;3&quot; )] &lt;- &quot;Fb3&quot;
seuratAllFb$clusterNameAllFb[which(seuratAllFb$RNA_snn_res.0.25 %in% &quot;4&quot; )] &lt;- &quot;VSMC&quot;
seuratAllFb$clusterNameAllFb[which(seuratAllFb$RNA_snn_res.0.25 %in% &quot;5&quot; )] &lt;- &quot;Fb4&quot;
seuratAllFb$clusterNameAllFb[which(seuratAllFb$RNA_snn_res.0.25 %in% &quot;6&quot; )] &lt;- &quot;PerivFb2&quot;
seuratAllFb$clusterNameAllFb[which(seuratAllFb$RNA_snn_res.0.25 %in% &quot;7&quot; )] &lt;- &quot;PerivFb3&quot;

table(seuratAllFb$clusterNameAllFb)</code></pre>
<pre><code>
     Fb1      Fb2      Fb3      Fb4 PerivFb1 PerivFb2 PerivFb3     VSMC 
   24378    13725    10481     2109    21315      737      341     3358 </code></pre>
<pre class="r"><code>table(seuratAllFb$RNA_snn_res.0.25)</code></pre>
<pre><code>
    0     1     2     3     4     5     6     7 
24378 21315 13725 10481  3358  2109   737   341 </code></pre>
<pre class="r"><code>###order
Idents(seuratAllFb) &lt;- seuratAllFb$clusterNameAllFb
seuratAllFb$clusterNameAllFb &lt;- factor(seuratAllFb$clusterNameAllFb, levels=c(&quot;Fb1&quot;, &quot;Fb2&quot;, &quot;Fb3&quot;, &quot;Fb4&quot;, &quot;PerivFb1&quot;, &quot;PerivFb2&quot;, &quot;PerivFb3&quot;, &quot;VSMC&quot;))
Idents(seuratAllFb) &lt;- seuratAllFb$clusterNameAllFb
table(seuratAllFb$clusterNameAllFb)</code></pre>
<pre><code>
     Fb1      Fb2      Fb3      Fb4 PerivFb1 PerivFb2 PerivFb3     VSMC 
   24378    13725    10481     2109    21315      737      341     3358 </code></pre>
<p>##set color vectors AllFb</p>
<pre class="r"><code>colAllFb &lt;- c(&quot;#D53E4F&quot;,&quot;#f4a582&quot;,&quot;#ff7b7b&quot;,&quot;#8e0b00&quot;,&quot;#FEE08B&quot;,&quot;#42090D&quot;,&quot;#FF7B00&quot;,&quot;#FFF4DF&quot;)
names(colAllFb) &lt;- c(&quot;Fb1&quot;,&quot;PerivFb1&quot;,&quot;Fb2&quot;,&quot;Fb3&quot;,&quot;VSMC&quot;,&quot;Fb4&quot;,&quot;PerivFb2&quot;, &quot;PerivFb3&quot;)</code></pre>
</div>
<div id="umaps" class="section level2">
<h2>umaps</h2>
<pre class="r"><code>Idents(seuratAllFb) &lt;- seuratAllFb$clusterName
DimPlot(seuratAllFb, reduction = &quot;umap&quot;, pt.size = 0.5, cols = colclusterName, label = TRUE)</code></pre>
<p><img src="figure/FbHighres.Rmd/umaps-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umaps-1">
Past versions of umaps-1.png
</button>
</p>
<div id="fig-umaps-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umaps-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Idents(seuratAllFb) &lt;- seuratAllFb$clusterNameAllFb
DimPlot(seuratAllFb, reduction = &quot;umap&quot;, pt.size = 0.5, cols = colAllFb, label = TRUE)</code></pre>
<p><img src="figure/FbHighres.Rmd/umaps-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umaps-2">
Past versions of umaps-2.png
</button>
</p>
<div id="fig-umaps-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umaps-2.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(seuratAllFb, reduction = &quot;umap&quot;, pt.size = 0.5, cols = colAllFb)  + theme(legend.position = &quot;null&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umaps-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umaps-3">
Past versions of umaps-3.png
</button>
</p>
<div id="fig-umaps-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umaps-3.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Idents(seuratAllFb) &lt;- seuratAllFb$diseaseCond
DimPlot(seuratAllFb, reduction = &quot;umap&quot;, pt.size = 0.5, cols = coldiseaseCond)</code></pre>
<p><img src="figure/FbHighres.Rmd/umaps-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umaps-4">
Past versions of umaps-4.png
</button>
</p>
<div id="fig-umaps-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umaps-4.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(seuratAllFb, reduction = &quot;umap&quot;, pt.size = 0.5, cols = coldiseaseCond) + theme(legend.position = &quot;null&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umaps-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umaps-5">
Past versions of umaps-5.png
</button>
</p>
<div id="fig-umaps-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umaps-5.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="calculate-cluster-marker-genes" class="section level2">
<h2>calculate cluster marker genes</h2>
<pre class="r"><code>##cluster marker
Idents(seuratAllFb) &lt;- seuratAllFb$clusterNameAllFb
markerGenes &lt;- FindAllMarkers(seuratAllFb, only.pos=T, logfc.threshold = 0.2) %&gt;% 
  dplyr::filter(p_val_adj &lt; 0.01)</code></pre>
</div>
<div id="order-datasets" class="section level2">
<h2>order datasets</h2>
</div>
<div id="abundance-plots" class="section level2">
<h2>abundance plots</h2>
<pre class="r"><code>###dataset
datList &lt;- NULL
for(con in unique(seuratAllFb$dataset)){
  seuratSub &lt;- subset(seuratAllFb, dataset==con)
  print(dim(seuratSub))
  dat_con &lt;- as.data.frame(table(seuratSub$clusterNameAllFb)) %&gt;%
  mutate(percent=Freq/ncol(seuratSub)) %&gt;% mutate(dataset=con)
  datList[[con]] &lt;- dat_con
}</code></pre>
<pre><code>[1] 39836  1663
[1] 39836  1029
[1] 39836  2048
[1] 39836  1671
[1] 39836  1700
[1] 39836  1281
[1] 39836  2630
[1] 39836  1343
[1] 39836   738
[1] 39836   894
[1] 39836  2093
[1] 39836  1287
[1] 39836  1807
[1] 39836   259
[1] 39836  1841
[1] 39836   288
[1] 39836   257
[1] 39836   622
[1] 39836   356
[1] 39836   679
[1] 39836   294
[1] 39836   641
[1] 39836   322
[1] 39836    37
[1] 39836   239
[1] 39836    91
[1] 39836   269
[1] 39836  1129
[1] 39836   129
[1] 39836   189
[1] 39836   107
[1] 39836    28
[1] 39836  1667
[1] 39836  1349
[1] 39836   316
[1] 39836  1096
[1] 39836   108
[1] 39836   249
[1] 39836   216
[1] 39836   458
[1] 39836   890
[1] 39836  1867
[1] 39836   884
[1] 39836   573
[1] 39836   330
[1] 39836    92
[1] 39836   447
[1] 39836  1212
[1] 39836   282
[1] 39836   216
[1] 39836   601
[1] 39836   579
[1] 39836   399
[1] 39836   107
[1] 39836   905
[1] 39836   290
[1] 39836   696
[1] 39836   884
[1] 39836  1031
[1] 39836   131
[1] 39836    68
[1] 39836   100
[1] 39836  1704
[1] 39836  1571
[1] 39836   338
[1] 39836   163
[1] 39836  3570
[1] 39836  4814
[1] 39836   193
[1] 39836    36
[1] 39836    86
[1] 39836   251
[1] 39836   276
[1] 39836    32
[1] 39836   135
[1] 39836  1884
[1] 39836  1494
[1] 39836  2014
[1] 39836  2194
[1] 39836  2440
[1] 39836  3976
[1] 39836   901
[1] 39836  1040
[1] 39836   404
[1] 39836   924</code></pre>
<pre class="r"><code>dat_all &lt;- do.call(&quot;rbind&quot;, datList)

## plot abundance
ggbarplot(dat_all, x= &quot;dataset&quot;, y= &quot;percent&quot;, fill = &quot;Var1&quot;, palette = colAllFb, legend = &quot;right&quot;, legend.titel = &quot;cluster&quot;, ylab = &quot;frequency&quot;)  + theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1)) + scale_x_discrete(limits=orddatasets)</code></pre>
<p><img src="figure/FbHighres.Rmd/abundance%20dataset-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-abundance-dataset-1">
Past versions of "abundance dataset-1.png"
</button>
</p>
<div id="fig-no-spaces-abundance-dataset-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/abundance dataset-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>###patient_diseaseCond
datList &lt;- NULL
for(con in unique(seuratAllFb$patient_diseaseCond)){
  seuratSub &lt;- subset(seuratAllFb, patient_diseaseCond==con)
  print(dim(seuratSub))
  dat_con &lt;- as.data.frame(table(seuratSub$clusterNameAllFb)) %&gt;%
  mutate(percent=Freq/ncol(seuratSub)) %&gt;% mutate(patient_diseaseCond=con)
  datList[[con]] &lt;- dat_con
}</code></pre>
<pre><code>[1] 39836  2950
[1] 39836  3077
[1] 39836  2952
[1] 39836  4330
[1] 39836  2081
[1] 39836  2987
[1] 39836  3648
[1] 39836   259
[1] 39836   288
[1] 39836   257
[1] 39836   622
[1] 39836   356
[1] 39836   679
[1] 39836   294
[1] 39836   641
[1] 39836   322
[1] 39836    37
[1] 39836   239
[1] 39836    91
[1] 39836   269
[1] 39836  1129
[1] 39836   129
[1] 39836   189
[1] 39836   107
[1] 39836    28
[1] 39836  3016
[1] 39836   316
[1] 39836  1096
[1] 39836   108
[1] 39836   249
[1] 39836   216
[1] 39836   458
[1] 39836  2757
[1] 39836  1457
[1] 39836   330
[1] 39836    92
[1] 39836  1659
[1] 39836   282
[1] 39836   216
[1] 39836  1180
[1] 39836   399
[1] 39836   107
[1] 39836  1936
[1] 39836   290
[1] 39836   696
[1] 39836   884
[1] 39836   131
[1] 39836    68
[1] 39836   100
[1] 39836  3275
[1] 39836   338
[1] 39836   163
[1] 39836  8384
[1] 39836   193
[1] 39836    36
[1] 39836    86
[1] 39836   251
[1] 39836   276
[1] 39836    32
[1] 39836   135
[1] 39836  1884
[1] 39836  1494
[1] 39836  2014
[1] 39836  2194
[1] 39836  2440
[1] 39836  3976
[1] 39836   901
[1] 39836  1040
[1] 39836   404
[1] 39836   924</code></pre>
<pre class="r"><code>dat_all &lt;- do.call(&quot;rbind&quot;, datList)

## plot abundance
ggbarplot(dat_all, x= &quot;patient_diseaseCond&quot;, y= &quot;percent&quot;, fill = &quot;Var1&quot;, palette = colAllFb, legend = &quot;right&quot;, legend.titel = &quot;cluster&quot;, ylab = &quot;frequency&quot;)  + theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1)) + scale_x_discrete(limits=ordpatient_diseasecond)</code></pre>
<p><img src="figure/FbHighres.Rmd/abundance%20patient_diseaseCond-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-abundance-patient_diseaseCond-1">
Past versions of "abundance patient_diseaseCond-1.png"
</button>
</p>
<div id="fig-no-spaces-abundance-patient_diseaseCond-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/abundance patient_diseaseCond-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>###diseaseCond
datList &lt;- NULL
for(con in unique(seuratAllFb$diseaseCond)){
  seuratSub &lt;- subset(seuratAllFb, diseaseCond==con)
  print(dim(seuratSub))
  dat_con &lt;- as.data.frame(table(seuratSub$clusterNameAllFb)) %&gt;%
  mutate(percent=Freq/ncol(seuratSub)) %&gt;% mutate(diseaseCond=con)
  datList[[con]] &lt;- dat_con
}</code></pre>
<pre><code>[1] 39836 45689
[1] 39836  4575
[1] 39836  4650
[1] 39836  4259
[1] 39836 17271</code></pre>
<pre class="r"><code>dat_all &lt;- do.call(&quot;rbind&quot;, datList)

## plot abundance
ggbarplot(dat_all, x= &quot;diseaseCond&quot;, y= &quot;percent&quot;, fill = &quot;Var1&quot;, palette = colAllFb, legend = &quot;right&quot;, legend.titel = &quot;cluster&quot;, ylab = &quot;frequency&quot;)  + theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1)) + scale_x_discrete(limits=orddiseaseCond)</code></pre>
<p><img src="figure/FbHighres.Rmd/abundance%20diseaseCond-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-abundance-diseaseCond-1">
Past versions of "abundance diseaseCond-1.png"
</button>
</p>
<div id="fig-no-spaces-abundance-diseaseCond-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/abundance diseaseCond-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>###subset healthy and eyplant
seuratAllFbhealexpl &lt;- subset(seuratAllFb, diseaseCond %in% c(&quot;healthy&quot;, &quot;explant&quot;))
table(seuratAllFbhealexpl$diseaseCond)</code></pre>
<pre><code>
explant healthy 
  45689   17271 </code></pre>
<pre class="r"><code>###diseaseCond
datList &lt;- NULL
for(con in unique(seuratAllFbhealexpl$diseaseCond)){
  seuratSub &lt;- subset(seuratAllFbhealexpl, diseaseCond==con)
  print(dim(seuratSub))
  dat_con &lt;- as.data.frame(table(seuratSub$clusterNameAllFb)) %&gt;%
  mutate(percent=Freq/ncol(seuratSub)) %&gt;% mutate(diseaseCond=con)
  datList[[con]] &lt;- dat_con
}</code></pre>
<pre><code>[1] 39836 45689
[1] 39836 17271</code></pre>
<pre class="r"><code>dat_all &lt;- do.call(&quot;rbind&quot;, datList)

## plot abundance
ggbarplot(dat_all, x= &quot;diseaseCond&quot;, y= &quot;percent&quot;, fill = &quot;Var1&quot;, palette = colAllFb, legend = &quot;right&quot;, legend.titel = &quot;cluster&quot;, ylab = &quot;frequency&quot;)  + theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1)) + scale_x_discrete(limits=orddiseaseCond2)</code></pre>
<p><img src="figure/FbHighres.Rmd/abundance%20diseaseCond%20heal%20and%20expl-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="fractions-according-to-patientsdisease-cond"
class="section level2">
<h2>fractions according to patients&amp;disease cond</h2>
<pre class="r"><code>##set order
seuratAllFb$diseaseCond &lt;- factor(seuratAllFb$diseaseCond, levels=c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;))

## 1. create data.frame with cluster counts per patient
## change &quot;RNA_snn_res.0.25&quot; to subset/cluster you&#39;re interested in ...
datFrac &lt;- data.frame(table(seuratAllFb$patient_diseaseCond, seuratAllFb$clusterNameAllFb))
colnames(datFrac) &lt;- c(&quot;patient_diseaseCond&quot;, &quot;subset&quot;, &quot;cnt&quot;)

## 2. get total counts per patient to compute relative abundances from
## I added cond here as grouping variable for the plotting later ...
datSumPat &lt;- data.frame(table(seuratAllFb$patient_diseaseCond, seuratAllFb$diseaseCond)) %&gt;% 
  filter(Freq &gt;0)
colnames(datSumPat) &lt;- c(&quot;patient_diseaseCond&quot;, &quot;diseaseCond&quot;, &quot;cntPatTot&quot;)

## 3. join data.frames to compute rel abundances per patient
datFracSum &lt;- datFrac %&gt;% left_join(., datSumPat, by = &quot;patient_diseaseCond&quot;) %&gt;% 
  mutate(relCnt = cnt/cntPatTot)

## plot barplot with abundances for each subset grouped by cond
ggbarplot(datFracSum, x = &quot;subset&quot;, y = &quot;relCnt&quot;,
          fill = &quot;diseaseCond&quot;, color = &quot;diseaseCond&quot;,
          palette = coldiseaseCond,
          add = c(&quot;mean_se&quot;, &quot;dotplot&quot;),
          add.params = list(color=&quot;black&quot;, fill=&quot;diseaseCond&quot;, size=0.2),
          position = position_dodge(0.9),
          xlab = &quot;subset&quot;,
          ylab = &quot;relative abundance&quot;,
          legend = &quot;right&quot;,
          legend.title = &quot;&quot;) +
  rotate_x_text(angle = 90) </code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-1">
Past versions of fractions-1.png
</button>
</p>
<div id="fig-fractions-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-1.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>## plot barplot with abundances for individual subsets
clusterVec &lt;- levels(seuratAllFb)
createClusterPlot &lt;- function(cluster) {
  datFracSumC &lt;- datFracSum %&gt;% filter(subset == cluster)

  ggbarplot(datFracSumC, x = &quot;diseaseCond&quot;, y = &quot;relCnt&quot;,
            fill = &quot;diseaseCond&quot;, color = &quot;diseaseCond&quot;,
            palette = coldiseaseCond,
            add = c(&quot;mean_se&quot;, &quot;dotplot&quot;),
            size = 0,
            add.params = list(color = &quot;black&quot;, fill = &quot;diseaseCond&quot;),
            position = position_dodge(0.9),
            xlab = cluster,
            ylab = &quot;relative abundance&quot;,
            legend = &quot;right&quot;,
            legend.title = &quot;&quot;) +
    stat_compare_means(method = &quot;kruskal.test&quot;, label.y = 0.0)
}
lapply(clusterVec, createClusterPlot)</code></pre>
<pre><code>[[1]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-2.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-2">
Past versions of fractions-2.png
</button>
</p>
<div id="fig-fractions-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-2.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[2]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-3.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-3">
Past versions of fractions-3.png
</button>
</p>
<div id="fig-fractions-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-3.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[3]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-4.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-4">
Past versions of fractions-4.png
</button>
</p>
<div id="fig-fractions-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-4.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[4]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-5.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-5">
Past versions of fractions-5.png
</button>
</p>
<div id="fig-fractions-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-5.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[5]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-6.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-6">
Past versions of fractions-6.png
</button>
</p>
<div id="fig-fractions-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-6.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[6]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-7.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-7">
Past versions of fractions-7.png
</button>
</p>
<div id="fig-fractions-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-7.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[7]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-8.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-8">
Past versions of fractions-8.png
</button>
</p>
<div id="fig-fractions-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-8.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[8]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-9.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-9">
Past versions of fractions-9.png
</button>
</p>
<div id="fig-fractions-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-9.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>## plot barplot with abundances for individual subsets
clusterVec &lt;- levels(seuratAllFb)
createClusterPlot &lt;- function(cluster) {
  datFracSumC &lt;- datFracSum %&gt;% filter(subset == cluster)

  ggbarplot(datFracSumC, x = &quot;diseaseCond&quot;, y = &quot;relCnt&quot;,
            fill = &quot;diseaseCond&quot;, color = &quot;diseaseCond&quot;,
            palette = coldiseaseCond,
            add = c(&quot;mean_se&quot;, &quot;jitter&quot;),
            add.params = list(color = &quot;black&quot;, fill = &quot;diseaseCond&quot;,size = 2),
            size = 5,
            position = position_dodge(0.9),
            xlab = cluster,
            ylab = &quot;relative abundance&quot;,
            legend = &quot;right&quot;,
            legend.title = &quot;&quot;) +
    stat_compare_means(method = &quot;kruskal.test&quot;, label.y = 0.0)
}
lapply(clusterVec, createClusterPlot)</code></pre>
<pre><code>[[1]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-10.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-10">
Past versions of fractions-10.png
</button>
</p>
<div id="fig-fractions-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-10.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[2]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-11.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-11">
Past versions of fractions-11.png
</button>
</p>
<div id="fig-fractions-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-11.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[3]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-12.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-12">
Past versions of fractions-12.png
</button>
</p>
<div id="fig-fractions-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-12.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[4]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-13.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-13">
Past versions of fractions-13.png
</button>
</p>
<div id="fig-fractions-13" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-13.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[5]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-14.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-14">
Past versions of fractions-14.png
</button>
</p>
<div id="fig-fractions-14" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-14.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[6]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-15.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-15">
Past versions of fractions-15.png
</button>
</p>
<div id="fig-fractions-15" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-15.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[7]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-16.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-16">
Past versions of fractions-16.png
</button>
</p>
<div id="fig-fractions-16" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-16.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[8]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-17.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-17">
Past versions of fractions-17.png
</button>
</p>
<div id="fig-fractions-17" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-17.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>## plot boxplot with abundances for individual subsets
clusterVec &lt;- levels(seuratAllFb)
createClusterPlot &lt;- function(cluster) {
  datFracSumC &lt;- datFracSum %&gt;% filter(subset == cluster)
  ggboxplot(datFracSumC, x = &quot;diseaseCond&quot;, y = &quot;relCnt&quot;, color = &quot;diseaseCond&quot;, palette = coldiseaseCond ,add = &quot;jitter&quot;, 
            add.params = list(shape = 21,        # Use shape 21 for fill + outline control
                            color = &quot;black&quot;,   # Black outline for points
                            fill = &quot;diseaseCond&quot;, # Fill based on diseaseCond
                            size = 4)) +
    ggtitle(cluster) +
    stat_compare_means(method = &quot;kruskal.test&quot;, label.y = 0.0)
}
lapply(clusterVec, createClusterPlot)</code></pre>
<pre><code>[[1]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-18.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-18">
Past versions of fractions-18.png
</button>
</p>
<div id="fig-fractions-18" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-18.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[2]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-19.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-19">
Past versions of fractions-19.png
</button>
</p>
<div id="fig-fractions-19" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-19.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[3]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-20.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-20">
Past versions of fractions-20.png
</button>
</p>
<div id="fig-fractions-20" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-20.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[4]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-21.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-21">
Past versions of fractions-21.png
</button>
</p>
<div id="fig-fractions-21" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-21.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[5]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-22.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-22">
Past versions of fractions-22.png
</button>
</p>
<div id="fig-fractions-22" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-22.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[6]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-23.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-23">
Past versions of fractions-23.png
</button>
</p>
<div id="fig-fractions-23" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-23.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[7]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-24.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-24">
Past versions of fractions-24.png
</button>
</p>
<div id="fig-fractions-24" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-24.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[8]]</code></pre>
<p><img src="figure/FbHighres.Rmd/fractions-25.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fractions-25">
Past versions of fractions-25.png
</button>
</p>
<div id="fig-fractions-25" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/fractions-25.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>avg heatmaps</p>
<pre class="r"><code>Idents(seuratAllFb) &lt;- seuratAllFb$clusterNameAllFb
levels(seuratAllFb)</code></pre>
<pre><code>[1] &quot;Fb1&quot;      &quot;Fb2&quot;      &quot;Fb3&quot;      &quot;Fb4&quot;      &quot;PerivFb1&quot; &quot;PerivFb2&quot; &quot;PerivFb3&quot; &quot;VSMC&quot;    </code></pre>
<pre class="r"><code>seurat &lt;- seuratAllFb

avgHeatmap &lt;- function(seurat, selGenes, colVecIdent, colVecCond=NULL,
                       ordVec=NULL, gapVecR=NULL, gapVecC=NULL,cc=FALSE,
                       cr=FALSE, condCol=FALSE){
  selGenes &lt;- selGenes$gene
  ## assay data
  clusterAssigned &lt;- as.data.frame(Idents(seurat)) %&gt;%
  dplyr::mutate(cell=rownames(.))
  colnames(clusterAssigned)[1] &lt;- &quot;ident&quot;
  seuratDat &lt;- GetAssayData(seurat)
  
  ## genes of interest
  genes &lt;- data.frame(gene=rownames(seurat)) %&gt;% 
    mutate(geneID=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene)) %&gt;% filter(geneID %in% selGenes)

  ## matrix with averaged cnts per ident
  logNormExpres &lt;- as.data.frame(t(as.matrix(
    seuratDat[which(rownames(seuratDat) %in% genes$gene),])))
  logNormExpres &lt;- logNormExpres %&gt;% dplyr::mutate(cell=rownames(.)) %&gt;%
    dplyr::left_join(.,clusterAssigned, by=c(&quot;cell&quot;)) %&gt;%
    dplyr::select(-cell) %&gt;% dplyr::group_by(ident) %&gt;%
    dplyr::summarise_all(mean)
  logNormExpresMa &lt;- logNormExpres %&gt;% dplyr::select(-ident) %&gt;% as.matrix()
  rownames(logNormExpresMa) &lt;- logNormExpres$ident
  logNormExpresMa &lt;- t(logNormExpresMa)
  rownames(logNormExpresMa) &lt;- gsub(&quot;^.*?\\.&quot;,&quot;&quot;,rownames(logNormExpresMa))
  
  ## remove genes if they are all the same in all groups
  ind &lt;- apply(logNormExpresMa, 1, sd) == 0
  logNormExpresMa &lt;- logNormExpresMa[!ind,]
  genes &lt;- genes[!ind,]

  ## color columns according to cluster
  annotation_col &lt;- as.data.frame(gsub(&quot;(^.*?_)&quot;,&quot;&quot;,
                                       colnames(logNormExpresMa)))%&gt;%
    dplyr::mutate(celltype=gsub(&quot;(_.*$)&quot;,&quot;&quot;,colnames(logNormExpresMa)))
  colnames(annotation_col)[1] &lt;- &quot;col1&quot;
  annotation_col &lt;- annotation_col %&gt;%
    dplyr::mutate(cond = gsub(&quot;(^[0-9]_?)&quot;,&quot;&quot;,col1)) %&gt;%
    dplyr::select(cond, celltype)
  rownames(annotation_col) &lt;- colnames(logNormExpresMa) 

  ann_colors = list(
      celltype=colVec)
  if(is.null(ann_colors$cond)){
    annotation_col$cond &lt;- NULL
  }
  
  ## adjust order
  logNormExpresMa &lt;- logNormExpresMa[selGenes,]
  if(is.null(ordVec)){
  ordVec &lt;- levels(seurat)
  ordVec &lt;-c(&quot;Fb1&quot;, &quot;Fb2&quot;, &quot;Fb3&quot;, &quot;Fb4&quot;, &quot;PerivFb1&quot;, &quot;PerivFb2&quot;, &quot;PerivFb3&quot;, &quot;VSMC&quot;)
  }
  logNormExpresMa &lt;- logNormExpresMa[,ordVec]

  ## scaled row-wise
  pheatmap(logNormExpresMa, scale=&quot;row&quot; ,treeheight_row = 0,cluster_rows = cr, 
         cluster_cols = cc, color = colorRampPalette(c(&quot;#2166AC&quot;, &quot;#F7F7F7&quot;, &quot;#B2182B&quot;))(50),
         annotation_col = annotation_col, cellwidth=15, cellheight=10,
         annotation_colors = ann_colors, gaps_row = gapVecR, gaps_col = gapVecC)
}

genesPlot &lt;- data.frame(gene=c(&quot;ABCA6&quot;, &quot;MEG3&quot;, &quot;FKBP5&quot;, &quot;ABCA8&quot;, &quot;TSC22D3&quot;, &quot;ABCA10&quot;, &quot;LAMA2&quot;, &quot;ZBTB16&quot;, &quot;LAMB1&quot;, &quot;USP53&quot;, &quot;MEG3&quot;, &quot;TNFAIP2&quot;, &quot;MEG8&quot;, &quot;PTGIR&quot;, &quot;C7&quot;, &quot;ABCA9&quot;, &quot;LSP1&quot;, &quot;TNXB&quot;, &quot;C3&quot;, &quot;ABCA10&quot;, &quot;CCN2&quot;, &quot;FN1&quot;, &quot;CCDC80&quot;, &quot;LTBP2&quot;, &quot;COL1A1&quot;, &quot;CEROX1&quot;,&quot;COL1A2&quot;, &quot;MEG3&quot;, &quot;VCAN&quot;, &quot;SERPINE2&quot;, &quot;FOS&quot;, &quot;EGR1&quot;, &quot;NR4A1&quot;, &quot;FOSB&quot;, &quot;BTG2&quot;, &quot;ZFP36&quot;, &quot;DUSP1&quot;, &quot;CCN1&quot;, &quot;FKBP5&quot;, &quot;MIR23AHG&quot;))

levels(seurat)</code></pre>
<pre><code>[1] &quot;Fb1&quot;      &quot;Fb2&quot;      &quot;Fb3&quot;      &quot;Fb4&quot;      &quot;PerivFb1&quot; &quot;PerivFb2&quot; &quot;PerivFb3&quot; &quot;VSMC&quot;    </code></pre>
<pre class="r"><code>colVec &lt;- colAllFb

# colVec &lt;- c(colPal, colPal, colPal)
# colVec &lt;- c(&quot;blue&quot;, &quot;red&quot;)

avgHeatmap(seurat, selGenes = genesPlot, colVecIdent = colVec)</code></pre>
<p><img src="figure/FbHighres.Rmd/heatmap%20top%20genes%20Fb1-4-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-heatmap-top-genes-Fb1-4-1">
Past versions of "heatmap top genes Fb1-4-1.png"
</button>
</p>
<div id="fig-no-spaces-heatmap-top-genes-Fb1-4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/heatmap top genes Fb1-4-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Idents(seuratAllFb) &lt;- seuratAllFb$clusterNameAllFb
levels(seuratAllFb)</code></pre>
<pre><code>[1] &quot;Fb1&quot;      &quot;Fb2&quot;      &quot;Fb3&quot;      &quot;Fb4&quot;      &quot;PerivFb1&quot; &quot;PerivFb2&quot; &quot;PerivFb3&quot; &quot;VSMC&quot;    </code></pre>
<pre class="r"><code>seurat &lt;- seuratAllFb

avgHeatmap &lt;- function(seurat, selGenes, colVecIdent, colVecCond=NULL,
                       ordVec=NULL, gapVecR=NULL, gapVecC=NULL,cc=FALSE,
                       cr=FALSE, condCol=FALSE){
  selGenes &lt;- selGenes$gene
  ## assay data
  clusterAssigned &lt;- as.data.frame(Idents(seurat)) %&gt;%
  dplyr::mutate(cell=rownames(.))
  colnames(clusterAssigned)[1] &lt;- &quot;ident&quot;
  seuratDat &lt;- GetAssayData(seurat)
  
  ## genes of interest
  genes &lt;- data.frame(gene=rownames(seurat)) %&gt;% 
    mutate(geneID=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene)) %&gt;% filter(geneID %in% selGenes)

  ## matrix with averaged cnts per ident
  logNormExpres &lt;- as.data.frame(t(as.matrix(
    seuratDat[which(rownames(seuratDat) %in% genes$gene),])))
  logNormExpres &lt;- logNormExpres %&gt;% dplyr::mutate(cell=rownames(.)) %&gt;%
    dplyr::left_join(.,clusterAssigned, by=c(&quot;cell&quot;)) %&gt;%
    dplyr::select(-cell) %&gt;% dplyr::group_by(ident) %&gt;%
    dplyr::summarise_all(mean)
  logNormExpresMa &lt;- logNormExpres %&gt;% dplyr::select(-ident) %&gt;% as.matrix()
  rownames(logNormExpresMa) &lt;- logNormExpres$ident
  logNormExpresMa &lt;- t(logNormExpresMa)
  rownames(logNormExpresMa) &lt;- gsub(&quot;^.*?\\.&quot;,&quot;&quot;,rownames(logNormExpresMa))
  
  ## remove genes if they are all the same in all groups
  ind &lt;- apply(logNormExpresMa, 1, sd) == 0
  logNormExpresMa &lt;- logNormExpresMa[!ind,]
  genes &lt;- genes[!ind,]

  ## color columns according to cluster
  annotation_col &lt;- as.data.frame(gsub(&quot;(^.*?_)&quot;,&quot;&quot;,
                                       colnames(logNormExpresMa)))%&gt;%
    dplyr::mutate(celltype=gsub(&quot;(_.*$)&quot;,&quot;&quot;,colnames(logNormExpresMa)))
  colnames(annotation_col)[1] &lt;- &quot;col1&quot;
  annotation_col &lt;- annotation_col %&gt;%
    dplyr::mutate(cond = gsub(&quot;(^[0-9]_?)&quot;,&quot;&quot;,col1)) %&gt;%
    dplyr::select(cond, celltype)
  rownames(annotation_col) &lt;- colnames(logNormExpresMa) 

  ann_colors = list(
      celltype=colVec)
  if(is.null(ann_colors$cond)){
    annotation_col$cond &lt;- NULL
  }
  
  ## adjust order
  logNormExpresMa &lt;- logNormExpresMa[selGenes,]
  if(is.null(ordVec)){
  ordVec &lt;- levels(seurat)
  ordVec &lt;-c(&quot;Fb1&quot;, &quot;Fb2&quot;, &quot;Fb3&quot;, &quot;Fb4&quot;, &quot;PerivFb1&quot;, &quot;PerivFb2&quot;, &quot;PerivFb3&quot;, &quot;VSMC&quot;)
  }
  logNormExpresMa &lt;- logNormExpresMa[,ordVec]

  ## scaled row-wise
  pheatmap(logNormExpresMa, scale=&quot;row&quot; ,treeheight_row = 0,cluster_rows = cr, 
         cluster_cols = cc, color = colorRampPalette(c(&quot;#2166AC&quot;, &quot;#F7F7F7&quot;, &quot;#B2182B&quot;))(50),
         annotation_col = annotation_col, cellwidth=15, cellheight=10,
         annotation_colors = ann_colors, gaps_row = gapVecR, gaps_col = gapVecC)
}

genesPlot &lt;- data.frame(gene=c(&quot;CD36&quot;, &quot;CARMN&quot;, &quot;RGS5&quot;, &quot;ABCC9&quot;, &quot;ITGA7&quot;, &quot;DLC1&quot;, &quot;NFASC&quot;, &quot;PDGFRB&quot;, &quot;AGAP2&quot;, &quot;GJC1&quot;, &quot;HIGD1B&quot;, &quot;KLHL23&quot;, &quot;DENND3&quot;, &quot;CPM&quot;, &quot;CCL2&quot;, &quot;RGS16&quot;, &quot;CXCL10&quot;, &quot;CXCL8&quot;, &quot;CXCL9&quot;, &quot;GBP1&quot;, &quot;ICAM1&quot;, &quot;IRF1&quot;, &quot;SOD2&quot;, &quot;RELB&quot;, &quot;MYH11&quot;, &quot;TBX2&quot;, &quot;MCAM&quot;, &quot;GRIP2&quot;, &quot;PPP1R12B&quot;, &quot;ACTA2&quot;, &quot;FLNA&quot;, &quot;IRAG1&quot;))

levels(seurat)</code></pre>
<pre><code>[1] &quot;Fb1&quot;      &quot;Fb2&quot;      &quot;Fb3&quot;      &quot;Fb4&quot;      &quot;PerivFb1&quot; &quot;PerivFb2&quot; &quot;PerivFb3&quot; &quot;VSMC&quot;    </code></pre>
<pre class="r"><code>colVec &lt;- colAllFb

# colVec &lt;- c(colPal, colPal, colPal)
# colVec &lt;- c(&quot;blue&quot;, &quot;red&quot;)

avgHeatmap(seurat, selGenes = genesPlot, colVecIdent = colVec)</code></pre>
<p><img src="figure/FbHighres.Rmd/heatmap%20top%20genes%20PerivFb1-3%20VSMC-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-heatmap-top-genes-PerivFb1-3-VSMC-1">
Past versions of "heatmap top genes PerivFb1-3 VSMC-1.png"
</button>
</p>
<div id="fig-no-spaces-heatmap-top-genes-PerivFb1-3-VSMC-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/heatmap top genes PerivFb1-3 VSMC-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Idents(seuratAllFb) &lt;- seuratAllFb$clusterNameAllFb
levels(seuratAllFb)</code></pre>
<pre><code>[1] &quot;Fb1&quot;      &quot;Fb2&quot;      &quot;Fb3&quot;      &quot;Fb4&quot;      &quot;PerivFb1&quot; &quot;PerivFb2&quot; &quot;PerivFb3&quot; &quot;VSMC&quot;    </code></pre>
<pre class="r"><code>seurat &lt;- seuratAllFb

avgHeatmap &lt;- function(seurat, selGenes, colVecIdent, colVecCond=NULL,
                       ordVec=NULL, gapVecR=NULL, gapVecC=NULL,cc=FALSE,
                       cr=FALSE, condCol=FALSE){
  selGenes &lt;- selGenes$gene
  ## assay data
  clusterAssigned &lt;- as.data.frame(Idents(seurat)) %&gt;%
  dplyr::mutate(cell=rownames(.))
  colnames(clusterAssigned)[1] &lt;- &quot;ident&quot;
  seuratDat &lt;- GetAssayData(seurat)
  
  ## genes of interest
  genes &lt;- data.frame(gene=rownames(seurat)) %&gt;% 
    mutate(geneID=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene)) %&gt;% filter(geneID %in% selGenes)

  ## matrix with averaged cnts per ident
  logNormExpres &lt;- as.data.frame(t(as.matrix(
    seuratDat[which(rownames(seuratDat) %in% genes$gene),])))
  logNormExpres &lt;- logNormExpres %&gt;% dplyr::mutate(cell=rownames(.)) %&gt;%
    dplyr::left_join(.,clusterAssigned, by=c(&quot;cell&quot;)) %&gt;%
    dplyr::select(-cell) %&gt;% dplyr::group_by(ident) %&gt;%
    dplyr::summarise_all(mean)
  logNormExpresMa &lt;- logNormExpres %&gt;% dplyr::select(-ident) %&gt;% as.matrix()
  rownames(logNormExpresMa) &lt;- logNormExpres$ident
  logNormExpresMa &lt;- t(logNormExpresMa)
  rownames(logNormExpresMa) &lt;- gsub(&quot;^.*?\\.&quot;,&quot;&quot;,rownames(logNormExpresMa))
  
  ## remove genes if they are all the same in all groups
  ind &lt;- apply(logNormExpresMa, 1, sd) == 0
  logNormExpresMa &lt;- logNormExpresMa[!ind,]
  genes &lt;- genes[!ind,]

  ## color columns according to cluster
  annotation_col &lt;- as.data.frame(gsub(&quot;(^.*?_)&quot;,&quot;&quot;,
                                       colnames(logNormExpresMa)))%&gt;%
    dplyr::mutate(celltype=gsub(&quot;(_.*$)&quot;,&quot;&quot;,colnames(logNormExpresMa)))
  colnames(annotation_col)[1] &lt;- &quot;col1&quot;
  annotation_col &lt;- annotation_col %&gt;%
    dplyr::mutate(cond = gsub(&quot;(^[0-9]_?)&quot;,&quot;&quot;,col1)) %&gt;%
    dplyr::select(cond, celltype)
  rownames(annotation_col) &lt;- colnames(logNormExpresMa) 

  ann_colors = list(
      celltype=colVec)
  if(is.null(ann_colors$cond)){
    annotation_col$cond &lt;- NULL
  }
  
  ## adjust order
  logNormExpresMa &lt;- logNormExpresMa[selGenes,]
  if(is.null(ordVec)){
  ordVec &lt;- levels(seurat)
  ordVec &lt;-c(&quot;Fb1&quot;, &quot;Fb2&quot;, &quot;Fb3&quot;, &quot;Fb4&quot;, &quot;PerivFb1&quot;, &quot;PerivFb2&quot;, &quot;PerivFb3&quot;, &quot;VSMC&quot;)
  }
  logNormExpresMa &lt;- logNormExpresMa[,ordVec]

  ## scaled row-wise
  pheatmap(logNormExpresMa, scale=&quot;row&quot; ,treeheight_row = 0,cluster_rows = cr, 
         cluster_cols = cc, color = colorRampPalette(c(&quot;#2166AC&quot;, &quot;#F7F7F7&quot;, &quot;#B2182B&quot;))(50),
         annotation_col = annotation_col, cellwidth=15, cellheight=10,
         annotation_colors = ann_colors, gaps_row = gapVecR, gaps_col = gapVecC)
}

genesPlot &lt;- data.frame(gene=c(&quot;CCL2&quot;,&quot;CCL11&quot; ,&quot;CCL19&quot;, &quot;CCL21&quot;, &quot;CXCL14&quot;, &quot;CXCL10&quot;, &quot;CXCL1&quot;, &quot;CXCL9&quot;, &quot;CXCL12&quot;, &quot;CXCL2&quot;, &quot;IL6R&quot;, &quot;IL32&quot;, &quot;IL15RA&quot;,&quot;IL4R&quot;,&quot;IL10RA&quot;, &quot;IL15&quot;, &quot;IL1R1&quot;, &quot;IL11RA&quot;,&quot;IL16&quot;,&quot;IL33&quot;, &quot;IL27RA&quot;, &quot;IL17D&quot;, &quot;IL6&quot;, &quot;IL1RL1&quot;))

levels(seurat)</code></pre>
<pre><code>[1] &quot;Fb1&quot;      &quot;Fb2&quot;      &quot;Fb3&quot;      &quot;Fb4&quot;      &quot;PerivFb1&quot; &quot;PerivFb2&quot; &quot;PerivFb3&quot; &quot;VSMC&quot;    </code></pre>
<pre class="r"><code>colVec &lt;- colAllFb

# colVec &lt;- c(colPal, colPal, colPal)
# colVec &lt;- c(&quot;blue&quot;, &quot;red&quot;)

avgHeatmap(seurat, selGenes = genesPlot, colVecIdent = colVec)</code></pre>
<p><img src="figure/FbHighres.Rmd/heatmap%20chemkines%20cytokine-1.png" width="768" style="display: block; margin: auto;" /></p>
</div>
<div id="calculate-de-genes-in-fb-1-4" class="section level2">
<h2>calculate DE genes in Fb 1-4</h2>
<pre class="r"><code>## subset Fb1-4
table(seuratAllFb$clusterNameAllFb)</code></pre>
<pre><code>
     Fb1      Fb2      Fb3      Fb4 PerivFb1 PerivFb2 PerivFb3     VSMC 
   24378    13725    10481     2109    21315      737      341     3358 </code></pre>
<pre class="r"><code>seuratFb1to4 &lt;- subset(seuratAllFb, clusterNameAllFb %in% c(&quot;Fb1&quot;, &quot;Fb2&quot;, &quot;Fb3&quot;, &quot;Fb4&quot;))
table(seuratFb1to4$clusterNameAllFb)</code></pre>
<pre><code>
     Fb1      Fb2      Fb3      Fb4 PerivFb1 PerivFb2 PerivFb3     VSMC 
   24378    13725    10481     2109        0        0        0        0 </code></pre>
<pre class="r"><code>Idents(seuratFb1to4) &lt;- seuratFb1to4$clusterNameAllFb
levels(seuratFb1to4)</code></pre>
<pre><code>[1] &quot;Fb1&quot; &quot;Fb2&quot; &quot;Fb3&quot; &quot;Fb4&quot;</code></pre>
<pre class="r"><code>DEGenesFb1to4 &lt;- FindAllMarkers(seuratFb1to4, only.pos=T, logfc.threshold = 0.2) %&gt;% 
  dplyr::filter(p_val_adj &lt; 0.01)</code></pre>
</div>
<div id="gsea-fb1-4" class="section level2">
<h2>GSEA Fb1-4</h2>
<pre class="r"><code>##adjust table
DEGenesFb1to4adj &lt;- DEGenesFb1to4 %&gt;% 
  mutate(Gene=gene) %&gt;%
  mutate(gene=gsub(&quot;^.*\\.&quot;, &quot;&quot;, Gene))  %&gt;%
  mutate(EnsID=gsub(&quot;\\..*&quot;,&quot;&quot;, Gene))

##GSEA Fb1
DEGenesFb1 &lt;- DEGenesFb1to4adj %&gt;% filter(cluster == &quot;Fb1&quot;)
egoFb1 &lt;- enrichGO(gene = unique(DEGenesFb1$EnsID),
                          OrgDb = org.Hs.eg.db,
                          keyType = &#39;ENSEMBL&#39;,
                          ont = &quot;BP&quot;,
                          pAdjustMethod = &quot;BH&quot;,
                          pvalueCutoff = 0.05,
                          qvalueCutoff = 0.05)
egoFb1 &lt;- setReadable(egoFb1, OrgDb = org.Hs.eg.db)
dotplot(egoFb1, showCategory=30, title = &quot;Fb1&quot;, font.size = 10)</code></pre>
<p><img src="figure/FbHighres.Rmd/GSEA%20Fb1-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-GSEA-Fb1-1">
Past versions of "GSEA Fb1-1.png"
</button>
</p>
<div id="fig-no-spaces-GSEA-Fb1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/GSEA Fb1-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="convert-to-sce" class="section level2">
<h2>convert to sce</h2>
<pre class="r"><code>##convert seurat object to sce object
sce &lt;- as.SingleCellExperiment(seuratAllFb)
genes &lt;- data.frame(geneID=rownames(sce)) %&gt;% mutate(gene=gsub(&quot;.*\\.&quot;, &quot;&quot;, geneID))
pal = colorRampPalette(c(&quot;#053061&quot;, &quot;#2166ac&quot;, &quot;#f7f7f7&quot;, &quot;#f4a582&quot;, &quot;#b2183c&quot;, &quot;#85122d&quot;))

## subset healthy and convert seurat object to sce object
seuratAllFbheal &lt;- subset(seuratAllFb, diseaseCond == &quot;healthy&quot;)
table(seuratAllFbheal$diseaseCond)</code></pre>
<pre><code>
healthy explant  visit1  visit2  visit3 
  17271       0       0       0       0 </code></pre>
<pre class="r"><code>sceheal &lt;- as.SingleCellExperiment(seuratAllFbheal)

## subset visit1 and convert seurat object to sce object
seuratAllFbvisit1 &lt;- subset(seuratAllFb, diseaseCond == &quot;visit1&quot;)
table(seuratAllFbvisit1$diseaseCond)</code></pre>
<pre><code>
healthy explant  visit1  visit2  visit3 
      0       0    4575       0       0 </code></pre>
<pre class="r"><code>scevisit1 &lt;- as.SingleCellExperiment(seuratAllFbvisit1)

## subset visit2 and convert seurat object to sce object
seuratAllFbvisit2 &lt;- subset(seuratAllFb, diseaseCond == &quot;visit2&quot;)
table(seuratAllFbvisit2$diseaseCond)</code></pre>
<pre><code>
healthy explant  visit1  visit2  visit3 
      0       0       0    4650       0 </code></pre>
<pre class="r"><code>scevisit2 &lt;- as.SingleCellExperiment(seuratAllFbvisit2)

## subset visit3 and convert seurat object to sce object
seuratAllFbvisit3 &lt;- subset(seuratAllFb, diseaseCond == &quot;visit3&quot;)
table(seuratAllFbvisit3$diseaseCond)</code></pre>
<pre><code>
healthy explant  visit1  visit2  visit3 
      0       0       0       0    4259 </code></pre>
<pre class="r"><code>scevisit3 &lt;- as.SingleCellExperiment(seuratAllFbvisit3)

## subset explant and convert seurat object to sce object
seuratAllFbexplant &lt;- subset(seuratAllFb, diseaseCond == &quot;explant&quot;)
table(seuratAllFbexplant$diseaseCond)</code></pre>
<pre><code>
healthy explant  visit1  visit2  visit3 
      0   45689       0       0       0 </code></pre>
<pre class="r"><code>sceexplant &lt;- as.SingleCellExperiment(seuratAllFbexplant)</code></pre>
<pre class="r"><code>ego1 &lt;- dplyr::filter(egoFb1@result,egoFb1@result$Description==&quot;extracellular matrix organization&quot;)
g1 &lt;- ego1$geneID
Str &lt;-(g1)
StrSub &lt;- strsplit(Str, &quot;/&quot;)
df &lt;- as.data.frame(StrSub)
colnames(df) &lt;- c(&quot;gene&quot;)

##make a count matrix of signature genes
signGenes &lt;- genes %&gt;% dplyr::filter(gene %in% df$gene)

sceSub &lt;- sce[which(rownames(sce) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceSub@assays@data$logcounts)))/nrow(signGenes)
sceSub$sign &lt;- cntMat
sceSub$sign2 &lt;- sceSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 2))
sceSub$sign2[which(sceSub$sign &gt; 2)] &lt;- 2
plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20ECM%20organization-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-ECM-organization-1">
Past versions of "umap ECM organization-1.png"
</button>
</p>
<div id="fig-no-spaces-umap-ECM-organization-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umap ECM organization-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20ECM%20organization-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-ECM-organization-2">
Past versions of "umap ECM organization-2.png"
</button>
</p>
<div id="fig-no-spaces-umap-ECM-organization-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umap ECM organization-2.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>##healthy only
scehealSub &lt;- sceheal[which(rownames(sceheal) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scehealSub@assays@data$logcounts)))/nrow(signGenes)
scehealSub$sign &lt;- cntMat
scehealSub$sign2 &lt;- scehealSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 2))
scehealSub$sign2[which(scehealSub$sign &gt; 2)] &lt;- 2
plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20ECM%20organization-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20ECM%20organization-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit1 only
scevisit1Sub &lt;- scevisit1[which(rownames(scevisit1) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit1Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit1Sub$sign &lt;- cntMat
scevisit1Sub$sign2 &lt;- scevisit1Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 2))
scevisit1Sub$sign2[which(scevisit1Sub$sign &gt; 2)] &lt;- 2
plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20ECM%20organization-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20ECM%20organization-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit2 only
scevisit2Sub &lt;- scevisit2[which(rownames(scevisit2) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit2Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit2Sub$sign &lt;- cntMat
scevisit2Sub$sign2 &lt;- scevisit2Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 2))
scevisit2Sub$sign2[which(scevisit2Sub$sign &gt; 2)] &lt;- 2
plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20ECM%20organization-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20ECM%20organization-8.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit3 only
scevisit3Sub &lt;- scevisit3[which(rownames(scevisit3) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit3Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit3Sub$sign &lt;- cntMat
scevisit3Sub$sign2 &lt;- scevisit3Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 2))
scevisit3Sub$sign2[which(scevisit3Sub$sign &gt; 2)] &lt;- 2
plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20ECM%20organization-9.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20ECM%20organization-10.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##explant only
sceexplantSub &lt;- sceexplant[which(rownames(sceexplant) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceexplantSub@assays@data$logcounts)))/nrow(signGenes)
sceexplantSub$sign &lt;- cntMat
sceexplantSub$sign2 &lt;- sceexplantSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 2))
sceexplantSub$sign2[which(sceexplantSub$sign &gt; 2)] &lt;- 2
plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20ECM%20organization-11.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20ECM%20organization-12.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##GSEA Fb2
DEGenesFb2 &lt;- DEGenesFb1to4adj %&gt;% filter(cluster == &quot;Fb2&quot;)
egoFb2 &lt;- enrichGO(gene = unique(DEGenesFb2$EnsID),
                          OrgDb = org.Hs.eg.db,
                          keyType = &#39;ENSEMBL&#39;,
                          ont = &quot;BP&quot;,
                          pAdjustMethod = &quot;BH&quot;,
                          pvalueCutoff = 0.05,
                          qvalueCutoff = 0.05)
egoFb2 &lt;- setReadable(egoFb2, OrgDb = org.Hs.eg.db)
dotplot(egoFb2, showCategory=30, title = &quot;Fb2&quot;, font.size = 10)</code></pre>
<p><img src="figure/FbHighres.Rmd/GSEA%20Fb2-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-GSEA-Fb2-1">
Past versions of "GSEA Fb2-1.png"
</button>
</p>
<div id="fig-no-spaces-GSEA-Fb2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/GSEA Fb2-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ego1 &lt;- dplyr::filter(egoFb2@result,egoFb2@result$Description==&quot;muscle cell differentiation&quot;)
g1 &lt;- ego1$geneID
Str &lt;-(g1)
StrSub &lt;- strsplit(Str, &quot;/&quot;)
df &lt;- as.data.frame(StrSub)
colnames(df) &lt;- c(&quot;gene&quot;)

##make a count matrix of signature genes
signGenes &lt;- genes %&gt;% dplyr::filter(gene %in% df$gene)

sceSub &lt;- sce[which(rownames(sce) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceSub@assays@data$logcounts)))/nrow(signGenes)
sceSub$sign &lt;- cntMat
sceSub$sign2 &lt;- sceSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.5))
sceSub$sign2[which(sceSub$sign &gt; 0.5)] &lt;- 0.5
plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20muscle%20cell%20differentiation-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-muscle-cell-differentiation-1">
Past versions of "umap muscle cell differentiation-1.png"
</button>
</p>
<div id="fig-no-spaces-umap-muscle-cell-differentiation-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umap muscle cell differentiation-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20muscle%20cell%20differentiation-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-muscle-cell-differentiation-2">
Past versions of "umap muscle cell differentiation-2.png"
</button>
</p>
<div id="fig-no-spaces-umap-muscle-cell-differentiation-2"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umap muscle cell differentiation-2.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>##healthy only
scehealSub &lt;- sceheal[which(rownames(sceheal) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scehealSub@assays@data$logcounts)))/nrow(signGenes)
scehealSub$sign &lt;- cntMat
scehealSub$sign2 &lt;- scehealSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.5))
scehealSub$sign2[which(scehealSub$sign &gt; 0.5)] &lt;- 0.5
plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20muscle%20cell%20differentiation-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20muscle%20cell%20differentiation-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit1 only
scevisit1Sub &lt;- scevisit1[which(rownames(scevisit1) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit1Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit1Sub$sign &lt;- cntMat
scevisit1Sub$sign2 &lt;- scevisit1Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.5))
scevisit1Sub$sign2[which(scevisit1Sub$sign &gt; 0.5)] &lt;- 0.5
plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20muscle%20cell%20differentiation-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20muscle%20cell%20differentiation-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit2 only
scevisit2Sub &lt;- scevisit2[which(rownames(scevisit2) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit2Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit2Sub$sign &lt;- cntMat
scevisit2Sub$sign2 &lt;- scevisit2Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.5))
scevisit2Sub$sign2[which(scevisit2Sub$sign &gt; 0.5)] &lt;- 0.5
plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20muscle%20cell%20differentiation-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20muscle%20cell%20differentiation-8.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit3 only
scevisit3Sub &lt;- scevisit3[which(rownames(scevisit3) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit3Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit3Sub$sign &lt;- cntMat
scevisit3Sub$sign2 &lt;- scevisit3Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.5))
scevisit3Sub$sign2[which(scevisit3Sub$sign &gt; 0.5)] &lt;- 0.5
plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20muscle%20cell%20differentiation-9.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20muscle%20cell%20differentiation-10.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##explant only
sceexplantSub &lt;- sceexplant[which(rownames(sceexplant) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceexplantSub@assays@data$logcounts)))/nrow(signGenes)
sceexplantSub$sign &lt;- cntMat
sceexplantSub$sign2 &lt;- sceexplantSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.5))
sceexplantSub$sign2[which(sceexplantSub$sign &gt; 0.5)] &lt;- 0.5
plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20muscle%20cell%20differentiation-11.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20muscle%20cell%20differentiation-12.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ego1 &lt;- dplyr::filter(egoFb2@result,egoFb2@result$Description==&quot;response to type II interferon&quot;)
g1 &lt;- ego1$geneID
Str &lt;-(g1)
StrSub &lt;- strsplit(Str, &quot;/&quot;)
df &lt;- as.data.frame(StrSub)
colnames(df) &lt;- c(&quot;gene&quot;)

##make a count matrix of signature genes
signGenes &lt;- genes %&gt;% dplyr::filter(gene %in% df$gene)

sceSub &lt;- sce[which(rownames(sce) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceSub@assays@data$logcounts)))/nrow(signGenes)
sceSub$sign &lt;- cntMat
sceSub$sign2 &lt;- sceSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 1))
sceSub$sign2[which(sceSub$sign &gt; 1)] &lt;- 1
plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20type%20II%20interferon-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-response-to-type-II-interferon-1">
Past versions of "umap response to type II interferon-1.png"
</button>
</p>
<div id="fig-no-spaces-umap-response-to-type-II-interferon-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umap response to type II interferon-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20type%20II%20interferon-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-response-to-type-II-interferon-2">
Past versions of "umap response to type II interferon-2.png"
</button>
</p>
<div id="fig-no-spaces-umap-response-to-type-II-interferon-2"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umap response to type II interferon-2.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>##healthy only
scehealSub &lt;- sceheal[which(rownames(sceheal) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scehealSub@assays@data$logcounts)))/nrow(signGenes)
scehealSub$sign &lt;- cntMat
scehealSub$sign2 &lt;- scehealSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 1))
scehealSub$sign2[which(scehealSub$sign &gt; 1)] &lt;- 1
plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20type%20II%20interferon-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20type%20II%20interferon-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit1 only
scevisit1Sub &lt;- scevisit1[which(rownames(scevisit1) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit1Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit1Sub$sign &lt;- cntMat
scevisit1Sub$sign2 &lt;- scevisit1Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 1))
scevisit1Sub$sign2[which(scevisit1Sub$sign &gt; 1)] &lt;- 1
plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20type%20II%20interferon-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20type%20II%20interferon-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit2 only
scevisit2Sub &lt;- scevisit2[which(rownames(scevisit2) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit2Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit2Sub$sign &lt;- cntMat
scevisit2Sub$sign2 &lt;- scevisit2Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 1))
scevisit2Sub$sign2[which(scevisit2Sub$sign &gt; 1)] &lt;- 1
plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20type%20II%20interferon-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20type%20II%20interferon-8.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit3 only
scevisit3Sub &lt;- scevisit3[which(rownames(scevisit3) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit3Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit3Sub$sign &lt;- cntMat
scevisit3Sub$sign2 &lt;- scevisit3Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 1))
scevisit3Sub$sign2[which(scevisit3Sub$sign &gt; 1)] &lt;- 1
plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20type%20II%20interferon-9.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20type%20II%20interferon-10.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##explant only
sceexplantSub &lt;- sceexplant[which(rownames(sceexplant) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceexplantSub@assays@data$logcounts)))/nrow(signGenes)
sceexplantSub$sign &lt;- cntMat
sceexplantSub$sign2 &lt;- sceexplantSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 1))
sceexplantSub$sign2[which(sceexplantSub$sign &gt; 1)] &lt;- 1
plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20type%20II%20interferon-11.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20type%20II%20interferon-12.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##GSEA Fb3
DEGenesFb3 &lt;- DEGenesFb1to4adj %&gt;% filter(cluster == &quot;Fb3&quot;)
egoFb3 &lt;- enrichGO(gene = unique(DEGenesFb3$EnsID),
                          OrgDb = org.Hs.eg.db,
                          keyType = &#39;ENSEMBL&#39;,
                          ont = &quot;BP&quot;,
                          pAdjustMethod = &quot;BH&quot;,
                          pvalueCutoff = 0.05,
                          qvalueCutoff = 0.05)
egoFb3 &lt;- setReadable(egoFb3, OrgDb = org.Hs.eg.db)
dotplot(egoFb3, showCategory=30, title = &quot;Fb3&quot;, font.size = 10)</code></pre>
<p><img src="figure/FbHighres.Rmd/GSEA%20Fb3-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-GSEA-Fb3-1">
Past versions of "GSEA Fb3-1.png"
</button>
</p>
<div id="fig-no-spaces-GSEA-Fb3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/GSEA Fb3-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ego1 &lt;- dplyr::filter(egoFb3@result,egoFb3@result$Description==&quot;Wnt signaling pathway&quot;)
g1 &lt;- ego1$geneID
Str &lt;-(g1)
StrSub &lt;- strsplit(Str, &quot;/&quot;)
df &lt;- as.data.frame(StrSub)
colnames(df) &lt;- c(&quot;gene&quot;)

##make a count matrix of signature genes
signGenes &lt;- genes %&gt;% dplyr::filter(gene %in% df$gene)

sceSub &lt;- sce[which(rownames(sce) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceSub@assays@data$logcounts)))/nrow(signGenes)
sceSub$sign &lt;- cntMat
sceSub$sign2 &lt;- sceSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.5))
sceSub$sign2[which(sceSub$sign &gt; 0.5)] &lt;- 0.5
plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20Wnt%20signaling%20pathway-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-Wnt-signaling-pathway-1">
Past versions of "umap Wnt signaling pathway-1.png"
</button>
</p>
<div id="fig-no-spaces-umap-Wnt-signaling-pathway-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umap Wnt signaling pathway-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20Wnt%20signaling%20pathway-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-Wnt-signaling-pathway-2">
Past versions of "umap Wnt signaling pathway-2.png"
</button>
</p>
<div id="fig-no-spaces-umap-Wnt-signaling-pathway-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umap Wnt signaling pathway-2.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>##healthy only
scehealSub &lt;- sceheal[which(rownames(sceheal) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scehealSub@assays@data$logcounts)))/nrow(signGenes)
scehealSub$sign &lt;- cntMat
scehealSub$sign2 &lt;- scehealSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.5))
scehealSub$sign2[which(scehealSub$sign &gt; 0.5)] &lt;- 0.5
plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20Wnt%20signaling%20pathway-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20Wnt%20signaling%20pathway-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit1 only
scevisit1Sub &lt;- scevisit1[which(rownames(scevisit1) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit1Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit1Sub$sign &lt;- cntMat
scevisit1Sub$sign2 &lt;- scevisit1Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.5))
scevisit1Sub$sign2[which(scevisit1Sub$sign &gt; 0.5)] &lt;- 0.5
plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20Wnt%20signaling%20pathway-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20Wnt%20signaling%20pathway-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit2 only
scevisit2Sub &lt;- scevisit2[which(rownames(scevisit2) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit2Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit2Sub$sign &lt;- cntMat
scevisit2Sub$sign2 &lt;- scevisit2Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.5))
scevisit2Sub$sign2[which(scevisit2Sub$sign &gt; 0.5)] &lt;- 0.5
plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20Wnt%20signaling%20pathway-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20Wnt%20signaling%20pathway-8.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit3 only
scevisit3Sub &lt;- scevisit3[which(rownames(scevisit3) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit3Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit3Sub$sign &lt;- cntMat
scevisit3Sub$sign2 &lt;- scevisit3Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.5))
scevisit3Sub$sign2[which(scevisit3Sub$sign &gt; 0.5)] &lt;- 0.5
plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20Wnt%20signaling%20pathway-9.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20Wnt%20signaling%20pathway-10.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##explant only
sceexplantSub &lt;- sceexplant[which(rownames(sceexplant) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceexplantSub@assays@data$logcounts)))/nrow(signGenes)
sceexplantSub$sign &lt;- cntMat
sceexplantSub$sign2 &lt;- sceexplantSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.5))
sceexplantSub$sign2[which(sceexplantSub$sign &gt; 0.5)] &lt;- 0.5
plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20Wnt%20signaling%20pathway-11.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20Wnt%20signaling%20pathway-12.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ego1 &lt;- dplyr::filter(egoFb3@result,egoFb3@result$Description==&quot;transforming growth factor beta receptor signaling pathway&quot;)
g1 &lt;- ego1$geneID
Str &lt;-(g1)
StrSub &lt;- strsplit(Str, &quot;/&quot;)
df &lt;- as.data.frame(StrSub)
colnames(df) &lt;- c(&quot;gene&quot;)

##make a count matrix of signature genes
signGenes &lt;- genes %&gt;% dplyr::filter(gene %in% df$gene)

sceSub &lt;- sce[which(rownames(sce) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceSub@assays@data$logcounts)))/nrow(signGenes)
sceSub$sign &lt;- cntMat
sceSub$sign2 &lt;- sceSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.8))
sceSub$sign2[which(sceSub$sign &gt; 0.8)] &lt;- 0.8
plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20transforming%20growth%20factor%20beta%20receptor%20signaling%20pathway-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-transforming-growth-factor-beta-receptor-signaling-pathway-1">
Past versions of "umap transforming growth factor beta receptor
signaling pathway-1.png"
</button>
</p>
<div
id="fig-no-spaces-umap-transforming-growth-factor-beta-receptor-signaling-pathway-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umap transforming growth factor beta receptor signaling pathway-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20transforming%20growth%20factor%20beta%20receptor%20signaling%20pathway-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-transforming-growth-factor-beta-receptor-signaling-pathway-2">
Past versions of "umap transforming growth factor beta receptor
signaling pathway-2.png"
</button>
</p>
<div
id="fig-no-spaces-umap-transforming-growth-factor-beta-receptor-signaling-pathway-2"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umap transforming growth factor beta receptor signaling pathway-2.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>##healthy only
scehealSub &lt;- sceheal[which(rownames(sceheal) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scehealSub@assays@data$logcounts)))/nrow(signGenes)
scehealSub$sign &lt;- cntMat
scehealSub$sign2 &lt;- scehealSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.8))
scehealSub$sign2[which(scehealSub$sign &gt; 0.8)] &lt;- 0.8
plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20transforming%20growth%20factor%20beta%20receptor%20signaling%20pathway-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20transforming%20growth%20factor%20beta%20receptor%20signaling%20pathway-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit1 only
scevisit1Sub &lt;- scevisit1[which(rownames(scevisit1) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit1Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit1Sub$sign &lt;- cntMat
scevisit1Sub$sign2 &lt;- scevisit1Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.8))
scevisit1Sub$sign2[which(scevisit1Sub$sign &gt; 0.8)] &lt;- 0.8
plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20transforming%20growth%20factor%20beta%20receptor%20signaling%20pathway-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20transforming%20growth%20factor%20beta%20receptor%20signaling%20pathway-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit2 only
scevisit2Sub &lt;- scevisit2[which(rownames(scevisit2) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit2Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit2Sub$sign &lt;- cntMat
scevisit2Sub$sign2 &lt;- scevisit2Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.8))
scevisit2Sub$sign2[which(scevisit2Sub$sign &gt; 0.8)] &lt;- 0.8
plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20transforming%20growth%20factor%20beta%20receptor%20signaling%20pathway-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20transforming%20growth%20factor%20beta%20receptor%20signaling%20pathway-8.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit3 only
scevisit3Sub &lt;- scevisit3[which(rownames(scevisit3) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit3Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit3Sub$sign &lt;- cntMat
scevisit3Sub$sign2 &lt;- scevisit3Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.8))
scevisit3Sub$sign2[which(scevisit3Sub$sign &gt; 0.8)] &lt;- 0.8
plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20transforming%20growth%20factor%20beta%20receptor%20signaling%20pathway-9.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20transforming%20growth%20factor%20beta%20receptor%20signaling%20pathway-10.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##explant only
sceexplantSub &lt;- sceexplant[which(rownames(sceexplant) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceexplantSub@assays@data$logcounts)))/nrow(signGenes)
sceexplantSub$sign &lt;- cntMat
sceexplantSub$sign2 &lt;- sceexplantSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.8))
sceexplantSub$sign2[which(sceexplantSub$sign &gt; 0.8)] &lt;- 0.8
plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20transforming%20growth%20factor%20beta%20receptor%20signaling%20pathway-11.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20transforming%20growth%20factor%20beta%20receptor%20signaling%20pathway-12.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ego1 &lt;- dplyr::filter(egoFb3@result,egoFb3@result$Description==&quot;BMP signaling pathway&quot;)
g1 &lt;- ego1$geneID
Str &lt;-(g1)
StrSub &lt;- strsplit(Str, &quot;/&quot;)
df &lt;- as.data.frame(StrSub)
colnames(df) &lt;- c(&quot;gene&quot;)

##make a count matrix of signature genes
signGenes &lt;- genes %&gt;% dplyr::filter(gene %in% df$gene)

sceSub &lt;- sce[which(rownames(sce) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceSub@assays@data$logcounts)))/nrow(signGenes)
sceSub$sign &lt;- cntMat
sceSub$sign2 &lt;- sceSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.8))
sceSub$sign2[which(sceSub$sign &gt; 0.8)] &lt;- 0.8
plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20BMP%20signaling%20pathway-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-BMP-signaling-pathway-1">
Past versions of "umap BMP signaling pathway-1.png"
</button>
</p>
<div id="fig-no-spaces-umap-BMP-signaling-pathway-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/FbHighres.Rmd/umap BMP signaling pathway-1.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20BMP%20signaling%20pathway-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-BMP-signaling-pathway-2">
Past versions of "umap BMP signaling pathway-2.png"
</button>
</p>
<div id="fig-no-spaces-umap-BMP-signaling-pathway-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/FbHighres.Rmd/umap BMP signaling pathway-2.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>##healthy only
scehealSub &lt;- sceheal[which(rownames(sceheal) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scehealSub@assays@data$logcounts)))/nrow(signGenes)
scehealSub$sign &lt;- cntMat
scehealSub$sign2 &lt;- scehealSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.8))
scehealSub$sign2[which(scehealSub$sign &gt; 0.8)] &lt;- 0.8
plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20BMP%20signaling%20pathway-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20BMP%20signaling%20pathway-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit1 only
scevisit1Sub &lt;- scevisit1[which(rownames(scevisit1) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit1Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit1Sub$sign &lt;- cntMat
scevisit1Sub$sign2 &lt;- scevisit1Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.8))
scevisit1Sub$sign2[which(scevisit1Sub$sign &gt; 0.8)] &lt;- 0.8
plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20BMP%20signaling%20pathway-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20BMP%20signaling%20pathway-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit2 only
scevisit2Sub &lt;- scevisit2[which(rownames(scevisit2) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit2Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit2Sub$sign &lt;- cntMat
scevisit2Sub$sign2 &lt;- scevisit2Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.8))
scevisit2Sub$sign2[which(scevisit2Sub$sign &gt; 0.8)] &lt;- 0.8
plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20BMP%20signaling%20pathway-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20BMP%20signaling%20pathway-8.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit3 only
scevisit3Sub &lt;- scevisit3[which(rownames(scevisit3) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit3Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit3Sub$sign &lt;- cntMat
scevisit3Sub$sign2 &lt;- scevisit3Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.8))
scevisit3Sub$sign2[which(scevisit3Sub$sign &gt; 0.8)] &lt;- 0.8
plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20BMP%20signaling%20pathway-9.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20BMP%20signaling%20pathway-10.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##explant only
sceexplantSub &lt;- sceexplant[which(rownames(sceexplant) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceexplantSub@assays@data$logcounts)))/nrow(signGenes)
sceexplantSub$sign &lt;- cntMat
sceexplantSub$sign2 &lt;- sceexplantSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 0.8))
sceexplantSub$sign2[which(sceexplantSub$sign &gt; 0.8)] &lt;- 0.8
plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20BMP%20signaling%20pathway-11.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20BMP%20signaling%20pathway-12.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##GSEA Fb4
DEGenesFb4 &lt;- DEGenesFb1to4adj %&gt;% filter(cluster == &quot;Fb4&quot;)
egoFb4 &lt;- enrichGO(gene = unique(DEGenesFb4$EnsID),
                          OrgDb = org.Hs.eg.db,
                          keyType = &#39;ENSEMBL&#39;,
                          ont = &quot;BP&quot;,
                          pAdjustMethod = &quot;BH&quot;,
                          pvalueCutoff = 0.05,
                          qvalueCutoff = 0.05)
egoFb4 &lt;- setReadable(egoFb4, OrgDb = org.Hs.eg.db)
dotplot(egoFb4, showCategory=30, title = &quot;Fb4&quot;, font.size = 10)</code></pre>
<p><img src="figure/FbHighres.Rmd/GSEA%20Fb4-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-GSEA-Fb4-1">
Past versions of "GSEA Fb4-1.png"
</button>
</p>
<div id="fig-no-spaces-GSEA-Fb4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/GSEA Fb4-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ego1 &lt;- dplyr::filter(egoFb4@result,egoFb4@result$Description==&quot;response to steroid hormone&quot;)
g1 &lt;- ego1$geneID
Str &lt;-(g1)
StrSub &lt;- strsplit(Str, &quot;/&quot;)
df &lt;- as.data.frame(StrSub)
colnames(df) &lt;- c(&quot;gene&quot;)

##make a count matrix of signature genes
signGenes &lt;- genes %&gt;% dplyr::filter(gene %in% df$gene)

sceSub &lt;- sce[which(rownames(sce) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceSub@assays@data$logcounts)))/nrow(signGenes)
sceSub$sign &lt;- cntMat
sceSub$sign2 &lt;- sceSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 1))
sceSub$sign2[which(sceSub$sign &gt; 1)] &lt;- 1
plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20steroid%20hormone-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-response-to-steroid-hormone-1">
Past versions of "umap response to steroid hormone-1.png"
</button>
</p>
<div id="fig-no-spaces-umap-response-to-steroid-hormone-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umap response to steroid hormone-1.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotUMAP(sceSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20steroid%20hormone-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-umap-response-to-steroid-hormone-2">
Past versions of "umap response to steroid hormone-2.png"
</button>
</p>
<div id="fig-no-spaces-umap-response-to-steroid-hormone-2"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6397a5e809c4ced4f594bd73907a741b7a20035e/docs/figure/FbHighres.Rmd/umap response to steroid hormone-2.png" target="_blank">6397a5e</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>##healthy only
scehealSub &lt;- sceheal[which(rownames(sceheal) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scehealSub@assays@data$logcounts)))/nrow(signGenes)
scehealSub$sign &lt;- cntMat
scehealSub$sign2 &lt;- scehealSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 1))
scehealSub$sign2[which(scehealSub$sign &gt; 1)] &lt;- 1
plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20steroid%20hormone-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scehealSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20steroid%20hormone-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit1 only
scevisit1Sub &lt;- scevisit1[which(rownames(scevisit1) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit1Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit1Sub$sign &lt;- cntMat
scevisit1Sub$sign2 &lt;- scevisit1Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 1))
scevisit1Sub$sign2[which(scevisit1Sub$sign &gt; 1)] &lt;- 1
plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20steroid%20hormone-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit1Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20steroid%20hormone-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit2 only
scevisit2Sub &lt;- scevisit2[which(rownames(scevisit2) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit2Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit2Sub$sign &lt;- cntMat
scevisit2Sub$sign2 &lt;- scevisit2Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 1))
scevisit2Sub$sign2[which(scevisit2Sub$sign &gt; 1)] &lt;- 1
plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20steroid%20hormone-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit2Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20steroid%20hormone-8.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##visit3 only
scevisit3Sub &lt;- scevisit3[which(rownames(scevisit3) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    scevisit3Sub@assays@data$logcounts)))/nrow(signGenes)
scevisit3Sub$sign &lt;- cntMat
scevisit3Sub$sign2 &lt;- scevisit3Sub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 1))
scevisit3Sub$sign2[which(scevisit3Sub$sign &gt; 1)] &lt;- 1
plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20steroid%20hormone-9.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(scevisit3Sub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20steroid%20hormone-10.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##explant only
sceexplantSub &lt;- sceexplant[which(rownames(sceexplant) %in% signGenes$geneID),]
cntMat &lt;- rowSums(t(as.matrix(
    sceexplantSub@assays@data$logcounts)))/nrow(signGenes)
sceexplantSub$sign &lt;- cntMat
sceexplantSub$sign2 &lt;- sceexplantSub$sign
##check max and min values
##max(sceSub$sign)
sc &lt;- scale_colour_gradientn(colours = pal(100), limits=c(0, 1))
sceexplantSub$sign2[which(sceexplantSub$sign &gt; 1)] &lt;- 1
plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc +
  theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20steroid%20hormone-11.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plotUMAP(sceexplantSub, colour_by = &quot;sign2&quot;, point_size = 0.5) + sc</code></pre>
<p><img src="figure/FbHighres.Rmd/umap%20response%20to%20steroid%20hormone-12.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="subset-fb4" class="section level2">
<h2>subset Fb4</h2>
<pre class="r"><code>Idents(seuratAllFb) &lt;- seuratAllFb$clusterNameAllFb
seuratFb2 &lt;- subset(seuratAllFb, clusterNameAllFb %in% c(&quot;Fb2&quot;))
levels(seuratFb2)</code></pre>
<pre><code>[1] &quot;Fb2&quot;</code></pre>
<pre class="r"><code>DimPlot(seuratFb2, reduction = &quot;umap&quot;, cols = colAllFb, pt.size = 0.5)</code></pre>
<p><img src="figure/FbHighres.Rmd/subset%20Fb4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-Fb4-1">
Past versions of "subset Fb4-1.png"
</button>
</p>
<div id="fig-no-spaces-subset-Fb4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/FbHighres.Rmd/subset Fb4-1.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>seuratFb2$diseaseCond &lt;- factor(seuratFb2$diseaseCond, levels=c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;))
Idents(seuratFb2) &lt;- seuratFb2$diseaseCond
levels(seuratFb2)</code></pre>
<pre><code>[1] &quot;healthy&quot; &quot;explant&quot; &quot;visit1&quot;  &quot;visit2&quot;  &quot;visit3&quot; </code></pre>
<pre class="r"><code>VlnPlot(object=seuratFb2, features = &quot;ENSG00000172724.CCL19&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/FbHighres.Rmd/violin%20plots%20Fb2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-violin-plots-Fb2-1">
Past versions of "violin plots Fb2-1.png"
</button>
</p>
<div id="fig-no-spaces-violin-plots-Fb2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/FbHighres.Rmd/violin plots Fb2-1.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratFb2, features = &quot;ENSG00000108691.CCL2&quot;, pt.size = 0, cols = coldiseaseCond)</code></pre>
<p><img src="figure/FbHighres.Rmd/violin%20plots%20Fb2-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-violin-plots-Fb2-2">
Past versions of "violin plots Fb2-2.png"
</button>
</p>
<div id="fig-no-spaces-violin-plots-Fb2-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/FbHighres.Rmd/violin plots Fb2-2.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratFb2, features = &quot;ENSG00000172156.CCL11&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/FbHighres.Rmd/violin%20plots%20Fb2-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-violin-plots-Fb2-3">
Past versions of "violin plots Fb2-3.png"
</button>
</p>
<div id="fig-no-spaces-violin-plots-Fb2-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/FbHighres.Rmd/violin plots Fb2-3.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratFb2, features = &quot;ENSG00000125347.IRF1&quot;, pt.size = 0, cols = coldiseaseCond)</code></pre>
<p><img src="figure/FbHighres.Rmd/violin%20plots%20Fb2-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-violin-plots-Fb2-4">
Past versions of "violin plots Fb2-4.png"
</button>
</p>
<div id="fig-no-spaces-violin-plots-Fb2-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/FbHighres.Rmd/violin plots Fb2-4.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratFb2, features = &quot;ENSG00000115415.STAT1&quot;, pt.size = 0, cols = coldiseaseCond)</code></pre>
<p><img src="figure/FbHighres.Rmd/violin%20plots%20Fb2-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-violin-plots-Fb2-5">
Past versions of "violin plots Fb2-5.png"
</button>
</p>
<div id="fig-no-spaces-violin-plots-Fb2-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/FbHighres.Rmd/violin plots Fb2-5.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratFb2, features = &quot;ENSG00000169245.CXCL10&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/FbHighres.Rmd/violin%20plots%20Fb2-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-violin-plots-Fb2-6">
Past versions of "violin plots Fb2-6.png"
</button>
</p>
<div id="fig-no-spaces-violin-plots-Fb2-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/FbHighres.Rmd/violin plots Fb2-6.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratFb2, features = &quot;ENSG00000163739.CXCL1&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/FbHighres.Rmd/violin%20plots%20Fb2-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-violin-plots-Fb2-7">
Past versions of "violin plots Fb2-7.png"
</button>
</p>
<div id="fig-no-spaces-violin-plots-Fb2-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/5bf6503e89f0251b3eaf131c688f07153582623f/docs/figure/FbHighres.Rmd/violin plots Fb2-7.png" target="_blank">5bf6503</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="feature-plots" class="section level2">
<h2>feature plots</h2>
<pre class="r"><code>FeaturePlot(seuratAllFb, features = &quot;ENSG00000108821.COL1A1&quot;, pt.size = 0.5, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;))</code></pre>
<p><img src="figure/FbHighres.Rmd/feature%20plot-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-1">
Past versions of "feature plot-1.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/feature plot-1.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Idents(seuratAllFb) &lt;- seuratAllFb$clusterNameAllFb
VlnPlot(object=seuratAllFb, features = &quot;ENSG00000108821.COL1A1&quot;, pt.size = 0, cols = colAllFb)</code></pre>
<p><img src="figure/FbHighres.Rmd/violin%20plots%20AllFb-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-violin-plots-AllFb-1">
Past versions of "violin plots AllFb-1.png"
</button>
</p>
<div id="fig-no-spaces-violin-plots-AllFb-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/f7bd86bcfe8b24c088b7827e1091a1822af78b91/docs/figure/FbHighres.Rmd/violin plots AllFb-1.png" target="_blank">f7bd86b</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-10-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="session-info" class="section level2">
<h2>session info</h2>
<pre class="r"><code>date()</code></pre>
<pre><code>[1] &quot;Sun Nov 17 22:44:35 2024&quot;</code></pre>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.0 (2024-04-24)
Platform: x86_64-apple-darwin20
Running under: macOS Ventura 13.7.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: Europe/Zurich
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] NCmisc_1.2.0                VennDiagram_1.7.3           futile.logger_1.4.3        
 [4] ggupset_0.4.0               gridExtra_2.3               DOSE_3.30.5                
 [7] enrichplot_1.24.4           msigdbr_7.5.1               org.Hs.eg.db_3.19.1        
[10] AnnotationDbi_1.66.0        clusterProfiler_4.12.6      multtest_2.60.0            
[13] metap_1.11                  scater_1.32.1               scuttle_1.14.0             
[16] destiny_3.18.0              circlize_0.4.16             muscat_1.18.0              
[19] viridis_0.6.5               viridisLite_0.4.2           lubridate_1.9.3            
[22] forcats_1.0.0               stringr_1.5.1               purrr_1.0.2                
[25] readr_2.1.5                 tidyr_1.3.1                 tibble_3.2.1               
[28] tidyverse_2.0.0             dplyr_1.1.4                 SingleCellExperiment_1.26.0
[31] SummarizedExperiment_1.34.0 Biobase_2.64.0              GenomicRanges_1.56.2       
[34] GenomeInfoDb_1.40.1         IRanges_2.38.1              S4Vectors_0.42.1           
[37] BiocGenerics_0.50.0         MatrixGenerics_1.16.0       matrixStats_1.4.1          
[40] pheatmap_1.0.12             ggpubr_0.6.0                ggplot2_3.5.1              
[43] Seurat_5.1.0                SeuratObject_5.0.2          sp_2.1-4                   
[46] runSeurat3_0.1.0            ExploreSCdataSeurat3_0.1.0 

loaded via a namespace (and not attached):
  [1] igraph_2.1.1              ica_1.0-3                 plotly_4.10.4            
  [4] Formula_1.2-5             zlibbioc_1.50.0           tidyselect_1.2.1         
  [7] bit_4.5.0                 doParallel_1.0.17         clue_0.3-65              
 [10] lattice_0.22-6            rjson_0.2.23              blob_1.2.4               
 [13] S4Arrays_1.4.1            pbkrtest_0.5.3            parallel_4.4.0           
 [16] png_0.1-8                 plotrix_3.8-4             ggplotify_0.1.2          
 [19] cli_3.6.3                 goftest_1.2-3             VIM_6.2.2                
 [22] variancePartition_1.34.0  BiocNeighbors_1.22.0      shadowtext_0.1.4         
 [25] uwot_0.2.2                curl_5.2.3                tidytree_0.4.6           
 [28] mime_0.12                 evaluate_1.0.1            leiden_0.4.3.1           
 [31] ComplexHeatmap_2.20.0     stringi_1.8.4             backports_1.5.0          
 [34] lmerTest_3.1-3            qqconf_1.3.2              httpuv_1.6.15            
 [37] magrittr_2.0.3            rappdirs_0.3.3            splines_4.4.0            
 [40] ggraph_2.2.1              sctransform_0.4.1         ggbeeswarm_0.7.2         
 [43] DBI_1.2.3                 jquerylib_0.1.4           smoother_1.3             
 [46] withr_3.0.1               git2r_0.35.0              corpcor_1.6.10           
 [49] reformulas_0.3.0          class_7.3-22              rprojroot_2.0.4          
 [52] lmtest_0.9-40             tidygraph_1.3.1           formatR_1.14             
 [55] colourpicker_1.3.0        htmlwidgets_1.6.4         fs_1.6.4                 
 [58] ggrepel_0.9.6             labeling_0.4.3            fANCOVA_0.6-1            
 [61] SparseArray_1.4.8         DESeq2_1.44.0             ranger_0.16.0            
 [64] DEoptimR_1.1-3            reticulate_1.39.0         hexbin_1.28.4            
 [67] zoo_1.8-12                XVector_0.44.0            knitr_1.48               
 [70] ggplot.multistats_1.0.1   UCSC.utils_1.0.0          RhpcBLASctl_0.23-42      
 [73] timechange_0.3.0          foreach_1.5.2             fansi_1.0.6              
 [76] patchwork_1.3.0           caTools_1.18.3            ggtree_3.12.0            
 [79] data.table_1.16.2         R.oo_1.26.0               RSpectra_0.16-2          
 [82] irlba_2.3.5.1             ggrastr_1.0.2             gridGraphics_0.5-1       
 [85] fastDummies_1.7.4         lazyeval_0.2.2            yaml_2.3.10              
 [88] survival_3.7-0            scattermore_1.2           crayon_1.5.3             
 [91] RcppAnnoy_0.0.22          RColorBrewer_1.1-3        progressr_0.14.0         
 [94] tweenr_2.0.3              later_1.3.2               ggridges_0.5.6           
 [97] codetools_0.2-20          GlobalOptions_0.1.2       aod_1.3.3                
[100] KEGGREST_1.44.1           Rtsne_0.17                shape_1.4.6.1            
[103] limma_3.60.6              pkgconfig_2.0.3           TMB_1.9.15               
[106] spatstat.univar_3.0-1     mathjaxr_1.6-0            EnvStats_3.0.0           
[109] aplot_0.2.3               scatterplot3d_0.3-44      ape_5.8                  
[112] spatstat.sparse_3.1-0     xtable_1.8-4              car_3.1-3                
[115] highr_0.11                plyr_1.8.9                httr_1.4.7               
[118] rbibutils_2.3             tools_4.4.0               globals_0.16.3           
[121] beeswarm_0.4.0            broom_1.0.7               nlme_3.1-166             
[124] lambda.r_1.2.4            lme4_1.1-35.5             digest_0.6.37            
[127] numDeriv_2016.8-1.1       Matrix_1.7-1              farver_2.1.2             
[130] tzdb_0.4.0                remaCor_0.0.18            reshape2_1.4.4           
[133] yulab.utils_0.1.7         glue_1.8.0                cachem_1.1.0             
[136] polyclip_1.10-7           generics_0.1.3            Biostrings_2.72.1        
[139] mvtnorm_1.3-1             presto_1.0.0              parallelly_1.38.0        
[142] mnormt_2.1.1              statmod_1.5.0             RcppHNSW_0.6.0           
[145] ScaledMatrix_1.12.0       carData_3.0-5             minqa_1.2.8              
[148] pbapply_1.7-2             httr2_1.0.5               spam_2.11-0              
[151] gson_0.1.0                utf8_1.2.4                graphlayouts_1.2.0       
[154] gtools_3.9.5              ggsignif_0.6.4            RcppEigen_0.3.4.0.2      
[157] shiny_1.9.1               GenomeInfoDbData_1.2.12   glmmTMB_1.1.10           
[160] R.utils_2.12.3            memoise_2.0.1             rmarkdown_2.28           
[163] scales_1.3.0              R.methodsS3_1.8.2         future_1.34.0            
[166] RANN_2.6.2                Cairo_1.6-2               spatstat.data_3.1-2      
[169] rstudioapi_0.17.1         cluster_2.1.6             whisker_0.4.1            
[172] mutoss_0.1-13             spatstat.utils_3.1-0      hms_1.1.3                
[175] fitdistrplus_1.2-1        munsell_0.5.1             cowplot_1.1.3            
[178] colorspace_2.1-1          rlang_1.1.4               DelayedMatrixStats_1.26.0
[181] sparseMatrixStats_1.16.0  xts_0.14.1                dotCall64_1.2            
[184] shinydashboard_0.7.2      ggforce_0.4.2             laeken_0.5.3             
[187] mgcv_1.9-1                xfun_0.48                 e1071_1.7-16             
[190] TH.data_1.1-2             iterators_1.0.14          abind_1.4-8              
[193] GOSemSim_2.30.2           treeio_1.28.0             futile.options_1.0.1     
[196] bitops_1.0-9              Rdpack_2.6.1              promises_1.3.0           
[199] scatterpie_0.2.4          RSQLite_2.3.7             qvalue_2.36.0            
[202] sandwich_3.1-1            fgsea_1.30.0              DelayedArray_0.30.1      
[205] proxy_0.4-27              GO.db_3.19.1              compiler_4.4.0           
[208] prettyunits_1.2.0         boot_1.3-31               beachmat_2.20.0          
[211] listenv_0.9.1             Rcpp_1.0.13               edgeR_4.2.2              
[214] workflowr_1.7.1           BiocSingular_1.20.0       tensor_1.5               
[217] MASS_7.3-61               progress_1.2.3            BiocParallel_1.38.0      
[220] babelgene_22.9            spatstat.random_3.3-2     R6_2.5.1                 
[223] fastmap_1.2.0             multcomp_1.4-26           fastmatch_1.1-4          
[226] rstatix_0.7.2             vipor_0.4.7               TTR_0.24.4               
[229] ROCR_1.0-11               TFisher_0.2.0             rsvd_1.0.5               
[232] vcd_1.4-13                nnet_7.3-19               gtable_0.3.5             
[235] KernSmooth_2.23-24        miniUI_0.1.1.1            deldir_2.0-4             
[238] htmltools_0.5.8.1         ggthemes_5.1.0            bit64_4.5.2              
[241] spatstat.explore_3.3-3    lifecycle_1.0.4           blme_1.0-6               
[244] nloptr_2.1.1              sass_0.4.9                vctrs_0.6.5              
[247] robustbase_0.99-4-1       spatstat.geom_3.3-3       sn_2.1.1                 
[250] ggfun_0.1.6               future.apply_1.11.2       bslib_0.8.0              
[253] pillar_1.9.0              gplots_3.2.0              pcaMethods_1.96.0        
[256] locfit_1.5-9.10           jsonlite_1.8.9            GetoptLong_1.0.5         </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.0 (2024-04-24)
Platform: x86_64-apple-darwin20
Running under: macOS Ventura 13.7.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: Europe/Zurich
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] NCmisc_1.2.0                VennDiagram_1.7.3           futile.logger_1.4.3        
 [4] ggupset_0.4.0               gridExtra_2.3               DOSE_3.30.5                
 [7] enrichplot_1.24.4           msigdbr_7.5.1               org.Hs.eg.db_3.19.1        
[10] AnnotationDbi_1.66.0        clusterProfiler_4.12.6      multtest_2.60.0            
[13] metap_1.11                  scater_1.32.1               scuttle_1.14.0             
[16] destiny_3.18.0              circlize_0.4.16             muscat_1.18.0              
[19] viridis_0.6.5               viridisLite_0.4.2           lubridate_1.9.3            
[22] forcats_1.0.0               stringr_1.5.1               purrr_1.0.2                
[25] readr_2.1.5                 tidyr_1.3.1                 tibble_3.2.1               
[28] tidyverse_2.0.0             dplyr_1.1.4                 SingleCellExperiment_1.26.0
[31] SummarizedExperiment_1.34.0 Biobase_2.64.0              GenomicRanges_1.56.2       
[34] GenomeInfoDb_1.40.1         IRanges_2.38.1              S4Vectors_0.42.1           
[37] BiocGenerics_0.50.0         MatrixGenerics_1.16.0       matrixStats_1.4.1          
[40] pheatmap_1.0.12             ggpubr_0.6.0                ggplot2_3.5.1              
[43] Seurat_5.1.0                SeuratObject_5.0.2          sp_2.1-4                   
[46] runSeurat3_0.1.0            ExploreSCdataSeurat3_0.1.0 

loaded via a namespace (and not attached):
  [1] igraph_2.1.1              ica_1.0-3                 plotly_4.10.4            
  [4] Formula_1.2-5             zlibbioc_1.50.0           tidyselect_1.2.1         
  [7] bit_4.5.0                 doParallel_1.0.17         clue_0.3-65              
 [10] lattice_0.22-6            rjson_0.2.23              blob_1.2.4               
 [13] S4Arrays_1.4.1            pbkrtest_0.5.3            parallel_4.4.0           
 [16] png_0.1-8                 plotrix_3.8-4             ggplotify_0.1.2          
 [19] cli_3.6.3                 goftest_1.2-3             VIM_6.2.2                
 [22] variancePartition_1.34.0  BiocNeighbors_1.22.0      shadowtext_0.1.4         
 [25] uwot_0.2.2                curl_5.2.3                tidytree_0.4.6           
 [28] mime_0.12                 evaluate_1.0.1            leiden_0.4.3.1           
 [31] ComplexHeatmap_2.20.0     stringi_1.8.4             backports_1.5.0          
 [34] lmerTest_3.1-3            qqconf_1.3.2              httpuv_1.6.15            
 [37] magrittr_2.0.3            rappdirs_0.3.3            splines_4.4.0            
 [40] ggraph_2.2.1              sctransform_0.4.1         ggbeeswarm_0.7.2         
 [43] DBI_1.2.3                 jquerylib_0.1.4           smoother_1.3             
 [46] withr_3.0.1               git2r_0.35.0              corpcor_1.6.10           
 [49] reformulas_0.3.0          class_7.3-22              rprojroot_2.0.4          
 [52] lmtest_0.9-40             tidygraph_1.3.1           formatR_1.14             
 [55] colourpicker_1.3.0        htmlwidgets_1.6.4         fs_1.6.4                 
 [58] ggrepel_0.9.6             labeling_0.4.3            fANCOVA_0.6-1            
 [61] SparseArray_1.4.8         DESeq2_1.44.0             ranger_0.16.0            
 [64] DEoptimR_1.1-3            reticulate_1.39.0         hexbin_1.28.4            
 [67] zoo_1.8-12                XVector_0.44.0            knitr_1.48               
 [70] ggplot.multistats_1.0.1   UCSC.utils_1.0.0          RhpcBLASctl_0.23-42      
 [73] timechange_0.3.0          foreach_1.5.2             fansi_1.0.6              
 [76] patchwork_1.3.0           caTools_1.18.3            ggtree_3.12.0            
 [79] data.table_1.16.2         R.oo_1.26.0               RSpectra_0.16-2          
 [82] irlba_2.3.5.1             ggrastr_1.0.2             gridGraphics_0.5-1       
 [85] fastDummies_1.7.4         lazyeval_0.2.2            yaml_2.3.10              
 [88] survival_3.7-0            scattermore_1.2           crayon_1.5.3             
 [91] RcppAnnoy_0.0.22          RColorBrewer_1.1-3        progressr_0.14.0         
 [94] tweenr_2.0.3              later_1.3.2               ggridges_0.5.6           
 [97] codetools_0.2-20          GlobalOptions_0.1.2       aod_1.3.3                
[100] KEGGREST_1.44.1           Rtsne_0.17                shape_1.4.6.1            
[103] limma_3.60.6              pkgconfig_2.0.3           TMB_1.9.15               
[106] spatstat.univar_3.0-1     mathjaxr_1.6-0            EnvStats_3.0.0           
[109] aplot_0.2.3               scatterplot3d_0.3-44      ape_5.8                  
[112] spatstat.sparse_3.1-0     xtable_1.8-4              car_3.1-3                
[115] highr_0.11                plyr_1.8.9                httr_1.4.7               
[118] rbibutils_2.3             tools_4.4.0               globals_0.16.3           
[121] beeswarm_0.4.0            broom_1.0.7               nlme_3.1-166             
[124] lambda.r_1.2.4            lme4_1.1-35.5             digest_0.6.37            
[127] numDeriv_2016.8-1.1       Matrix_1.7-1              farver_2.1.2             
[130] tzdb_0.4.0                remaCor_0.0.18            reshape2_1.4.4           
[133] yulab.utils_0.1.7         glue_1.8.0                cachem_1.1.0             
[136] polyclip_1.10-7           generics_0.1.3            Biostrings_2.72.1        
[139] mvtnorm_1.3-1             presto_1.0.0              parallelly_1.38.0        
[142] mnormt_2.1.1              statmod_1.5.0             RcppHNSW_0.6.0           
[145] ScaledMatrix_1.12.0       carData_3.0-5             minqa_1.2.8              
[148] pbapply_1.7-2             httr2_1.0.5               spam_2.11-0              
[151] gson_0.1.0                utf8_1.2.4                graphlayouts_1.2.0       
[154] gtools_3.9.5              ggsignif_0.6.4            RcppEigen_0.3.4.0.2      
[157] shiny_1.9.1               GenomeInfoDbData_1.2.12   glmmTMB_1.1.10           
[160] R.utils_2.12.3            memoise_2.0.1             rmarkdown_2.28           
[163] scales_1.3.0              R.methodsS3_1.8.2         future_1.34.0            
[166] RANN_2.6.2                Cairo_1.6-2               spatstat.data_3.1-2      
[169] rstudioapi_0.17.1         cluster_2.1.6             whisker_0.4.1            
[172] mutoss_0.1-13             spatstat.utils_3.1-0      hms_1.1.3                
[175] fitdistrplus_1.2-1        munsell_0.5.1             cowplot_1.1.3            
[178] colorspace_2.1-1          rlang_1.1.4               DelayedMatrixStats_1.26.0
[181] sparseMatrixStats_1.16.0  xts_0.14.1                dotCall64_1.2            
[184] shinydashboard_0.7.2      ggforce_0.4.2             laeken_0.5.3             
[187] mgcv_1.9-1                xfun_0.48                 e1071_1.7-16             
[190] TH.data_1.1-2             iterators_1.0.14          abind_1.4-8              
[193] GOSemSim_2.30.2           treeio_1.28.0             futile.options_1.0.1     
[196] bitops_1.0-9              Rdpack_2.6.1              promises_1.3.0           
[199] scatterpie_0.2.4          RSQLite_2.3.7             qvalue_2.36.0            
[202] sandwich_3.1-1            fgsea_1.30.0              DelayedArray_0.30.1      
[205] proxy_0.4-27              GO.db_3.19.1              compiler_4.4.0           
[208] prettyunits_1.2.0         boot_1.3-31               beachmat_2.20.0          
[211] listenv_0.9.1             Rcpp_1.0.13               edgeR_4.2.2              
[214] workflowr_1.7.1           BiocSingular_1.20.0       tensor_1.5               
[217] MASS_7.3-61               progress_1.2.3            BiocParallel_1.38.0      
[220] babelgene_22.9            spatstat.random_3.3-2     R6_2.5.1                 
[223] fastmap_1.2.0             multcomp_1.4-26           fastmatch_1.1-4          
[226] rstatix_0.7.2             vipor_0.4.7               TTR_0.24.4               
[229] ROCR_1.0-11               TFisher_0.2.0             rsvd_1.0.5               
[232] vcd_1.4-13                nnet_7.3-19               gtable_0.3.5             
[235] KernSmooth_2.23-24        miniUI_0.1.1.1            deldir_2.0-4             
[238] htmltools_0.5.8.1         ggthemes_5.1.0            bit64_4.5.2              
[241] spatstat.explore_3.3-3    lifecycle_1.0.4           blme_1.0-6               
[244] nloptr_2.1.1              sass_0.4.9                vctrs_0.6.5              
[247] robustbase_0.99-4-1       spatstat.geom_3.3-3       sn_2.1.1                 
[250] ggfun_0.1.6               future.apply_1.11.2       bslib_0.8.0              
[253] pillar_1.9.0              gplots_3.2.0              pcaMethods_1.96.0        
[256] locfit_1.5-9.10           jsonlite_1.8.9            GetoptLong_1.0.5         </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
