<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="A.DeMartin" />

<meta name="date" content="2024-02-21" />

<title>HumanHeartCarTrans</title>

<script src="site_libs/header-attrs-2.27/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">HumanHeartCarTrans</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">HumanHeartCarTrans</h1>
<h4 class="author">A.DeMartin</h4>
<h4 class="date">2024-02-21</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-09-03
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>HumanHeartCarTrans/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20240221code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20240221)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20240221code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20240221)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it
difficult for you and others to run your code on a different machine.
Change the absolute path(s) below to the suggested relative path(s) to
make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/Users/immbio/Desktop/Project/Angelina/HumanHeartCarTrans/data/seurat
files/
</td>
<td style="text-align:left;">
data/seurat files
</td>
</tr>
<tr>
<td style="text-align:left;">
/Users/immbio/Desktop/Project/Angelina/HumanHeartCarTrans/data/Human_heart_allmerged_seurat.rds
</td>
<td style="text-align:left;">
data/Human_heart_allmerged_seurat.rds
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomangeldemartinHumanHeartCarTranstree6ec9eda1708beae596b99c4a51d7ce64c634fa96targetblank6ec9edaa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/tree/6ec9eda1708beae596b99c4a51d7ce64c634fa96" target="_blank">6ec9eda</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomangeldemartinHumanHeartCarTranstree6ec9eda1708beae596b99c4a51d7ce64c634fa96targetblank6ec9edaa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/tree/6ec9eda1708beae596b99c4a51d7ce64c634fa96" target="_blank">6ec9eda</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory
    Ignored:    data/seurat files/

Unstaged changes:
    Modified:   analysis/HumanHeartCarTrans.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/HumanHeartCarTrans.Rmd</code>)
and HTML (<code>docs/HumanHeartCarTrans.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/analysis/HumanHeartCarTrans.Rmd" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
<td>
Sept24
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/angeldemartin/HumanHeartCarTrans/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/HumanHeartCarTrans.html" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
<td>
Sept24
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/angeldemartin/HumanHeartCarTrans/a155827964ac8d7e4f75a4838025a4ca1d96da00/docs/HumanHeartCarTrans.html" target="_blank">a155827</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-03-02
</td>
<td>
update march2-2
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/703f7d9a69a37cf792ea9e44396e28f9539e8c2d/analysis/HumanHeartCarTrans.Rmd" target="_blank">703f7d9</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-03-02
</td>
<td>
update march3
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/angeldemartin/HumanHeartCarTrans/703f7d9a69a37cf792ea9e44396e28f9539e8c2d/docs/HumanHeartCarTrans.html" target="_blank">703f7d9</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-03-02
</td>
<td>
update march3
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/angeldemartin/HumanHeartCarTrans/57d61d1fcc11a9b583f80064614551badcdb1f67/docs/HumanHeartCarTrans.html" target="_blank">57d61d1</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-02-29
</td>
<td>
update website
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/fe192c53b186e1a52ea024ab83149af440c03716/analysis/HumanHeartCarTrans.Rmd" target="_blank">fe192c5</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-02-29
</td>
<td>
update website
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/3b7da724c758b0362d6b5f4b162eed76f8e62a3d/analysis/HumanHeartCarTrans.Rmd" target="_blank">3b7da72</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-02-23
</td>
<td>
update gitignore
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/angeldemartin/HumanHeartCarTrans/3b7da724c758b0362d6b5f4b162eed76f8e62a3d/docs/HumanHeartCarTrans.html" target="_blank">3b7da72</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-02-23
</td>
<td>
update gitignore
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="load-packages" class="section level2">
<h2>load packages</h2>
<p>##############################start
pre-processing##############################</p>
</div>
<div id="load-files-and-merge" class="section level2">
<h2>load files and merge</h2>
<pre class="r"><code>### load and merge all 
basedir &lt;- &quot;/Users/immbio/Desktop/Project/Angelina/HumanHeartCarTrans/data/seurat files/&quot;
fileNamList &lt;- list.files(path = basedir)

for(i in 1:length(fileNamList)){
  seuratS &lt;- readRDS(paste0(basedir, fileNamList[i]))
  if(exists(&quot;seuratM&quot;)){
    seuratM &lt;- merge(x = seuratM, y = seuratS)
  }else{
    seuratM &lt;- seuratS
  }
}

remove(seuratS)
table(seuratM$dataset)
table(seuratM$orig.ident)

## exclude patient_9 (&quot;347741_1-1_20240326_Hu_nucseq_USZ_HTx009_RV&quot;, &quot;347741_2-2_20240326_Hu_nucseq_USZ_HTx009_LV&quot;)
## exclude &quot;o28576_1_09-9_20220525_Hu_nucseq_Graz_9_HH_GEM&quot; (separate CM cluster)
## exclude visit 4 sample: 347741_3-3_20240326_Hu_nucseq_USZ_EMB002_V4_inf
seuratMfilt &lt;- subset(seuratM, dataset %in% c(&quot;o28576_1_09-9_20220525_Hu_nucseq_Graz_9_HH_GEM&quot;, &quot;347741_1-1_20240326_Hu_nucseq_USZ_HTx009_RV&quot;, &quot;347741_2-2_20240326_Hu_nucseq_USZ_HTx009_LV&quot;, &quot;347741_3-3_20240326_Hu_nucseq_USZ_EMB002_V4_inf&quot;), invert = T)
unique(seuratMfilt$dataset)
table(seuratMfilt$dataset)
table(seuratMfilt$orig.ident)

seuratM &lt;- seuratMfilt
remove(seuratMfilt)
table(seuratM$orig.ident)

#rerun seurat
seuratM &lt;- NormalizeData (object = seuratM)
seuratM &lt;- FindVariableFeatures(object = seuratM)
seuratM &lt;- ScaleData(object = seuratM, verbose = TRUE)
seuratM &lt;- RunPCA(object=seuratM, npcs = 30, verbose = FALSE)
#seuratM &lt;- RunTSNE(object=seuratM, reduction=&quot;pca&quot;, dims = 1:20)
seuratM &lt;- RunUMAP(object=seuratM, reduction=&quot;pca&quot;, dims = 1:20)
seuratM &lt;- FindNeighbors(object = seuratM, reduction = &quot;pca&quot;, dims= 1:20)

res &lt;- c(0.25, 0.6, 0.8, 0.4)
for (i in 1:length(res)) {
  seuratM &lt;- FindClusters(object = seuratM, resolution = res[i], random.seed = 1234)
}</code></pre>
<pre class="r"><code>### save seurat object
saveRDS(seuratM, file=&quot;/Users/immbio/Desktop/Project/Angelina/HumanHeartCarTrans/data/Human_heart_allmerged_seurat.rds&quot;)</code></pre>
<p>##############################end
pre-processing##############################</p>
</div>
<div id="load-file" class="section level2">
<h2>load file</h2>
<pre class="r"><code>##load merged file 
fileNam &lt;- &quot;/Users/immbio/Desktop/Project/Angelina/HumanHeartCarTrans/data/Human_heart_allmerged_seurat.rds&quot;
seuratM &lt;- readRDS(fileNam)
table(seuratM$dataset)</code></pre>
<pre><code>
        331571_3-5_20231012_Hu_nucseq_USZ_HTx001         331571_4-6_20231012_Hu_nucseq_USZ_HTx002 
                                            2934                                             2059 
    334131_01-1_20231103_Hu_nucseq_USZ_HTx002_RV     334131_02-2_20231103_Hu_nucseq_USZ_HTx003_LV 
                                            3718                                             3905 
    334131_03-3_20231103_Hu_nucseq_USZ_HTx004_RV     334131_04-4_20231106_Hu_nucseq_USZ_HTx003_RV 
                                            3377                                             3377 
    334131_05-5_20231106_Hu_nucseq_USZ_HTx004_LV     334131_06-6_20231106_Hu_nucseq_USZ_HTx005_RV 
                                            4861                                             2694 
    334131_07-7_20231107_Hu_nucseq_USZ_HTx005_LV     334131_08-8_20231107_Hu_nucseq_USZ_HTx006_RV 
                                            1739                                             1672 
    334131_09-9_20231108_Hu_nucseq_USZ_HTx006_LV    334131_10-10_20231108_Hu_nucseq_USZ_HTx001_RV 
                                            3101                                             1901 
    336431_07-7_20231129_Hu_nucseq_USZ_HTx008_RV     336431_08-8_20231129_Hu_nucseq_USZ_EMB001_V1 
                                            3679                                              483 
    336431_09-9_20231129_Hu_nucseq_USZ_HTx008_LV    336431_10-10_20231129_Hu_nucseq_USZ_EMB005_V1 
                                            3082                                              606 
   336431_11-11_20231129_Hu_nucseq_USZ_EMB006_V1    336431_12-12_20231129_Hu_nucseq_USZ_EMB007_V1 
                                             420                                             1326 
   336431_13-13_20231129_Hu_nucseq_USZ_EMB002_V1    336431_14-14_20231129_Hu_nucseq_USZ_EMB003_V1 
                                            1338                                             1362 
   336431_15-15_20231129_Hu_nucseq_USZ_EMB004_V1     340821_01-7_20240118_Hu_nucseq_USZ_EMB007_V2 
                                             986                                             1600 
    340821_02-8_20240118_Hu_nucseq_USZ_EMB008_V2     340821_03-9_20240123_Hu_nucseq_USZ_EMB001_V3 
                                             567                                               68 
   340821_04-10_20240123_Hu_nucseq_USZ_EMB002_V3    340821_05-11_20240123_Hu_nucseq_USZ_EMB003_V3 
                                             349                                              196 
   340821_06-12_20240123_Hu_nucseq_USZ_EMB004_V3    340821_07-13_20240123_Hu_nucseq_USZ_EMB005_V3 
                                             745                                             1896 
   340821_08-14_20240123_Hu_nucseq_USZ_EMB006_V3    340821_09-15_20240123_Hu_nucseq_USZ_EMB007_V3 
                                             230                                              361 
   340821_10-16_20240123_Hu_nucseq_USZ_EMB008_V3    340821_11-17_20240123_Hu_nucseq_USZ_EMB008_V1 
                                             194                                               56 
   340821_12-18_20240123_Hu_nucseq_USZ_HTx007_RV    340821_13-19_20240123_Hu_nucseq_USZ_HTx007_LV 
                                            2352                                             2148 
     340831_1-1_20240118_Hu_nucseq_USZ_EMB001_V2      340831_2-2_20240118_Hu_nucseq_USZ_EMB002_V2 
                                             804                                             2575 
     340831_3-3_20240118_Hu_nucseq_USZ_EMB003_V2      340831_4-4_20240118_Hu_nucseq_USZ_EMB004_V2 
                                             323                                              521 
     340831_5-5_20240118_Hu_nucseq_USZ_EMB005_V2      340831_6-6_20240118_Hu_nucseq_USZ_EMB006_V2 
                                             654                                              770 
     347741_4-4_20240326_Hu_nucseq_USZ_HTx010_RV      347741_5-5_20240326_Hu_nucseq_USZ_HTx010_LV 
                                            2034                                             3004 
    353921_01-1_20240515_Hu_nucseq_USZ_HTx011_RV     353921_02-2_20240515_Hu_nucseq_USZ_HTx011_LV 
                                            1846                                             1351 
    353921_03-3_20240515_Hu_nucseq_USZ_EMB011_V1     353921_04-4_20240515_Hu_nucseq_USZ_EMB011_V2 
                                             738                                              193 
    353921_05-5_20240515_Hu_nucseq_USZ_HTx012_RV     353921_06-6_20240515_Hu_nucseq_USZ_HTx012_LV 
                                            1101                                             2666 
    353921_07-7_20240515_Hu_nucseq_USZ_EMB012_V1     353921_08-8_20240515_Hu_nucseq_USZ_EMB010_V3 
                                             715                                              677 
    353921_09-9_20240515_Hu_nucseq_USZ_HTx013_RV    353921_10-10_20240515_Hu_nucseq_USZ_HTx013_LV 
                                            1715                                             1965 
 353921_11-11_20240515_Hu_nucseq_USZ_EMB010_V1_1  353921_12-12_20240515_Hu_nucseq_USZ_EMB010_V1_2 
                                             996                                              265 
   353921_13-13_20240524_Hu_nucseq_USZ_HTx015_RV    353921_14-14_20240524_Hu_nucseq_USZ_EMB011_V3 
                                            1745                                              623 
   353921_15-15_20240524_Hu_nucseq_USZ_EMB012_V2    353921_16-16_20240524_Hu_nucseq_USZ_EMB012_V3 
                                            1547                                             1846 
   353921_17-17_20240524_Hu_nucseq_USZ_HTx015_LV    353921_18-18_20240524_Hu_nucseq_USZ_EMB013_V1 
                                            1448                                              379 
   353921_19-19_20240524_Hu_nucseq_USZ_EMB013_V2    353921_20-20_20240524_Hu_nucseq_USZ_EMB013_V3 
                                             138                                              327 
   353921_21-21_20240524_Hu_nucseq_USZ_HTx014_RV    353921_22-22_20240524_Hu_nucseq_USZ_HTx014_LV 
                                            2348                                             2337 
   353921_23-23_20240524_Hu_nucseq_USZ_EMB014_V1    353921_24-24_20240524_Hu_nucseq_USZ_EMB014_V2 
                                             866                                              311 
    359861_01-1_20240715_Hu_nucseq_USZ_HTx016_LV     359861_02-2_20240715_Hu_nucseq_USZ_HTx016_RV 
                                            5034                                             7278 
    359861_03-3_20240715_Hu_nucseq_USZ_EMB014_V3     359861_04-4_20240715_Hu_nucseq_USZ_EMB015_V1 
                                             395                                              134 
    359861_05-5_20240715_Hu_nucseq_USZ_EMB015_V2     359861_06-6_20240715_Hu_nucseq_USZ_EMB015_V3 
                                             207                                              449 
    359861_07-7_20240715_Hu_nucseq_USZ_EMB016_V1     359861_08-8_20240715_Hu_nucseq_USZ_EMB016_V2 
                                             547                                               94 
   359861_18-10_20240730_Hu_nucseq_USZ_EMB016_V3   o28576_1_08-8_20220525_Hu_nucseq_Graz_8_HH_GEM 
                                             287                                             3921 
o28576_1_10-10_20220525_Hu_nucseq_Graz_10_HH_GEM o28576_1_11-11_20220525_Hu_nucseq_Graz_11_HH_GEM 
                                            3731                                             3991 
o28576_1_12-12_20220525_Hu_nucseq_Graz_12_HH_GEM     o292731_1-1_20220818_Hu_nucseq_Graz_9_HH_GEM 
                                            3818                                             4908 
   o292731_2-2_20220818_Hu_nucseq_Graz_13_HH_GEM   o294781_01-1_20220912_Hu_nucseq_Graz_21_HH_GEM 
                                            9882                                             1442 
  o294781_02-2_20220912_Hu_nucseq_Graz_22_HH_GEM   o294781_03-3_20220912_Hu_nucseq_Graz_23_HH_GEM 
                                            1998                                              841 
  o294781_04-4_20220912_Hu_nucseq_Graz_24_HH_GEM 
                                            1480 </code></pre>
<pre class="r"><code>table(seuratM$RNA_snn_res.0.25)</code></pre>
<pre><code>
    0     1     2     3     4     5     6     7     8     9    10    11    12    13 
50676 24558 22350 16898 12792  6349  3418  2478  2458  2167  1844  1087   884   688 </code></pre>
<pre class="r"><code>table(seuratM$orig.ident)</code></pre>
<pre><code>
       
148647 </code></pre>
<pre class="r"><code>### add any type of metadata
### patient
pat_1 &lt;- c(&quot;331571_3-5_20231012_Hu_nucseq_USZ_HTx001&quot;,&quot;334131_10-10_20231108_Hu_nucseq_USZ_HTx001_RV&quot;,&quot;336431_08-8_20231129_Hu_nucseq_USZ_EMB001_V1&quot;, &quot;340831_1-1_20240118_Hu_nucseq_USZ_EMB001_V2&quot;, &quot;340821_03-9_20240123_Hu_nucseq_USZ_EMB001_V3&quot;)
pat_2 &lt;- c(&quot;331571_4-6_20231012_Hu_nucseq_USZ_HTx002&quot;,&quot;334131_01-1_20231103_Hu_nucseq_USZ_HTx002_RV&quot;,&quot;336431_13-13_20231129_Hu_nucseq_USZ_EMB002_V1&quot;, &quot;340831_2-2_20240118_Hu_nucseq_USZ_EMB002_V2&quot;, &quot;340821_04-10_20240123_Hu_nucseq_USZ_EMB002_V3&quot;, &quot;347741_3-3_20240326_Hu_nucseq_USZ_EMB002_V4_inf&quot;)
pat_3 &lt;- c(&quot;334131_02-2_20231103_Hu_nucseq_USZ_HTx003_LV&quot;,&quot;334131_04-4_20231106_Hu_nucseq_USZ_HTx003_RV&quot;,&quot;336431_14-14_20231129_Hu_nucseq_USZ_EMB003_V1&quot;,&quot;340831_3-3_20240118_Hu_nucseq_USZ_EMB003_V2&quot;,&quot;340821_05-11_20240123_Hu_nucseq_USZ_EMB003_V3&quot;)
pat_4 &lt;- c(&quot;334131_05-5_20231106_Hu_nucseq_USZ_HTx004_LV&quot;,&quot;334131_03-3_20231103_Hu_nucseq_USZ_HTx004_RV&quot;,&quot;336431_15-15_20231129_Hu_nucseq_USZ_EMB004_V1&quot;, &quot;340831_4-4_20240118_Hu_nucseq_USZ_EMB004_V2&quot; ,&quot;340821_06-12_20240123_Hu_nucseq_USZ_EMB004_V3&quot;)
pat_5 &lt;- c(&quot;334131_07-7_20231107_Hu_nucseq_USZ_HTx005_LV&quot;,&quot;334131_06-6_20231106_Hu_nucseq_USZ_HTx005_RV&quot;,&quot;336431_10-10_20231129_Hu_nucseq_USZ_EMB005_V1&quot;,&quot;340831_5-5_20240118_Hu_nucseq_USZ_EMB005_V2&quot; ,&quot;340821_07-13_20240123_Hu_nucseq_USZ_EMB005_V3&quot;)
pat_6 &lt;- c(&quot;334131_09-9_20231108_Hu_nucseq_USZ_HTx006_LV&quot;,&quot;334131_08-8_20231107_Hu_nucseq_USZ_HTx006_RV&quot;,&quot;336431_11-11_20231129_Hu_nucseq_USZ_EMB006_V1&quot;, &quot;340831_6-6_20240118_Hu_nucseq_USZ_EMB006_V2&quot;, &quot;340821_08-14_20240123_Hu_nucseq_USZ_EMB006_V3&quot;)
pat_7 &lt;- c(&quot;340821_13-19_20240123_Hu_nucseq_USZ_HTx007_LV&quot;,&quot;340821_12-18_20240123_Hu_nucseq_USZ_HTx007_RV&quot;,&quot;336431_12-12_20231129_Hu_nucseq_USZ_EMB007_V1&quot;,&quot;340821_01-7_20240118_Hu_nucseq_USZ_EMB007_V2&quot;,&quot;340821_09-15_20240123_Hu_nucseq_USZ_EMB007_V3&quot;)
pat_8 &lt;- c(&quot;336431_07-7_20231129_Hu_nucseq_USZ_HTx008_RV&quot;,&quot;336431_09-9_20231129_Hu_nucseq_USZ_HTx008_LV&quot;,&quot;340821_11-17_20240123_Hu_nucseq_USZ_EMB008_V1&quot;, &quot;340821_02-8_20240118_Hu_nucseq_USZ_EMB008_V2&quot;, &quot;340821_10-16_20240123_Hu_nucseq_USZ_EMB008_V3&quot;)
### excluded - pat_9 &lt;- c(&quot;347741_1-1_20240326_Hu_nucseq_USZ_HTx009_RV&quot;, &quot;347741_2-2_20240326_Hu_nucseq_USZ_HTx009_LV&quot;)
pat_10 &lt;- c(&quot;347741_4-4_20240326_Hu_nucseq_USZ_HTx010_RV&quot;,&quot;347741_5-5_20240326_Hu_nucseq_USZ_HTx010_LV&quot;,&quot;353921_11-11_20240515_Hu_nucseq_USZ_EMB010_V1_1&quot;,&quot;353921_12-12_20240515_Hu_nucseq_USZ_EMB010_V1_2&quot;,&quot;353921_08-8_20240515_Hu_nucseq_USZ_EMB010_V3&quot;)
pat_11 &lt;- c(&quot;353921_01-1_20240515_Hu_nucseq_USZ_HTx011_RV&quot;,&quot;353921_02-2_20240515_Hu_nucseq_USZ_HTx011_LV&quot;, &quot;353921_03-3_20240515_Hu_nucseq_USZ_EMB011_V1&quot;, &quot;353921_04-4_20240515_Hu_nucseq_USZ_EMB011_V2&quot;,&quot;353921_14-14_20240524_Hu_nucseq_USZ_EMB011_V3&quot;)
pat_12 &lt;- c(&quot;353921_05-5_20240515_Hu_nucseq_USZ_HTx012_RV&quot;,&quot;353921_06-6_20240515_Hu_nucseq_USZ_HTx012_LV&quot;, &quot;353921_07-7_20240515_Hu_nucseq_USZ_EMB012_V1&quot;,&quot;353921_15-15_20240524_Hu_nucseq_USZ_EMB012_V2&quot;,&quot;353921_16-16_20240524_Hu_nucseq_USZ_EMB012_V3&quot;)
pat_13 &lt;- c(&quot;353921_09-9_20240515_Hu_nucseq_USZ_HTx013_RV&quot;,&quot;353921_10-10_20240515_Hu_nucseq_USZ_HTx013_LV&quot;, &quot;353921_18-18_20240524_Hu_nucseq_USZ_EMB013_V1&quot;,&quot;353921_19-19_20240524_Hu_nucseq_USZ_EMB013_V2&quot;,&quot;353921_20-20_20240524_Hu_nucseq_USZ_EMB013_V3&quot;)
pat_14 &lt;- c(&quot;353921_21-21_20240524_Hu_nucseq_USZ_HTx014_RV&quot;,&quot;353921_22-22_20240524_Hu_nucseq_USZ_HTx014_LV&quot;,&quot;353921_23-23_20240524_Hu_nucseq_USZ_EMB014_V1&quot;,&quot;353921_24-24_20240524_Hu_nucseq_USZ_EMB014_V2&quot;, &quot;359861_03-3_20240715_Hu_nucseq_USZ_EMB014_V3&quot;)
pat_15 &lt;- c(&quot;353921_13-13_20240524_Hu_nucseq_USZ_HTx015_RV&quot;,&quot;353921_17-17_20240524_Hu_nucseq_USZ_HTx015_LV&quot;,&quot;359861_04-4_20240715_Hu_nucseq_USZ_EMB015_V1&quot;,&quot;359861_05-5_20240715_Hu_nucseq_USZ_EMB015_V2&quot;,&quot;359861_06-6_20240715_Hu_nucseq_USZ_EMB015_V3&quot;)
pat_16 &lt;- c(&quot;359861_02-2_20240715_Hu_nucseq_USZ_HTx016_RV&quot;,&quot;359861_01-1_20240715_Hu_nucseq_USZ_HTx016_LV&quot;,&quot;359861_07-7_20240715_Hu_nucseq_USZ_EMB016_V1&quot;,&quot;359861_08-8_20240715_Hu_nucseq_USZ_EMB016_V2&quot;,&quot;359861_18-10_20240730_Hu_nucseq_USZ_EMB016_V3&quot;)

HH_1 &lt;- c(&quot;o28576_1_08-8_20220525_Hu_nucseq_Graz_8_HH_GEM&quot;)
### excluded - CM cluster separately### HH_2 &lt;- c(&quot;o28576_1_09-9_20220525_Hu_nucseq_Graz_9_HH_GEM&quot;)
HH_3 &lt;- c(&quot;o28576_1_10-10_20220525_Hu_nucseq_Graz_10_HH_GEM&quot;)
HH_4 &lt;- c(&quot;o28576_1_11-11_20220525_Hu_nucseq_Graz_11_HH_GEM&quot;)
HH_5 &lt;- c(&quot;o28576_1_12-12_20220525_Hu_nucseq_Graz_12_HH_GEM&quot;)
HH_6 &lt;- c(&quot;o292731_1-1_20220818_Hu_nucseq_Graz_9_HH_GEM&quot;)
HH_7 &lt;- c(&quot;o292731_2-2_20220818_Hu_nucseq_Graz_13_HH_GEM&quot;)
HH_8 &lt;- c(&quot;o294781_01-1_20220912_Hu_nucseq_Graz_21_HH_GEM&quot;)
HH_9 &lt;- c(&quot;o294781_02-2_20220912_Hu_nucseq_Graz_22_HH_GEM&quot;)
HH_10 &lt;- c(&quot;o294781_03-3_20220912_Hu_nucseq_Graz_23_HH_GEM&quot;)
HH_11 &lt;- c(&quot;o294781_04-4_20220912_Hu_nucseq_Graz_24_HH_GEM&quot;)

seuratM$patient &lt;- &quot;pat_nr&quot;
seuratM$patient[which(seuratM$dataset %in% pat_1)] &lt;- &quot;CarTransPat1&quot;
seuratM$patient[which(seuratM$dataset %in% pat_2)] &lt;- &quot;CarTransPat2&quot;
seuratM$patient[which(seuratM$dataset %in% pat_3)] &lt;- &quot;CarTransPat3&quot;
seuratM$patient[which(seuratM$dataset %in% pat_4)] &lt;- &quot;CarTransPat4&quot;
seuratM$patient[which(seuratM$dataset %in% pat_5)] &lt;- &quot;CarTransPat5&quot;
seuratM$patient[which(seuratM$dataset %in% pat_6)] &lt;- &quot;CarTransPat6&quot;
seuratM$patient[which(seuratM$dataset %in% pat_7)] &lt;- &quot;CarTransPat7&quot;
seuratM$patient[which(seuratM$dataset %in% pat_8)] &lt;- &quot;CarTransPat8&quot;
### seuratM$patient[which(seuratM$dataset %in% pat_9)] &lt;- &quot;CarTransPat9&quot;
seuratM$patient[which(seuratM$dataset %in% pat_10)] &lt;- &quot;CarTransPat10&quot;
seuratM$patient[which(seuratM$dataset %in% pat_11)] &lt;- &quot;CarTransPat11&quot;
seuratM$patient[which(seuratM$dataset %in% pat_12)] &lt;- &quot;CarTransPat12&quot;
seuratM$patient[which(seuratM$dataset %in% pat_13)] &lt;- &quot;CarTransPat13&quot;
seuratM$patient[which(seuratM$dataset %in% pat_14)] &lt;- &quot;CarTransPat14&quot;
seuratM$patient[which(seuratM$dataset %in% pat_15)] &lt;- &quot;CarTransPat15&quot;
seuratM$patient[which(seuratM$dataset %in% pat_16)] &lt;- &quot;CarTransPat16&quot;
seuratM$patient[which(seuratM$dataset %in% HH_1)] &lt;- &quot;HH1&quot;
seuratM$patient[which(seuratM$dataset %in% HH_3)] &lt;- &quot;HH3&quot;
seuratM$patient[which(seuratM$dataset %in% HH_4)] &lt;- &quot;HH4&quot;
seuratM$patient[which(seuratM$dataset %in% HH_5)] &lt;- &quot;HH5&quot;
seuratM$patient[which(seuratM$dataset %in% HH_6)] &lt;- &quot;HH6&quot;
seuratM$patient[which(seuratM$dataset %in% HH_7)] &lt;- &quot;HH7&quot;
seuratM$patient[which(seuratM$dataset %in% HH_8)] &lt;- &quot;HH8&quot;
seuratM$patient[which(seuratM$dataset %in% HH_9)] &lt;- &quot;HH9&quot;
seuratM$patient[which(seuratM$dataset %in% HH_10)] &lt;- &quot;HH10&quot;
seuratM$patient[which(seuratM$dataset %in% HH_11)] &lt;- &quot;HH2&quot;

table(seuratM$patient)</code></pre>
<pre><code>
 CarTransPat1 CarTransPat10 CarTransPat11 CarTransPat12 CarTransPat13 CarTransPat14 CarTransPat15 
         6190          6976          4751          7875          4524          6257          3983 
CarTransPat16  CarTransPat2  CarTransPat3  CarTransPat4  CarTransPat5  CarTransPat6  CarTransPat7 
        13240         10039          9163         10490          7589          6193          7787 
 CarTransPat8           HH1          HH10           HH2           HH3           HH4           HH5 
         7578          3921           841          1480          3731          3991          3818 
          HH6           HH7           HH8           HH9 
         4908          9882          1442          1998 </code></pre>
<pre class="r"><code>orddatasets &lt;- c(&quot;o28576_1_08-8_20220525_Hu_nucseq_Graz_8_HH_GEM&quot;,&quot;o28576_1_10-10_20220525_Hu_nucseq_Graz_10_HH_GEM&quot;,&quot;o28576_1_11-11_20220525_Hu_nucseq_Graz_11_HH_GEM&quot;,&quot;o28576_1_12-12_20220525_Hu_nucseq_Graz_12_HH_GEM&quot;,&quot;o292731_1-1_20220818_Hu_nucseq_Graz_9_HH_GEM&quot;,&quot;o292731_2-2_20220818_Hu_nucseq_Graz_13_HH_GEM&quot;,&quot;o294781_01-1_20220912_Hu_nucseq_Graz_21_HH_GEM&quot;,&quot;o294781_02-2_20220912_Hu_nucseq_Graz_22_HH_GEM&quot;,&quot;o294781_03-3_20220912_Hu_nucseq_Graz_23_HH_GEM&quot;,&quot;o294781_04-4_20220912_Hu_nucseq_Graz_24_HH_GEM&quot;,
&quot;334131_10-10_20231108_Hu_nucseq_USZ_HTx001_RV&quot;,&quot;331571_3-5_20231012_Hu_nucseq_USZ_HTx001&quot;,
&quot;334131_01-1_20231103_Hu_nucseq_USZ_HTx002_RV&quot;,&quot;331571_4-6_20231012_Hu_nucseq_USZ_HTx002&quot;,
&quot;334131_04-4_20231106_Hu_nucseq_USZ_HTx003_RV&quot;,&quot;334131_02-2_20231103_Hu_nucseq_USZ_HTx003_LV&quot;,
&quot;334131_03-3_20231103_Hu_nucseq_USZ_HTx004_RV&quot;,&quot;334131_05-5_20231106_Hu_nucseq_USZ_HTx004_LV&quot;, 
&quot;334131_06-6_20231106_Hu_nucseq_USZ_HTx005_RV&quot;,&quot;334131_07-7_20231107_Hu_nucseq_USZ_HTx005_LV&quot;,
&quot;334131_08-8_20231107_Hu_nucseq_USZ_HTx006_RV&quot;,&quot;334131_09-9_20231108_Hu_nucseq_USZ_HTx006_LV&quot;,
&quot;340821_12-18_20240123_Hu_nucseq_USZ_HTx007_RV&quot;,&quot;340821_13-19_20240123_Hu_nucseq_USZ_HTx007_LV&quot;,
&quot;336431_07-7_20231129_Hu_nucseq_USZ_HTx008_RV&quot;,&quot;336431_09-9_20231129_Hu_nucseq_USZ_HTx008_LV&quot;,
&quot;347741_4-4_20240326_Hu_nucseq_USZ_HTx010_RV&quot;,&quot;347741_5-5_20240326_Hu_nucseq_USZ_HTx010_LV&quot;,
&quot;353921_01-1_20240515_Hu_nucseq_USZ_HTx011_RV&quot;,&quot;353921_02-2_20240515_Hu_nucseq_USZ_HTx011_LV&quot;,
&quot;353921_05-5_20240515_Hu_nucseq_USZ_HTx012_RV&quot;,&quot;353921_06-6_20240515_Hu_nucseq_USZ_HTx012_LV&quot;,
&quot;353921_09-9_20240515_Hu_nucseq_USZ_HTx013_RV&quot;,&quot;353921_10-10_20240515_Hu_nucseq_USZ_HTx013_LV&quot;,
&quot;353921_21-21_20240524_Hu_nucseq_USZ_HTx014_RV&quot;,&quot;353921_22-22_20240524_Hu_nucseq_USZ_HTx014_LV&quot;,
&quot;353921_13-13_20240524_Hu_nucseq_USZ_HTx015_RV&quot;, &quot;353921_17-17_20240524_Hu_nucseq_USZ_HTx015_LV&quot;,
&quot;359861_02-2_20240715_Hu_nucseq_USZ_HTx016_RV&quot;,&quot;359861_01-1_20240715_Hu_nucseq_USZ_HTx016_LV&quot;,
&quot;336431_08-8_20231129_Hu_nucseq_USZ_EMB001_V1&quot;,&quot;340831_1-1_20240118_Hu_nucseq_USZ_EMB001_V2&quot;,&quot;340821_03-9_20240123_Hu_nucseq_USZ_EMB001_V3&quot;,
&quot;336431_13-13_20231129_Hu_nucseq_USZ_EMB002_V1&quot;,&quot;340831_2-2_20240118_Hu_nucseq_USZ_EMB002_V2&quot;,&quot;340821_04-10_20240123_Hu_nucseq_USZ_EMB002_V3&quot;,
&quot;336431_14-14_20231129_Hu_nucseq_USZ_EMB003_V1&quot;,&quot;340831_3-3_20240118_Hu_nucseq_USZ_EMB003_V2&quot;,&quot;340821_05-11_20240123_Hu_nucseq_USZ_EMB003_V3&quot;,
&quot;336431_15-15_20231129_Hu_nucseq_USZ_EMB004_V1&quot;,&quot;340831_4-4_20240118_Hu_nucseq_USZ_EMB004_V2&quot;,&quot;340821_06-12_20240123_Hu_nucseq_USZ_EMB004_V3&quot;,
&quot;336431_10-10_20231129_Hu_nucseq_USZ_EMB005_V1&quot;,&quot;340831_5-5_20240118_Hu_nucseq_USZ_EMB005_V2&quot;,&quot;340821_07-13_20240123_Hu_nucseq_USZ_EMB005_V3&quot;,
&quot;336431_11-11_20231129_Hu_nucseq_USZ_EMB006_V1&quot;,&quot;340831_6-6_20240118_Hu_nucseq_USZ_EMB006_V2&quot;,&quot;340821_08-14_20240123_Hu_nucseq_USZ_EMB006_V3&quot;,
&quot;336431_12-12_20231129_Hu_nucseq_USZ_EMB007_V1&quot;,&quot;340821_01-7_20240118_Hu_nucseq_USZ_EMB007_V2&quot;,&quot;340821_09-15_20240123_Hu_nucseq_USZ_EMB007_V3&quot;,
&quot;340821_11-17_20240123_Hu_nucseq_USZ_EMB008_V1&quot;,&quot;340821_02-8_20240118_Hu_nucseq_USZ_EMB008_V2&quot;,&quot;340821_10-16_20240123_Hu_nucseq_USZ_EMB008_V3&quot;,
&quot;353921_11-11_20240515_Hu_nucseq_USZ_EMB010_V1_1&quot;,&quot;353921_12-12_20240515_Hu_nucseq_USZ_EMB010_V1_2&quot;,&quot;353921_08-8_20240515_Hu_nucseq_USZ_EMB010_V3&quot;, &quot;353921_03-3_20240515_Hu_nucseq_USZ_EMB011_V1&quot;,&quot;353921_04-4_20240515_Hu_nucseq_USZ_EMB011_V2&quot;,&quot;353921_14-14_20240524_Hu_nucseq_USZ_EMB011_V3&quot;, &quot;353921_07-7_20240515_Hu_nucseq_USZ_EMB012_V1&quot;,&quot;353921_15-15_20240524_Hu_nucseq_USZ_EMB012_V2&quot;,&quot;353921_16-16_20240524_Hu_nucseq_USZ_EMB012_V3&quot;, &quot;353921_18-18_20240524_Hu_nucseq_USZ_EMB013_V1&quot;,&quot;353921_19-19_20240524_Hu_nucseq_USZ_EMB013_V2&quot;,&quot;353921_20-20_20240524_Hu_nucseq_USZ_EMB013_V3&quot;,
&quot;353921_23-23_20240524_Hu_nucseq_USZ_EMB014_V1&quot;,&quot;353921_24-24_20240524_Hu_nucseq_USZ_EMB014_V2&quot;, &quot;359861_03-3_20240715_Hu_nucseq_USZ_EMB014_V3&quot;,
&quot;359861_04-4_20240715_Hu_nucseq_USZ_EMB015_V1&quot;,&quot;359861_05-5_20240715_Hu_nucseq_USZ_EMB015_V2&quot;,&quot;359861_06-6_20240715_Hu_nucseq_USZ_EMB015_V3&quot;,
&quot;359861_07-7_20240715_Hu_nucseq_USZ_EMB016_V1&quot;,&quot;359861_08-8_20240715_Hu_nucseq_USZ_EMB016_V2&quot;,&quot;359861_18-10_20240730_Hu_nucseq_USZ_EMB016_V3&quot;)

#### diseaseCondSp and diseaseCond
#healthy
healthy &lt;- c(&quot;o28576_1_08-8_20220525_Hu_nucseq_Graz_8_HH_GEM&quot;,&quot;o28576_1_10-10_20220525_Hu_nucseq_Graz_10_HH_GEM&quot;,&quot;o28576_1_11-11_20220525_Hu_nucseq_Graz_11_HH_GEM&quot;, &quot;o28576_1_12-12_20220525_Hu_nucseq_Graz_12_HH_GEM&quot;,&quot;o292731_1-1_20220818_Hu_nucseq_Graz_9_HH_GEM&quot;,&quot;o292731_2-2_20220818_Hu_nucseq_Graz_13_HH_GEM&quot;, &quot;o294781_01-1_20220912_Hu_nucseq_Graz_21_HH_GEM&quot;, &quot;o294781_02-2_20220912_Hu_nucseq_Graz_22_HH_GEM&quot;, &quot;o294781_03-3_20220912_Hu_nucseq_Graz_23_HH_GEM&quot;, &quot;o294781_04-4_20220912_Hu_nucseq_Graz_24_HH_GEM&quot;)
#Giant cell myocarditis
GCM &lt;- c(&quot;331571_3-5_20231012_Hu_nucseq_USZ_HTx001&quot;,&quot;334131_10-10_20231108_Hu_nucseq_USZ_HTx001_RV&quot;)
#Dilated Cardiomyopathy
DCM &lt;- c(&quot;334131_02-2_20231103_Hu_nucseq_USZ_HTx003_LV&quot;,&quot;334131_04-4_20231106_Hu_nucseq_USZ_HTx003_RV&quot;, &quot;334131_05-5_20231106_Hu_nucseq_USZ_HTx004_LV&quot;,&quot;334131_03-3_20231103_Hu_nucseq_USZ_HTx004_RV&quot;, &quot;334131_07-7_20231107_Hu_nucseq_USZ_HTx005_LV&quot;,&quot;334131_06-6_20231106_Hu_nucseq_USZ_HTx005_RV&quot;, &quot;334131_09-9_20231108_Hu_nucseq_USZ_HTx006_LV&quot;,&quot;334131_08-8_20231107_Hu_nucseq_USZ_HTx006_RV&quot;,
&quot;340821_13-19_20240123_Hu_nucseq_USZ_HTx007_LV&quot;,&quot;340821_12-18_20240123_Hu_nucseq_USZ_HTx007_RV&quot;,
&quot;347741_4-4_20240326_Hu_nucseq_USZ_HTx010_RV&quot;,&quot;347741_5-5_20240326_Hu_nucseq_USZ_HTx010_LV&quot;,
&quot;353921_01-1_20240515_Hu_nucseq_USZ_HTx011_RV&quot;,&quot;353921_02-2_20240515_Hu_nucseq_USZ_HTx011_LV&quot;,
&quot;353921_21-21_20240524_Hu_nucseq_USZ_HTx014_RV&quot;,&quot;353921_22-22_20240524_Hu_nucseq_USZ_HTx014_LV&quot;,
&quot;359861_02-2_20240715_Hu_nucseq_USZ_HTx016_RV&quot;,&quot;359861_01-1_20240715_Hu_nucseq_USZ_HTx016_LV&quot;)
#LoefflerEndocarditis
LoeEndoc &lt;- c(&quot;331571_4-6_20231012_Hu_nucseq_USZ_HTx002&quot;,&quot;334131_01-1_20231103_Hu_nucseq_USZ_HTx002_RV&quot;)
#hypertrophic cardiomyopathy (non-obstructive)
HCM &lt;- c(&quot;336431_07-7_20231129_Hu_nucseq_USZ_HTx008_RV&quot;,&quot;336431_09-9_20231129_Hu_nucseq_USZ_HTx008_LV&quot;)
#Ischemic cardiomyopathy
ICM &lt;- c(&quot;353921_05-5_20240515_Hu_nucseq_USZ_HTx012_RV&quot;,&quot;353921_06-6_20240515_Hu_nucseq_USZ_HTx012_LV&quot;,&quot;353921_09-9_20240515_Hu_nucseq_USZ_HTx013_RV&quot;,&quot;353921_10-10_20240515_Hu_nucseq_USZ_HTx013_LV&quot;,&quot;353921_13-13_20240524_Hu_nucseq_USZ_HTx015_RV&quot;, &quot;353921_17-17_20240524_Hu_nucseq_USZ_HTx015_LV&quot;)
#EMBs of transplanted hearts
visit1 &lt;- c(&quot;336431_08-8_20231129_Hu_nucseq_USZ_EMB001_V1&quot;,&quot;336431_13-13_20231129_Hu_nucseq_USZ_EMB002_V1&quot;,&quot;336431_14-14_20231129_Hu_nucseq_USZ_EMB003_V1&quot;,&quot;336431_15-15_20231129_Hu_nucseq_USZ_EMB004_V1&quot;,&quot;336431_10-10_20231129_Hu_nucseq_USZ_EMB005_V1&quot;,&quot;336431_11-11_20231129_Hu_nucseq_USZ_EMB006_V1&quot;, &quot;336431_12-12_20231129_Hu_nucseq_USZ_EMB007_V1&quot;,&quot;340821_11-17_20240123_Hu_nucseq_USZ_EMB008_V1&quot;,&quot;353921_11-11_20240515_Hu_nucseq_USZ_EMB010_V1_1&quot;,&quot;353921_03-3_20240515_Hu_nucseq_USZ_EMB011_V1&quot;,&quot;353921_07-7_20240515_Hu_nucseq_USZ_EMB012_V1&quot;,&quot;353921_18-18_20240524_Hu_nucseq_USZ_EMB013_V1&quot;,&quot;353921_23-23_20240524_Hu_nucseq_USZ_EMB014_V1&quot;,&quot;359861_04-4_20240715_Hu_nucseq_USZ_EMB015_V1&quot;,&quot;359861_07-7_20240715_Hu_nucseq_USZ_EMB016_V1&quot;)
visit2 &lt;- c(&quot;340831_1-1_20240118_Hu_nucseq_USZ_EMB001_V2&quot;,&quot;340831_2-2_20240118_Hu_nucseq_USZ_EMB002_V2&quot;,&quot;340831_3-3_20240118_Hu_nucseq_USZ_EMB003_V2&quot;,&quot;340831_4-4_20240118_Hu_nucseq_USZ_EMB004_V2&quot;,&quot;340831_5-5_20240118_Hu_nucseq_USZ_EMB005_V2&quot;,&quot;340831_6-6_20240118_Hu_nucseq_USZ_EMB006_V2&quot;,&quot;340821_01-7_20240118_Hu_nucseq_USZ_EMB007_V2&quot;,&quot;340821_02-8_20240118_Hu_nucseq_USZ_EMB008_V2&quot;,&quot;353921_12-12_20240515_Hu_nucseq_USZ_EMB010_V1_2&quot;,&quot;353921_04-4_20240515_Hu_nucseq_USZ_EMB011_V2&quot;,&quot;353921_15-15_20240524_Hu_nucseq_USZ_EMB012_V2&quot;,&quot;353921_19-19_20240524_Hu_nucseq_USZ_EMB013_V2&quot;,&quot;353921_24-24_20240524_Hu_nucseq_USZ_EMB014_V2&quot;,&quot;359861_05-5_20240715_Hu_nucseq_USZ_EMB015_V2&quot;,&quot;359861_08-8_20240715_Hu_nucseq_USZ_EMB016_V2&quot;)
visit3 &lt;- c(&quot;340821_03-9_20240123_Hu_nucseq_USZ_EMB001_V3&quot;,&quot;340821_04-10_20240123_Hu_nucseq_USZ_EMB002_V3&quot;,&quot;340821_05-11_20240123_Hu_nucseq_USZ_EMB003_V3&quot;,&quot;340821_06-12_20240123_Hu_nucseq_USZ_EMB004_V3&quot;,&quot;340821_07-13_20240123_Hu_nucseq_USZ_EMB005_V3&quot;,&quot;340821_08-14_20240123_Hu_nucseq_USZ_EMB006_V3&quot;,&quot;340821_09-15_20240123_Hu_nucseq_USZ_EMB007_V3&quot;,&quot;340821_10-16_20240123_Hu_nucseq_USZ_EMB008_V3&quot;,&quot;353921_08-8_20240515_Hu_nucseq_USZ_EMB010_V3&quot;,&quot;353921_14-14_20240524_Hu_nucseq_USZ_EMB011_V3&quot;,&quot;353921_16-16_20240524_Hu_nucseq_USZ_EMB012_V3&quot;,&quot;353921_20-20_20240524_Hu_nucseq_USZ_EMB013_V3&quot;,&quot;359861_03-3_20240715_Hu_nucseq_USZ_EMB014_V3&quot;,&quot;359861_06-6_20240715_Hu_nucseq_USZ_EMB015_V3&quot;,&quot;359861_18-10_20240730_Hu_nucseq_USZ_EMB016_V3&quot;)

seuratM$diseaseCondSp &lt;- &quot;diseaseCondSp&quot;
seuratM$diseaseCondSp[which(seuratM$dataset %in% GCM )] &lt;- &quot;GCM&quot;
seuratM$diseaseCondSp[which(seuratM$dataset %in% DCM)] &lt;- &quot;DCM&quot;
seuratM$diseaseCondSp[which(seuratM$dataset %in% HCM)] &lt;- &quot;HCM&quot;
seuratM$diseaseCondSp[which(seuratM$dataset %in% ICM)] &lt;- &quot;ICM&quot;
seuratM$diseaseCondSp[which(seuratM$dataset %in% LoeEndoc)] &lt;- &quot;LoeEndoc&quot;
seuratM$diseaseCondSp[which(seuratM$dataset %in% healthy)] &lt;- &quot;healthy&quot;
seuratM$diseaseCondSp[which(seuratM$dataset %in% visit1)] &lt;- &quot;visit1&quot;
seuratM$diseaseCondSp[which(seuratM$dataset %in% visit2)] &lt;- &quot;visit2&quot;
seuratM$diseaseCondSp[which(seuratM$dataset %in% visit3)] &lt;- &quot;visit3&quot;
table(seuratM$diseaseCondSp)</code></pre>
<pre><code>
     DCM      GCM      HCM  healthy      ICM LoeEndoc   visit1   visit2   visit3 
   54458     4835     6761    36012    10640     5777    10952    10569     8643 </code></pre>
<pre class="r"><code>seuratM$diseaseCond &lt;- &quot;diseaseCond&quot;
seuratM$diseaseCond[which(seuratM$dataset %in% c(GCM, DCM, HCM,ICM,LoeEndoc))] &lt;- &quot;explant&quot;
seuratM$diseaseCond[which(seuratM$dataset %in% visit1)] &lt;- &quot;visit1&quot;
seuratM$diseaseCond[which(seuratM$dataset %in% visit2)] &lt;- &quot;visit2&quot;
seuratM$diseaseCond[which(seuratM$dataset %in% visit3)] &lt;- &quot;visit3&quot;
seuratM$diseaseCond[which(seuratM$dataset %in% healthy)] &lt;- &quot;healthy&quot;
table(seuratM$diseaseCond)</code></pre>
<pre><code>
explant healthy  visit1  visit2  visit3 
  82471   36012   10952   10569    8643 </code></pre>
<pre class="r"><code>orddiseaseCond &lt;- c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot; ,&quot;visit3&quot;)

#### cluster_name
seuratM$clusterName &lt;- &quot;clusterName&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;0&quot; )] &lt;- &quot;Fb&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;1&quot; )] &lt;- &quot;BEC1&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;2&quot; )] &lt;- &quot;PerivFb&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;3&quot; )] &lt;- &quot;Mph&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;4&quot; )] &lt;- &quot;CM&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;5&quot; )] &lt;- &quot;Tcell&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;6&quot; )] &lt;- &quot;VSMC&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;7&quot; )] &lt;- &quot;Mph2&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;8&quot; )] &lt;- &quot;BEC2&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;9&quot; )] &lt;- &quot;Int&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;10&quot; )] &lt;- &quot;NC&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;11&quot; )] &lt;- &quot;Tcell2&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;12&quot; )] &lt;- &quot;LEC&quot;
seuratM$clusterName[which(seuratM$RNA_snn_res.0.25 %in% &quot;13&quot; )] &lt;- &quot;AdipoC&quot;
table(seuratM$clusterName)</code></pre>
<pre><code>
 AdipoC    BEC1    BEC2      CM      Fb     Int     LEC     Mph    Mph2      NC PerivFb   Tcell 
    688   24558    2458   12792   50676    2167     884   16898    2478    1844   22350    6349 
 Tcell2    VSMC 
   1087    3418 </code></pre>
<pre class="r"><code>table(seuratM$RNA_snn_res.0.25)</code></pre>
<pre><code>
    0     1     2     3     4     5     6     7     8     9    10    11    12    13 
50676 24558 22350 16898 12792  6349  3418  2478  2458  2167  1844  1087   884   688 </code></pre>
<pre class="r"><code>###order
Idents(seuratM) &lt;- seuratM$clusterName
seuratM$clusterName &lt;- factor(seuratM$clusterName, levels=c(&quot;CM&quot;,&quot;Fb&quot;,&quot;PerivFb&quot;,&quot;VSMC&quot;,&quot;Int&quot;,&quot;BEC1&quot;, &quot;BEC2&quot;,&quot;LEC&quot;,&quot;NC&quot;,&quot;AdipoC&quot;,&quot;Mph&quot;,&quot;Mph2&quot;,&quot;Tcell&quot;,&quot;Tcell2&quot;))
Idents(seuratM) &lt;- seuratM$clusterName
table(seuratM$clusterName)</code></pre>
<pre><code>
     CM      Fb PerivFb    VSMC     Int    BEC1    BEC2     LEC      NC  AdipoC     Mph    Mph2 
  12792   50676   22350    3418    2167   24558    2458     884    1844     688   16898    2478 
  Tcell  Tcell2 
   6349    1087 </code></pre>
<pre class="r"><code>#### ventricle
LV &lt;- c(&quot;331571_3-5_20231012_Hu_nucseq_USZ_HTx001&quot;,&quot;331571_4-6_20231012_Hu_nucseq_USZ_HTx002&quot;,&quot;334131_02-2_20231103_Hu_nucseq_USZ_HTx003_LV&quot;,&quot;334131_05-5_20231106_Hu_nucseq_USZ_HTx004_LV&quot;,&quot;334131_07-7_20231107_Hu_nucseq_USZ_HTx005_LV&quot;,&quot;334131_09-9_20231108_Hu_nucseq_USZ_HTx006_LV&quot;,&quot;340821_13-19_20240123_Hu_nucseq_USZ_HTx007_LV&quot;,&quot;336431_09-9_20231129_Hu_nucseq_USZ_HTx008_LV&quot;,&quot;347741_1-1_20240326_Hu_nucseq_USZ_HTx009_RV&quot;,&quot;347741_4-4_20240326_Hu_nucseq_USZ_HTx010_RV&quot;,&quot;353921_01-1_20240515_Hu_nucseq_USZ_HTx011_RV&quot;,&quot;353921_05-5_20240515_Hu_nucseq_USZ_HTx012_RV&quot;,&quot;353921_09-9_20240515_Hu_nucseq_USZ_HTx013_RV&quot;,&quot;353921_21-21_20240524_Hu_nucseq_USZ_HTx014_RV&quot;,&quot;353921_13-13_20240524_Hu_nucseq_USZ_HTx015_RV&quot;,&quot;359861_02-2_20240715_Hu_nucseq_USZ_HTx016_RV&quot;)
RV &lt;- c(&quot;334131_10-10_20231108_Hu_nucseq_USZ_HTx001_RV&quot;,&quot;334131_01-1_20231103_Hu_nucseq_USZ_HTx002_RV&quot;,&quot;334131_04-4_20231106_Hu_nucseq_USZ_HTx003_RV&quot;,&quot;334131_03-3_20231103_Hu_nucseq_USZ_HTx004_RV&quot;,&quot;334131_06-6_20231106_Hu_nucseq_USZ_HTx005_RV&quot;,&quot;334131_08-8_20231107_Hu_nucseq_USZ_HTx006_RV&quot;,&quot;340821_12-18_20240123_Hu_nucseq_USZ_HTx007_RV&quot;,&quot;336431_07-7_20231129_Hu_nucseq_USZ_HTx008_RV&quot;,&quot;347741_2-2_20240326_Hu_nucseq_USZ_HTx009_LV&quot;,&quot;347741_5-5_20240326_Hu_nucseq_USZ_HTx010_LV&quot;,&quot;353921_02-2_20240515_Hu_nucseq_USZ_HTx011_LV&quot;,&quot;353921_06-6_20240515_Hu_nucseq_USZ_HTx012_LV&quot;,&quot;353921_10-10_20240515_Hu_nucseq_USZ_HTx013_LV&quot;,&quot;353921_22-22_20240524_Hu_nucseq_USZ_HTx014_LV&quot;,&quot;353921_17-17_20240524_Hu_nucseq_USZ_HTx015_LV&quot;,&quot;359861_01-1_20240715_Hu_nucseq_USZ_HTx016_LV&quot;)

seuratM$ventricle &lt;- &quot;ventricle-NA&quot;
seuratM$ventricle[which(seuratM$dataset %in% LV)] &lt;- &quot;LV&quot;
seuratM$ventricle[which(seuratM$dataset %in% RV)] &lt;- &quot;RV&quot;
table(seuratM$ventricle)</code></pre>
<pre><code>
          LV           RV ventricle-NA 
       41896        40575        66176 </code></pre>
<pre class="r"><code>###combined slots
seuratM$patient_diseaseCond &lt;- paste0(seuratM$patient, &#39;_&#39;, seuratM$diseaseCond)
table(seuratM$patient_diseaseCond)</code></pre>
<pre><code>
 CarTransPat1_explant   CarTransPat1_visit1   CarTransPat1_visit2   CarTransPat1_visit3 
                 4835                   483                   804                    68 
CarTransPat10_explant  CarTransPat10_visit1  CarTransPat10_visit2  CarTransPat10_visit3 
                 5038                   996                   265                   677 
CarTransPat11_explant  CarTransPat11_visit1  CarTransPat11_visit2  CarTransPat11_visit3 
                 3197                   738                   193                   623 
CarTransPat12_explant  CarTransPat12_visit1  CarTransPat12_visit2  CarTransPat12_visit3 
                 3767                   715                  1547                  1846 
CarTransPat13_explant  CarTransPat13_visit1  CarTransPat13_visit2  CarTransPat13_visit3 
                 3680                   379                   138                   327 
CarTransPat14_explant  CarTransPat14_visit1  CarTransPat14_visit2  CarTransPat14_visit3 
                 4685                   866                   311                   395 
CarTransPat15_explant  CarTransPat15_visit1  CarTransPat15_visit2  CarTransPat15_visit3 
                 3193                   134                   207                   449 
CarTransPat16_explant  CarTransPat16_visit1  CarTransPat16_visit2  CarTransPat16_visit3 
                12312                   547                    94                   287 
 CarTransPat2_explant   CarTransPat2_visit1   CarTransPat2_visit2   CarTransPat2_visit3 
                 5777                  1338                  2575                   349 
 CarTransPat3_explant   CarTransPat3_visit1   CarTransPat3_visit2   CarTransPat3_visit3 
                 7282                  1362                   323                   196 
 CarTransPat4_explant   CarTransPat4_visit1   CarTransPat4_visit2   CarTransPat4_visit3 
                 8238                   986                   521                   745 
 CarTransPat5_explant   CarTransPat5_visit1   CarTransPat5_visit2   CarTransPat5_visit3 
                 4433                   606                   654                  1896 
 CarTransPat6_explant   CarTransPat6_visit1   CarTransPat6_visit2   CarTransPat6_visit3 
                 4773                   420                   770                   230 
 CarTransPat7_explant   CarTransPat7_visit1   CarTransPat7_visit2   CarTransPat7_visit3 
                 4500                  1326                  1600                   361 
 CarTransPat8_explant   CarTransPat8_visit1   CarTransPat8_visit2   CarTransPat8_visit3 
                 6761                    56                   567                   194 
          HH1_healthy          HH10_healthy           HH2_healthy           HH3_healthy 
                 3921                   841                  1480                  3731 
          HH4_healthy           HH5_healthy           HH6_healthy           HH7_healthy 
                 3991                  3818                  4908                  9882 
          HH8_healthy           HH9_healthy 
                 1442                  1998 </code></pre>
<pre class="r"><code>ordpatient_diseasecond &lt;- c(&quot;HH1_healthy&quot;,&quot;HH10_healthy&quot;,&quot;HH2_healthy&quot;,&quot;HH3_healthy&quot;,&quot;HH4_healthy&quot;,&quot;HH5_healthy&quot;,&quot;HH6_healthy&quot;,&quot;HH7_healthy&quot;,&quot;HH8_healthy&quot;,&quot;HH9_healthy&quot;,&quot;CarTransPat1_explant&quot;,&quot;CarTransPat2_explant&quot;,&quot;CarTransPat3_explant&quot;,&quot;CarTransPat4_explant&quot;,&quot;CarTransPat5_explant&quot;,&quot;CarTransPat6_explant&quot;,&quot;CarTransPat7_explant&quot;,&quot;CarTransPat8_explant&quot;,&quot;CarTransPat10_explant&quot;,&quot;CarTransPat11_explant&quot;,&quot;CarTransPat12_explant&quot;,&quot;CarTransPat13_explant&quot;,&quot;CarTransPat14_explant&quot;,&quot;CarTransPat15_explant&quot;, &quot;CarTransPat16_explant&quot;,
&quot;CarTransPat1_visit1&quot;,&quot;CarTransPat1_visit2&quot;,&quot;CarTransPat1_visit3&quot;,&quot;CarTransPat2_visit1&quot;,&quot;CarTransPat2_visit2&quot;,&quot;CarTransPat2_visit3&quot;,&quot;CarTransPat3_visit1&quot;,&quot;CarTransPat3_visit2&quot;,&quot;CarTransPat3_visit3&quot;,&quot;CarTransPat4_visit1&quot;,&quot;CarTransPat4_visit2&quot;,&quot;CarTransPat4_visit3&quot;,&quot;CarTransPat5_visit1&quot;,&quot;CarTransPat5_visit2&quot;,&quot;CarTransPat5_visit3&quot;,&quot;CarTransPat6_visit1&quot;,&quot;CarTransPat6_visit2&quot;,&quot;CarTransPat6_visit3&quot;,&quot;CarTransPat7_visit1&quot;,&quot;CarTransPat7_visit2&quot;,&quot;CarTransPat7_visit3&quot;,&quot;CarTransPat8_visit1&quot;,&quot;CarTransPat8_visit2&quot;,&quot;CarTransPat8_visit3&quot;,&quot;CarTransPat10_visit1&quot;,&quot;CarTransPat10_visit2&quot;,&quot;CarTransPat10_visit3&quot;,&quot;CarTransPat11_visit1&quot;,&quot;CarTransPat11_visit2&quot;,&quot;CarTransPat11_visit3&quot;,&quot;CarTransPat12_visit1&quot;,&quot;CarTransPat12_visit2&quot;,&quot;CarTransPat12_visit3&quot;,&quot;CarTransPat13_visit1&quot;,&quot;CarTransPat13_visit2&quot;,&quot;CarTransPat13_visit3&quot;,&quot;CarTransPat14_visit1&quot;,&quot;CarTransPat14_visit2&quot;, &quot;CarTransPat14_visit3&quot;,&quot;CarTransPat15_visit1&quot;,&quot;CarTransPat15_visit2&quot;, &quot;CarTransPat15_visit3&quot;,&quot;CarTransPat16_visit1&quot;,&quot;CarTransPat16_visit2&quot;, &quot;CarTransPat16_visit3&quot;)

seuratM$patient_ventricle &lt;- paste0(seuratM$patient, &#39;_&#39;, seuratM$ventricle)
table(seuratM$patient_ventricle)</code></pre>
<pre><code>
           CarTransPat1_LV            CarTransPat1_RV  CarTransPat1_ventricle-NA 
                      2934                       1901                       1355 
          CarTransPat10_LV           CarTransPat10_RV CarTransPat10_ventricle-NA 
                      2034                       3004                       1938 
          CarTransPat11_LV           CarTransPat11_RV CarTransPat11_ventricle-NA 
                      1846                       1351                       1554 
          CarTransPat12_LV           CarTransPat12_RV CarTransPat12_ventricle-NA 
                      1101                       2666                       4108 
          CarTransPat13_LV           CarTransPat13_RV CarTransPat13_ventricle-NA 
                      1715                       1965                        844 
          CarTransPat14_LV           CarTransPat14_RV CarTransPat14_ventricle-NA 
                      2348                       2337                       1572 
          CarTransPat15_LV           CarTransPat15_RV CarTransPat15_ventricle-NA 
                      1745                       1448                        790 
          CarTransPat16_LV           CarTransPat16_RV CarTransPat16_ventricle-NA 
                      7278                       5034                        928 
           CarTransPat2_LV            CarTransPat2_RV  CarTransPat2_ventricle-NA 
                      2059                       3718                       4262 
           CarTransPat3_LV            CarTransPat3_RV  CarTransPat3_ventricle-NA 
                      3905                       3377                       1881 
           CarTransPat4_LV            CarTransPat4_RV  CarTransPat4_ventricle-NA 
                      4861                       3377                       2252 
           CarTransPat5_LV            CarTransPat5_RV  CarTransPat5_ventricle-NA 
                      1739                       2694                       3156 
           CarTransPat6_LV            CarTransPat6_RV  CarTransPat6_ventricle-NA 
                      3101                       1672                       1420 
           CarTransPat7_LV            CarTransPat7_RV  CarTransPat7_ventricle-NA 
                      2148                       2352                       3287 
           CarTransPat8_LV            CarTransPat8_RV  CarTransPat8_ventricle-NA 
                      3082                       3679                        817 
          HH1_ventricle-NA          HH10_ventricle-NA           HH2_ventricle-NA 
                      3921                        841                       1480 
          HH3_ventricle-NA           HH4_ventricle-NA           HH5_ventricle-NA 
                      3731                       3991                       3818 
          HH6_ventricle-NA           HH7_ventricle-NA           HH8_ventricle-NA 
                      4908                       9882                       1442 
          HH9_ventricle-NA 
                      1998 </code></pre>
<pre class="r"><code>###combined slots
seuratM$clusterName_diseaseCond &lt;- paste0(seuratM$clusterName, &#39;_&#39;, seuratM$diseaseCond)
table(seuratM$clusterName_diseaseCond)</code></pre>
<pre><code>
 AdipoC_explant  AdipoC_healthy   AdipoC_visit1   AdipoC_visit2   AdipoC_visit3    BEC1_explant 
            580              76               5              20               7           14584 
   BEC1_healthy     BEC1_visit1     BEC1_visit2     BEC1_visit3    BEC2_explant    BEC2_healthy 
           6644            1053            1309             968             821             686 
    BEC2_visit1     BEC2_visit2     BEC2_visit3      CM_explant      CM_healthy       CM_visit1 
            481             218             252            3937            5519            1509 
      CM_visit2       CM_visit3      Fb_explant      Fb_healthy       Fb_visit1       Fb_visit2 
            702            1125           32045            8426            3486            3530 
      Fb_visit3     Int_explant     Int_healthy      Int_visit1      Int_visit2      Int_visit3 
           3189            1444             469              77              84              93 
    LEC_explant     LEC_healthy      LEC_visit1      LEC_visit2      LEC_visit3     Mph_explant 
            740              88               3              36              17            8163 
    Mph_healthy      Mph_visit1      Mph_visit2      Mph_visit3    Mph2_explant    Mph2_healthy 
           3819            1551            2185            1180            1582             388 
    Mph2_visit1     Mph2_visit2     Mph2_visit3      NC_explant      NC_healthy       NC_visit1 
            156             189             163            1047             449             119 
      NC_visit2       NC_visit3 PerivFb_explant PerivFb_healthy  PerivFb_visit1  PerivFb_visit2 
            104             125           11528            8015             961             955 
 PerivFb_visit3   Tcell_explant   Tcell_healthy    Tcell_visit1    Tcell_visit2    Tcell_visit3 
            891            3147             513            1325             962             402 
 Tcell2_explant  Tcell2_healthy   Tcell2_visit1   Tcell2_visit2   Tcell2_visit3    VSMC_explant 
            737              90              98             110              52            2116 
   VSMC_healthy     VSMC_visit1     VSMC_visit2     VSMC_visit3 
            830             128             165             179 </code></pre>
<p>##set color vectors</p>
<pre class="r"><code>colclusterName &lt;- c(&quot;#67001f&quot;, &quot;#D53E4F&quot;, &quot;#f4a582&quot;, &quot;#FEE08B&quot;, &quot;#003c30&quot;,&quot;#01665e&quot;,&quot;#66C2A5&quot;, &quot;#3288BD&quot;,&quot;#BEAED4&quot;, &quot;#c7eae5&quot;,&quot;#355C7D&quot;,&quot;#202547&quot;,&quot;#B45B5C&quot;,&quot;#8c510a&quot;)
names(colclusterName) &lt;- c(&quot;CM&quot;,&quot;Fb&quot;,&quot;PerivFb&quot;,&quot;VSMC&quot;,&quot;Int&quot;,&quot;BEC1&quot;, &quot;BEC2&quot;,&quot;LEC&quot;,&quot;NC&quot;,&quot;AdipoC&quot;,&quot;Mph&quot;,&quot;Mph2&quot;,&quot;Tcell&quot;,&quot;Tcell2&quot;)

coldiseaseCond &lt;- c(&quot;#dfc27d&quot;,&quot;#BE3144&quot;,&quot;#202547&quot;,&quot;#355C7D&quot;,&quot;#779d8d&quot;)
names(coldiseaseCond) &lt;- c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;)</code></pre>
</div>
<div id="qc-merged" class="section level2">
<h2>QC merged</h2>
<pre class="r"><code># Extract meta.data from the Seurat object
meta.data &lt;- seuratM@meta.data
# Create the density plot
ptotalpat &lt;- ggplot(data = meta.data, aes(x = total, color = patient, fill = patient)) +
  geom_density(alpha = 0.2) +
  #scale_fill_manual(values = colpat) +
  #scale_color_manual(values = colpat) +
  theme_classic() +
  scale_x_log10() +
  ylab(&quot;density&quot;) +
  geom_vline(xintercept = 100)

pdetectedpat &lt;- ggplot(data = meta.data, aes(x = detected, color = patient, fill = patient)) +
  geom_density(alpha = 0.2) +
 #scale_fill_manual(values = colpat) +
  #scale_color_manual(values = colpat) +
  theme_classic() +
  scale_x_log10() +
  ylab(&quot;density&quot;) +
  geom_vline(xintercept = 100)

# Return the plots as a list
list(ptotalpat, pdetectedpat)</code></pre>
<pre><code>[[1]]</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/QC%20patient-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-QC-patient-1">
Past versions of "QC patient-1.png"
</button>
</p>
<div id="fig-no-spaces-QC-patient-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/QC patient-1.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[2]]</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/QC%20patient-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-QC-patient-2">
Past versions of "QC patient-2.png"
</button>
</p>
<div id="fig-no-spaces-QC-patient-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/QC patient-2.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Extract meta.data from the Seurat object
meta.data &lt;- seuratM@meta.data
# Create the density plot
ptotalpat &lt;- ggplot(data = meta.data, aes(x = total, color = dataset, fill = dataset)) +
  geom_density(alpha = 0.2) +
  #scale_fill_manual(values = colpat) +
  #scale_color_manual(values = colpat) +
  theme_classic() +
  scale_x_log10() +
  ylab(&quot;density&quot;) +
  geom_vline(xintercept = 100)

pdetectedpat &lt;- ggplot(data = meta.data, aes(x = detected, color = dataset, fill = dataset)) +
  geom_density(alpha = 0.2) +
 #scale_fill_manual(values = colpat) +
  #scale_color_manual(values = colpat) +
  theme_classic() +
  scale_x_log10() +
  ylab(&quot;density&quot;) +
  geom_vline(xintercept = 100)

# Return the plots as a list
list(ptotalpat, pdetectedpat)</code></pre>
<pre><code>[[1]]</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/QC%20dataset-1.png" width="3840" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-QC-dataset-1">
Past versions of "QC dataset-1.png"
</button>
</p>
<div id="fig-no-spaces-QC-dataset-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/QC dataset-1.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre><code>
[[2]]</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/QC%20dataset-2.png" width="3840" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-QC-dataset-2">
Past versions of "QC dataset-2.png"
</button>
</p>
<div id="fig-no-spaces-QC-dataset-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/QC dataset-2.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="plot-cell-numbers" class="section level2">
<h2>plot cell numbers</h2>
<pre class="r"><code>cell_count &lt;- data.frame(table(seuratM$patient))
colnames(cell_count) &lt;- c(&quot;patient&quot;, &quot;Freq&quot;)
hsize &lt;- 1.5

ggplot(cell_count, aes(x = hsize, y = Freq, fill = patient)) +
  #scale_fill_manual(values = colpat2) +
  geom_col(color = &quot;white&quot;) +
  coord_polar(theta = &quot;y&quot;) +
  xlim(c(0.2, hsize + 0.5)) +
  theme_void() +
  ggtitle(&quot;cell number&quot;) +
  theme(plot.title = element_text(hjust = 0.5, size = 15)) +
  geom_text(aes(label = Freq), position = position_stack(vjust = 0.5))</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/cellnumber%20circle%20patient-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cellnumber-circle-patient-1">
Past versions of "cellnumber circle patient-1.png"
</button>
</p>
<div id="fig-no-spaces-cellnumber-circle-patient-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/cellnumber circle patient-1.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>cell_count &lt;- data.frame(table(seuratM$dataset))
colnames(cell_count) &lt;- c(&quot;dataset&quot;, &quot;Freq&quot;)
hsize &lt;- 1.5

ggplot(cell_count, aes(x = hsize, y = Freq, fill = dataset)) +
  #scale_fill_manual(values = colpat2) +
  geom_col(color = &quot;white&quot;) +
  coord_polar(theta = &quot;y&quot;) +
  xlim(c(0.2, hsize + 0.5)) +
  theme_void() +
  ggtitle(&quot;cell number&quot;) +
  theme(plot.title = element_text(hjust = 0.5, size = 15)) +
  geom_text(aes(label = Freq), position = position_stack(vjust = 0.5))</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/cellnumber%20circle%20dataset-1.png" width="3840" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cellnumber-circle-dataset-1">
Past versions of "cellnumber circle dataset-1.png"
</button>
</p>
<div id="fig-no-spaces-cellnumber-circle-dataset-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/cellnumber circle dataset-1.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="plot-umaps" class="section level2">
<h2>plot umaps</h2>
<pre class="r"><code>Idents(seuratM) &lt;- seuratM$RNA_snn_res.0.25
DimPlot(seuratM, reduction = &quot;umap&quot;, pt.size = 0.1, raster = FALSE) </code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/umap-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umap-1">
Past versions of umap-1.png
</button>
</p>
<div id="fig-umap-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/umap-1.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/57d61d1fcc11a9b583f80064614551badcdb1f67/docs/figure/HumanHeartCarTrans.Rmd/umap-1.png" target="_blank">57d61d1</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/3b7da724c758b0362d6b5f4b162eed76f8e62a3d/docs/figure/HumanHeartCarTrans.Rmd/umap-1.png" target="_blank">3b7da72</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-02-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Idents(seuratM) &lt;- seuratM$patient
DimPlot(seuratM, reduction = &quot;umap&quot;, pt.size = 0.1, raster = FALSE) </code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/umap-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umap-2">
Past versions of umap-2.png
</button>
</p>
<div id="fig-umap-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/umap-2.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/3b7da724c758b0362d6b5f4b162eed76f8e62a3d/docs/figure/HumanHeartCarTrans.Rmd/umap-2.png" target="_blank">3b7da72</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-02-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(seuratM, reduction = &quot;umap&quot;, pt.size = 0.1, raster = FALSE) + theme(legend.position = &quot;null&quot;)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/umap-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umap-3">
Past versions of umap-3.png
</button>
</p>
<div id="fig-umap-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/umap-3.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/57d61d1fcc11a9b583f80064614551badcdb1f67/docs/figure/HumanHeartCarTrans.Rmd/umap-3.png" target="_blank">57d61d1</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-02-29
</td>
</tr>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/3b7da724c758b0362d6b5f4b162eed76f8e62a3d/docs/figure/HumanHeartCarTrans.Rmd/umap-3.png" target="_blank">3b7da72</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-02-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Idents(seuratM) &lt;- seuratM$clusterName
DimPlot(seuratM, reduction = &quot;umap&quot;, pt.size = 0.1, cols = colclusterName, raster = FALSE) </code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/umap-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umap-4">
Past versions of umap-4.png
</button>
</p>
<div id="fig-umap-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/umap-4.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/3b7da724c758b0362d6b5f4b162eed76f8e62a3d/docs/figure/HumanHeartCarTrans.Rmd/umap-4.png" target="_blank">3b7da72</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-02-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(seuratM, reduction = &quot;umap&quot;, pt.size = 0.1, cols = colclusterName, raster = FALSE) + theme(legend.position = &quot;null&quot;)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/umap-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umap-5">
Past versions of umap-5.png
</button>
</p>
<div id="fig-umap-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/umap-5.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(seuratM, reduction = &quot;umap&quot;, pt.size = 0.1, cols = colclusterName, raster = FALSE, label = TRUE) + theme(legend.position = &quot;null&quot;)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/umap-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umap-6">
Past versions of umap-6.png
</button>
</p>
<div id="fig-umap-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/umap-6.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Idents(seuratM) &lt;- seuratM$diseaseCond
order1 &lt;- c(&quot;visit3&quot;,&quot;visit2&quot;,&quot;visit1&quot;,&quot;explant&quot;,&quot;healthy&quot;)
DimPlot(seuratM, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, order = order1, raster=FALSE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/umap-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umap-7">
Past versions of umap-7.png
</button>
</p>
<div id="fig-umap-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/umap-7.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(seuratM, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, order = order1, raster = FALSE) + theme(legend.position = &quot;null&quot;)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/umap-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-umap-8">
Past versions of umap-8.png
</button>
</p>
<div id="fig-umap-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/umap-8.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>seuratHealthy &lt;- subset(seuratM, diseaseCond == &quot;healthy&quot;)
DimPlot(seuratHealthy, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, raster=FALSE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/subset%20umaps-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-umaps-1">
Past versions of "subset umaps-1.png"
</button>
</p>
<div id="fig-no-spaces-subset-umaps-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/subset umaps-1.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(seuratHealthy, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, raster=FALSE) + theme(legend.position = &quot;null&quot;)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/subset%20umaps-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-umaps-2">
Past versions of "subset umaps-2.png"
</button>
</p>
<div id="fig-no-spaces-subset-umaps-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/subset umaps-2.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>seuratExplant &lt;- subset(seuratM, diseaseCond == &quot;explant&quot;)
DimPlot(seuratExplant, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, raster=FALSE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/subset%20umaps-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-umaps-3">
Past versions of "subset umaps-3.png"
</button>
</p>
<div id="fig-no-spaces-subset-umaps-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/subset umaps-3.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(seuratExplant, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, raster=FALSE) + theme(legend.position = &quot;null&quot;)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/subset%20umaps-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-umaps-4">
Past versions of "subset umaps-4.png"
</button>
</p>
<div id="fig-no-spaces-subset-umaps-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/subset umaps-4.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>seuratVisit1 &lt;- subset(seuratM, diseaseCond == &quot;visit1&quot;)
DimPlot(seuratVisit1, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, raster=FALSE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/subset%20umaps-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-umaps-5">
Past versions of "subset umaps-5.png"
</button>
</p>
<div id="fig-no-spaces-subset-umaps-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/subset umaps-5.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(seuratVisit1, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, raster=FALSE) + theme(legend.position = &quot;null&quot;)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/subset%20umaps-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-umaps-6">
Past versions of "subset umaps-6.png"
</button>
</p>
<div id="fig-no-spaces-subset-umaps-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/subset umaps-6.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>seuratVisit2 &lt;- subset(seuratM, diseaseCond == &quot;visit2&quot;)
DimPlot(seuratVisit2, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, raster=FALSE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/subset%20umaps-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-umaps-7">
Past versions of "subset umaps-7.png"
</button>
</p>
<div id="fig-no-spaces-subset-umaps-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/subset umaps-7.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(seuratVisit2, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, raster=FALSE) + theme(legend.position = &quot;null&quot;)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/subset%20umaps-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-umaps-8">
Past versions of "subset umaps-8.png"
</button>
</p>
<div id="fig-no-spaces-subset-umaps-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/subset umaps-8.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>seuratVisit3 &lt;- subset(seuratM, diseaseCond == &quot;visit3&quot;)
DimPlot(seuratVisit3, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, raster=FALSE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/subset%20umaps-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-umaps-9">
Past versions of "subset umaps-9.png"
</button>
</p>
<div id="fig-no-spaces-subset-umaps-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/subset umaps-9.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(seuratVisit3, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, raster=FALSE) + theme(legend.position = &quot;null&quot;)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/subset%20umaps-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-umaps-10">
Past versions of "subset umaps-10.png"
</button>
</p>
<div id="fig-no-spaces-subset-umaps-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/subset umaps-10.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>seuratVisits &lt;- subset(seuratM, diseaseCond %in% c(&quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;))
DimPlot(seuratVisits, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, raster=FALSE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/subset%20umaps-11.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-umaps-11">
Past versions of "subset umaps-11.png"
</button>
</p>
<div id="fig-no-spaces-subset-umaps-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/subset umaps-11.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DimPlot(seuratVisits, reduction = &quot;umap&quot;, pt.size = 0.1, cols = coldiseaseCond, raster=FALSE) + theme(legend.position = &quot;null&quot;)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/subset%20umaps-12.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-subset-umaps-12">
Past versions of "subset umaps-12.png"
</button>
</p>
<div id="fig-no-spaces-subset-umaps-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/subset umaps-12.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="plot-features" class="section level2">
<h2>plot features</h2>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000197614.MFAP5&quot;, pt.size = 1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-1">
Past versions of "feature plot-1.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-1.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000159674.SPON2&quot;, pt.size = 1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-2">
Past versions of "feature plot-2.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-2.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000065320.NTN1&quot;, pt.size = 1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-3">
Past versions of "feature plot-3.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-3.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000164136.IL15&quot;, pt.size = 1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-4">
Past versions of "feature plot-4.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-4.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSG00000172638.EFEMP2&quot;, pt.size = 1, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), raster = FALSE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-5">
Past versions of "feature plot-5.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-5.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Idents(seuratM) &lt;- seuratM$clusterName
VlnPlot(object=seuratM, features = &quot;ENSG00000197614.MFAP5&quot;, pt.size = 1, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-6">
Past versions of "feature plot-6.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-6.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratM, features = &quot;ENSG00000159674.SPON2&quot;, pt.size = 1, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-7">
Past versions of "feature plot-7.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-7.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratM, features = &quot;ENSG00000065320.NTN1&quot;, pt.size = 1, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-8">
Past versions of "feature plot-8.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-8.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratM, features = &quot;ENSG00000164136.IL15&quot;, pt.size = 1, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-9">
Past versions of "feature plot-9.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-9.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratM, features = &quot;ENSG00000172638.EFEMP2&quot;, pt.size = 1, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-10">
Past versions of "feature plot-10.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-10.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>seuratM$diseaseCond &lt;- factor(seuratM$diseaseCond, levels=c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;))
Idents(seuratM) &lt;- seuratM$diseaseCond
VlnPlot(object=seuratM, features = &quot;ENSG00000197614.MFAP5&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-11.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-11">
Past versions of "feature plot-11.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-11.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratM, features = &quot;ENSG00000159674.SPON2&quot;, pt.size = 1, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-12.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-12">
Past versions of "feature plot-12.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-12.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratM, features = &quot;ENSG00000065320.NTN1&quot;, pt.size = 1, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-13.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-13">
Past versions of "feature plot-13.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-13" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-13.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratM, features = &quot;ENSG00000164136.IL15&quot;, pt.size = 1, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-14.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-14">
Past versions of "feature plot-14.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-14" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-14.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratM, features = &quot;ENSG00000172638.EFEMP2&quot;, pt.size = 1, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot-15.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-15">
Past versions of "feature plot-15.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-15" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot-15.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>## subset PerivFb
seuratPerivFb &lt;- subset(seuratM, clusterName %in% c(&quot;PerivFb&quot;))
Idents(seuratPerivFb) &lt;- seuratPerivFb$clusterName
DimPlot(seuratPerivFb, reduction = &quot;umap&quot;, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20PerivFb-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-PerivFb-1">
Past versions of "feature plot PerivFb-1.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-PerivFb-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot PerivFb-1.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#make violin plot
seuratPerivFb$diseaseCond &lt;- factor(seuratPerivFb$diseaseCond, levels=c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;))
Idents(seuratPerivFb) &lt;- seuratPerivFb$diseaseCond
VlnPlot(object=seuratPerivFb, features = &quot;ENSG00000197614.MFAP5&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20PerivFb-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-PerivFb-2">
Past versions of "feature plot PerivFb-2.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-PerivFb-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot PerivFb-2.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratPerivFb, features = &quot;ENSG00000159674.SPON2&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20PerivFb-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-PerivFb-3">
Past versions of "feature plot PerivFb-3.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-PerivFb-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot PerivFb-3.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratPerivFb, features = &quot;ENSG00000065320.NTN1&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20PerivFb-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-PerivFb-4">
Past versions of "feature plot PerivFb-4.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-PerivFb-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot PerivFb-4.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratPerivFb, features = &quot;ENSG00000164136.IL15&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20PerivFb-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-PerivFb-5">
Past versions of "feature plot PerivFb-5.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-PerivFb-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot PerivFb-5.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratPerivFb, features = &quot;ENSG00000172638.EFEMP2&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20PerivFb-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-PerivFb-6">
Past versions of "feature plot PerivFb-6.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-PerivFb-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot PerivFb-6.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>## subset Fb
seuratFb &lt;- subset(seuratM, clusterName %in% c(&quot;Fb&quot;))
Idents(seuratFb) &lt;- seuratFb$clusterName
DimPlot(seuratFb, reduction = &quot;umap&quot;, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20Fb-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-Fb-1">
Past versions of "feature plot Fb-1.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-Fb-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot Fb-1.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#make violin plot
seuratFb$diseaseCond &lt;- factor(seuratFb$diseaseCond, levels=c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;))
Idents(seuratFb) &lt;- seuratFb$diseaseCond
VlnPlot(object=seuratFb, features = &quot;ENSG00000197614.MFAP5&quot;, pt.size = 0, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20Fb-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-Fb-2">
Past versions of "feature plot Fb-2.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-Fb-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot Fb-2.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratFb, features = &quot;ENSG00000159674.SPON2&quot;, pt.size = 0, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20Fb-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-Fb-3">
Past versions of "feature plot Fb-3.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-Fb-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot Fb-3.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratFb, features = &quot;ENSG00000065320.NTN1&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20Fb-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-Fb-4">
Past versions of "feature plot Fb-4.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-Fb-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot Fb-4.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratFb, features = &quot;ENSG00000164136.IL15&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20Fb-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-Fb-5">
Past versions of "feature plot Fb-5.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-Fb-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot Fb-5.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratFb, features = &quot;ENSG00000172638.EFEMP2&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20Fb-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-Fb-6">
Past versions of "feature plot Fb-6.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-Fb-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot Fb-6.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>## subset CM
seuratCM &lt;- subset(seuratM, clusterName %in% c(&quot;CM&quot;))
Idents(seuratCM) &lt;- seuratCM$clusterName
DimPlot(seuratCM, reduction = &quot;umap&quot;, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20CM-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-CM-1">
Past versions of "feature plot CM-1.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-CM-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot CM-1.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#make violin plot
seuratCM$diseaseCond &lt;- factor(seuratCM$diseaseCond, levels=c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;))
Idents(seuratCM) &lt;- seuratCM$diseaseCond
VlnPlot(object=seuratCM, features = &quot;ENSG00000197614.MFAP5&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20CM-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-CM-2">
Past versions of "feature plot CM-2.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-CM-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot CM-2.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratCM, features = &quot;ENSG00000159674.SPON2&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20CM-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-CM-3">
Past versions of "feature plot CM-3.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-CM-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot CM-3.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratCM, features = &quot;ENSG00000065320.NTN1&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20CM-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-CM-4">
Past versions of "feature plot CM-4.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-CM-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot CM-4.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratCM, features = &quot;ENSG00000164136.IL15&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20CM-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-CM-5">
Past versions of "feature plot CM-5.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-CM-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot CM-5.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratCM, features = &quot;ENSG00000172638.EFEMP2&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20CM-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-CM-6">
Past versions of "feature plot CM-6.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-CM-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot CM-6.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>## subset BEC
seuratBEC &lt;- subset(seuratM, clusterName %in% c(&quot;BEC1&quot;, &quot;BEC2&quot;))
Idents(seuratBEC) &lt;- seuratBEC$clusterName
DimPlot(seuratBEC, reduction = &quot;umap&quot;, cols = colclusterName)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20BEC-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-BEC-1">
Past versions of "feature plot BEC-1.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-BEC-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot BEC-1.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#make violin plot
seuratBEC$diseaseCond &lt;- factor(seuratBEC$diseaseCond, levels=c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;))
Idents(seuratBEC) &lt;- seuratBEC$diseaseCond
VlnPlot(object=seuratBEC, features = &quot;ENSG00000197614.MFAP5&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20BEC-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-BEC-2">
Past versions of "feature plot BEC-2.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-BEC-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot BEC-2.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratBEC, features = &quot;ENSG00000159674.SPON2&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20BEC-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-BEC-3">
Past versions of "feature plot BEC-3.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-BEC-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot BEC-3.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratBEC, features = &quot;ENSG00000065320.NTN1&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20BEC-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-BEC-4">
Past versions of "feature plot BEC-4.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-BEC-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot BEC-4.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratBEC, features = &quot;ENSG00000164136.IL15&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20BEC-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-BEC-5">
Past versions of "feature plot BEC-5.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-BEC-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot BEC-5.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratBEC, features = &quot;ENSG00000172638.EFEMP2&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20BEC-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-BEC-6">
Past versions of "feature plot BEC-6.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-BEC-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot BEC-6.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>## subset Mph
seuratMph &lt;- subset(seuratM, clusterName %in% c(&quot;Mph&quot;, &quot;Mph2&quot;))
Idents(seuratMph) &lt;- seuratMph$clusterName
DimPlot(seuratMph, reduction = &quot;umap&quot;, cols = colclusterName, label = TRUE)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20Mph-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-Mph-1">
Past versions of "feature plot Mph-1.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-Mph-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot Mph-1.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>#make violin plot
seuratMph$diseaseCond &lt;- factor(seuratMph$diseaseCond, levels=c(&quot;healthy&quot;, &quot;explant&quot;, &quot;visit1&quot;, &quot;visit2&quot;, &quot;visit3&quot;))
Idents(seuratMph) &lt;- seuratMph$diseaseCond
VlnPlot(object=seuratMph, features = &quot;ENSG00000197614.MFAP5&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20Mph-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-Mph-2">
Past versions of "feature plot Mph-2.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-Mph-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot Mph-2.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratMph, features = &quot;ENSG00000159674.SPON2&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20Mph-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-Mph-3">
Past versions of "feature plot Mph-3.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-Mph-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot Mph-3.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratMph, features = &quot;ENSG00000065320.NTN1&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20Mph-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-Mph-4">
Past versions of "feature plot Mph-4.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-Mph-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot Mph-4.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratMph, features = &quot;ENSG00000164136.IL15&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20Mph-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-Mph-5">
Past versions of "feature plot Mph-5.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-Mph-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot Mph-5.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>VlnPlot(object=seuratMph, features = &quot;ENSG00000172638.EFEMP2&quot;, pt.size = 1, cols = coldiseaseCond)</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/feature%20plot%20Mph-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plot-Mph-6">
Past versions of "feature plot Mph-6.png"
</button>
</p>
<div id="fig-no-spaces-feature-plot-Mph-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/HumanHeartCarTrans/blob/6ec9eda1708beae596b99c4a51d7ce64c634fa96/docs/figure/HumanHeartCarTrans.Rmd/feature plot Mph-6.png" target="_blank">6ec9eda</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-09-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="dotplot" class="section level3">
<h3>dotplot</h3>
<pre class="r"><code>genes &lt;- data.frame(gene=rownames(seuratM)) %&gt;% 
  mutate(geneID=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene))

selGenes &lt;- data.frame(geneID=c(&quot;NTN1&quot;,&quot;IL15&quot;,&quot;EFEMP2&quot;)) %&gt;% left_join(., genes, by=&quot;geneID&quot;)

DotPlot(seuratM, features = selGenes, group.by= &quot;clusterName_diseaseCond&quot;) + RotatedAxis() + scale_color_viridis(option=&quot;D&quot;) + coord_flip()</code></pre>
<p><img src="figure/HumanHeartCarTrans.Rmd/dotplot-1.png" width="1920" style="display: block; margin: auto;" />
## session info</p>
<pre class="r"><code>date()</code></pre>
<pre><code>[1] &quot;Tue Sep  3 19:22:39 2024&quot;</code></pre>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.0 (2024-04-24)
Platform: x86_64-apple-darwin20
Running under: macOS Ventura 13.6.7

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: Europe/Zurich
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] NCmisc_1.2.0                VennDiagram_1.7.3           futile.logger_1.4.3        
 [4] ggupset_0.4.0               gridExtra_2.3               DOSE_3.30.1                
 [7] enrichplot_1.24.0           msigdbr_7.5.1               org.Hs.eg.db_3.19.1        
[10] AnnotationDbi_1.66.0        clusterProfiler_4.12.0      multtest_2.60.0            
[13] metap_1.11                  scater_1.32.0               scuttle_1.14.0             
[16] destiny_3.18.0              circlize_0.4.16             muscat_1.18.0              
[19] viridis_0.6.5               viridisLite_0.4.2           lubridate_1.9.3            
[22] forcats_1.0.0               stringr_1.5.1               purrr_1.0.2                
[25] readr_2.1.5                 tidyr_1.3.1                 tibble_3.2.1               
[28] tidyverse_2.0.0             dplyr_1.1.4                 SingleCellExperiment_1.26.0
[31] SummarizedExperiment_1.34.0 Biobase_2.64.0              GenomicRanges_1.56.1       
[34] GenomeInfoDb_1.40.1         IRanges_2.38.1              S4Vectors_0.42.1           
[37] BiocGenerics_0.50.0         MatrixGenerics_1.16.0       matrixStats_1.3.0          
[40] pheatmap_1.0.12             ggpubr_0.6.0                ggplot2_3.5.1              
[43] Seurat_5.1.0                SeuratObject_5.0.2          sp_2.1-4                   
[46] runSeurat3_0.1.0            ExploreSCdataSeurat3_0.1.0 

loaded via a namespace (and not attached):
  [1] igraph_2.0.3              ica_1.0-3                 plotly_4.10.4            
  [4] zlibbioc_1.50.0           tidyselect_1.2.1          bit_4.0.5                
  [7] doParallel_1.0.17         clue_0.3-65               lattice_0.22-6           
 [10] rjson_0.2.21              blob_1.2.4                S4Arrays_1.4.1           
 [13] pbkrtest_0.5.3            parallel_4.4.0            png_0.1-8                
 [16] plotrix_3.8-4             cli_3.6.3                 ggplotify_0.1.2          
 [19] goftest_1.2-3             VIM_6.2.2                 variancePartition_1.34.0 
 [22] BiocNeighbors_1.22.0      uwot_0.2.2                shadowtext_0.1.3         
 [25] curl_5.2.1                mime_0.12                 evaluate_0.24.0          
 [28] tidytree_0.4.6            leiden_0.4.3.1            ComplexHeatmap_2.20.0    
 [31] stringi_1.8.4             backports_1.5.0           lmerTest_3.1-3           
 [34] qqconf_1.3.2              httpuv_1.6.15             magrittr_2.0.3           
 [37] splines_4.4.0             ggraph_2.2.1              sctransform_0.4.1        
 [40] ggbeeswarm_0.7.2          DBI_1.2.3                 jquerylib_0.1.4          
 [43] smoother_1.3              withr_3.0.0               git2r_0.33.0             
 [46] corpcor_1.6.10            class_7.3-22              rprojroot_2.0.4          
 [49] lmtest_0.9-40             tidygraph_1.3.1           formatR_1.14             
 [52] colourpicker_1.3.0        htmlwidgets_1.6.4         fs_1.6.4                 
 [55] ggrepel_0.9.5             labeling_0.4.3            fANCOVA_0.6-1            
 [58] SparseArray_1.4.8         DESeq2_1.44.0             ranger_0.16.0            
 [61] DEoptimR_1.1-3            reticulate_1.38.0         hexbin_1.28.3            
 [64] zoo_1.8-12                XVector_0.44.0            knitr_1.48               
 [67] ggplot.multistats_1.0.0   UCSC.utils_1.0.0          RhpcBLASctl_0.23-42      
 [70] timechange_0.3.0          foreach_1.5.2             fansi_1.0.6              
 [73] patchwork_1.2.0           caTools_1.18.2            data.table_1.15.4        
 [76] ggtree_3.12.0             RSpectra_0.16-1           irlba_2.3.5.1            
 [79] ggrastr_1.0.2             fastDummies_1.7.3         gridGraphics_0.5-1       
 [82] lazyeval_0.2.2            yaml_2.3.9                survival_3.7-0           
 [85] scattermore_1.2           crayon_1.5.3              RcppAnnoy_0.0.22         
 [88] RColorBrewer_1.1-3        progressr_0.14.0          tweenr_2.0.3             
 [91] later_1.3.2               ggridges_0.5.6            codetools_0.2-20         
 [94] GlobalOptions_0.1.2       aod_1.3.3                 KEGGREST_1.44.1          
 [97] Rtsne_0.17                shape_1.4.6.1             limma_3.60.3             
[100] pkgconfig_2.0.3           TMB_1.9.14                mathjaxr_1.6-0           
[103] EnvStats_2.8.1            aplot_0.2.3               scatterplot3d_0.3-44     
[106] spatstat.sparse_3.1-0     ape_5.8                   xtable_1.8-4             
[109] car_3.1-2                 highr_0.11                plyr_1.8.9               
[112] httr_1.4.7                rbibutils_2.2.16          tools_4.4.0              
[115] globals_0.16.3            beeswarm_0.4.0            broom_1.0.6              
[118] nlme_3.1-165              lambda.r_1.2.4            HDO.db_0.99.1            
[121] lme4_1.1-35.5             digest_0.6.36             numDeriv_2016.8-1.1      
[124] Matrix_1.7-0              farver_2.1.2              tzdb_0.4.0               
[127] remaCor_0.0.18            reshape2_1.4.4            yulab.utils_0.1.4        
[130] glue_1.7.0                cachem_1.1.0              polyclip_1.10-6          
[133] generics_0.1.3            Biostrings_2.72.1         mvtnorm_1.2-5            
[136] parallelly_1.37.1         mnormt_2.1.1              statmod_1.5.0            
[139] RcppHNSW_0.6.0            ScaledMatrix_1.12.0       carData_3.0-5            
[142] minqa_1.2.7               pbapply_1.7-2             spam_2.10-0              
[145] gson_0.1.0                utf8_1.2.4                graphlayouts_1.1.1       
[148] gtools_3.9.5              ggsignif_0.6.4            RcppEigen_0.3.4.0.0      
[151] shiny_1.8.1.1             GenomeInfoDbData_1.2.12   glmmTMB_1.1.9            
[154] memoise_2.0.1             rmarkdown_2.27            scales_1.3.0             
[157] future_1.33.2             RANN_2.6.1                Cairo_1.6-2              
[160] spatstat.data_3.1-2       rstudioapi_0.16.0         cluster_2.1.6            
[163] whisker_0.4.1             mutoss_0.1-13             spatstat.utils_3.0-5     
[166] hms_1.1.3                 fitdistrplus_1.1-11       munsell_0.5.1            
[169] cowplot_1.1.3             colorspace_2.1-0          rlang_1.1.4              
[172] DelayedMatrixStats_1.26.0 sparseMatrixStats_1.16.0  xts_0.14.0               
[175] dotCall64_1.1-1           shinydashboard_0.7.2      ggforce_0.4.2            
[178] laeken_0.5.3              mgcv_1.9-1                xfun_0.45                
[181] e1071_1.7-14              TH.data_1.1-2             iterators_1.0.14         
[184] abind_1.4-5               GOSemSim_2.30.0           treeio_1.28.0            
[187] futile.options_1.0.1      bitops_1.0-7              Rdpack_2.6               
[190] promises_1.3.0            scatterpie_0.2.3          RSQLite_2.3.7            
[193] qvalue_2.36.0             sandwich_3.1-0            fgsea_1.30.0             
[196] DelayedArray_0.30.1       proxy_0.4-27              GO.db_3.19.1             
[199] compiler_4.4.0            prettyunits_1.2.0         boot_1.3-30              
[202] beachmat_2.20.0           listenv_0.9.1             Rcpp_1.0.12              
[205] edgeR_4.2.0               workflowr_1.7.1           BiocSingular_1.20.0      
[208] tensor_1.5                MASS_7.3-61               progress_1.2.3           
[211] BiocParallel_1.38.0       babelgene_22.9            spatstat.random_3.2-3    
[214] R6_2.5.1                  fastmap_1.2.0             multcomp_1.4-25          
[217] fastmatch_1.1-4           rstatix_0.7.2             vipor_0.4.7              
[220] TTR_0.24.4                ROCR_1.0-11               TFisher_0.2.0            
[223] rsvd_1.0.5                vcd_1.4-12                nnet_7.3-19              
[226] gtable_0.3.5              KernSmooth_2.23-24        miniUI_0.1.1.1           
[229] deldir_2.0-4              htmltools_0.5.8.1         ggthemes_5.1.0           
[232] bit64_4.0.5               spatstat.explore_3.2-7    lifecycle_1.0.4          
[235] blme_1.0-5                nloptr_2.1.1              sass_0.4.9               
[238] vctrs_0.6.5               robustbase_0.99-3         spatstat.geom_3.2-9      
[241] sn_2.1.1                  ggfun_0.1.5               future.apply_1.11.2      
[244] bslib_0.7.0               pillar_1.9.0              gplots_3.1.3.1           
[247] pcaMethods_1.96.0         locfit_1.5-9.10           jsonlite_1.8.8           
[250] GetoptLong_1.0.5         </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.0 (2024-04-24)
Platform: x86_64-apple-darwin20
Running under: macOS Ventura 13.6.7

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: Europe/Zurich
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] NCmisc_1.2.0                VennDiagram_1.7.3           futile.logger_1.4.3        
 [4] ggupset_0.4.0               gridExtra_2.3               DOSE_3.30.1                
 [7] enrichplot_1.24.0           msigdbr_7.5.1               org.Hs.eg.db_3.19.1        
[10] AnnotationDbi_1.66.0        clusterProfiler_4.12.0      multtest_2.60.0            
[13] metap_1.11                  scater_1.32.0               scuttle_1.14.0             
[16] destiny_3.18.0              circlize_0.4.16             muscat_1.18.0              
[19] viridis_0.6.5               viridisLite_0.4.2           lubridate_1.9.3            
[22] forcats_1.0.0               stringr_1.5.1               purrr_1.0.2                
[25] readr_2.1.5                 tidyr_1.3.1                 tibble_3.2.1               
[28] tidyverse_2.0.0             dplyr_1.1.4                 SingleCellExperiment_1.26.0
[31] SummarizedExperiment_1.34.0 Biobase_2.64.0              GenomicRanges_1.56.1       
[34] GenomeInfoDb_1.40.1         IRanges_2.38.1              S4Vectors_0.42.1           
[37] BiocGenerics_0.50.0         MatrixGenerics_1.16.0       matrixStats_1.3.0          
[40] pheatmap_1.0.12             ggpubr_0.6.0                ggplot2_3.5.1              
[43] Seurat_5.1.0                SeuratObject_5.0.2          sp_2.1-4                   
[46] runSeurat3_0.1.0            ExploreSCdataSeurat3_0.1.0 

loaded via a namespace (and not attached):
  [1] igraph_2.0.3              ica_1.0-3                 plotly_4.10.4            
  [4] zlibbioc_1.50.0           tidyselect_1.2.1          bit_4.0.5                
  [7] doParallel_1.0.17         clue_0.3-65               lattice_0.22-6           
 [10] rjson_0.2.21              blob_1.2.4                S4Arrays_1.4.1           
 [13] pbkrtest_0.5.3            parallel_4.4.0            png_0.1-8                
 [16] plotrix_3.8-4             cli_3.6.3                 ggplotify_0.1.2          
 [19] goftest_1.2-3             VIM_6.2.2                 variancePartition_1.34.0 
 [22] BiocNeighbors_1.22.0      uwot_0.2.2                shadowtext_0.1.3         
 [25] curl_5.2.1                mime_0.12                 evaluate_0.24.0          
 [28] tidytree_0.4.6            leiden_0.4.3.1            ComplexHeatmap_2.20.0    
 [31] stringi_1.8.4             backports_1.5.0           lmerTest_3.1-3           
 [34] qqconf_1.3.2              httpuv_1.6.15             magrittr_2.0.3           
 [37] splines_4.4.0             ggraph_2.2.1              sctransform_0.4.1        
 [40] ggbeeswarm_0.7.2          DBI_1.2.3                 jquerylib_0.1.4          
 [43] smoother_1.3              withr_3.0.0               git2r_0.33.0             
 [46] corpcor_1.6.10            class_7.3-22              rprojroot_2.0.4          
 [49] lmtest_0.9-40             tidygraph_1.3.1           formatR_1.14             
 [52] colourpicker_1.3.0        htmlwidgets_1.6.4         fs_1.6.4                 
 [55] ggrepel_0.9.5             labeling_0.4.3            fANCOVA_0.6-1            
 [58] SparseArray_1.4.8         DESeq2_1.44.0             ranger_0.16.0            
 [61] DEoptimR_1.1-3            reticulate_1.38.0         hexbin_1.28.3            
 [64] zoo_1.8-12                XVector_0.44.0            knitr_1.48               
 [67] ggplot.multistats_1.0.0   UCSC.utils_1.0.0          RhpcBLASctl_0.23-42      
 [70] timechange_0.3.0          foreach_1.5.2             fansi_1.0.6              
 [73] patchwork_1.2.0           caTools_1.18.2            data.table_1.15.4        
 [76] ggtree_3.12.0             RSpectra_0.16-1           irlba_2.3.5.1            
 [79] ggrastr_1.0.2             fastDummies_1.7.3         gridGraphics_0.5-1       
 [82] lazyeval_0.2.2            yaml_2.3.9                survival_3.7-0           
 [85] scattermore_1.2           crayon_1.5.3              RcppAnnoy_0.0.22         
 [88] RColorBrewer_1.1-3        progressr_0.14.0          tweenr_2.0.3             
 [91] later_1.3.2               ggridges_0.5.6            codetools_0.2-20         
 [94] GlobalOptions_0.1.2       aod_1.3.3                 KEGGREST_1.44.1          
 [97] Rtsne_0.17                shape_1.4.6.1             limma_3.60.3             
[100] pkgconfig_2.0.3           TMB_1.9.14                mathjaxr_1.6-0           
[103] EnvStats_2.8.1            aplot_0.2.3               scatterplot3d_0.3-44     
[106] spatstat.sparse_3.1-0     ape_5.8                   xtable_1.8-4             
[109] car_3.1-2                 highr_0.11                plyr_1.8.9               
[112] httr_1.4.7                rbibutils_2.2.16          tools_4.4.0              
[115] globals_0.16.3            beeswarm_0.4.0            broom_1.0.6              
[118] nlme_3.1-165              lambda.r_1.2.4            HDO.db_0.99.1            
[121] lme4_1.1-35.5             digest_0.6.36             numDeriv_2016.8-1.1      
[124] Matrix_1.7-0              farver_2.1.2              tzdb_0.4.0               
[127] remaCor_0.0.18            reshape2_1.4.4            yulab.utils_0.1.4        
[130] glue_1.7.0                cachem_1.1.0              polyclip_1.10-6          
[133] generics_0.1.3            Biostrings_2.72.1         mvtnorm_1.2-5            
[136] parallelly_1.37.1         mnormt_2.1.1              statmod_1.5.0            
[139] RcppHNSW_0.6.0            ScaledMatrix_1.12.0       carData_3.0-5            
[142] minqa_1.2.7               pbapply_1.7-2             spam_2.10-0              
[145] gson_0.1.0                utf8_1.2.4                graphlayouts_1.1.1       
[148] gtools_3.9.5              ggsignif_0.6.4            RcppEigen_0.3.4.0.0      
[151] shiny_1.8.1.1             GenomeInfoDbData_1.2.12   glmmTMB_1.1.9            
[154] memoise_2.0.1             rmarkdown_2.27            scales_1.3.0             
[157] future_1.33.2             RANN_2.6.1                Cairo_1.6-2              
[160] spatstat.data_3.1-2       rstudioapi_0.16.0         cluster_2.1.6            
[163] whisker_0.4.1             mutoss_0.1-13             spatstat.utils_3.0-5     
[166] hms_1.1.3                 fitdistrplus_1.1-11       munsell_0.5.1            
[169] cowplot_1.1.3             colorspace_2.1-0          rlang_1.1.4              
[172] DelayedMatrixStats_1.26.0 sparseMatrixStats_1.16.0  xts_0.14.0               
[175] dotCall64_1.1-1           shinydashboard_0.7.2      ggforce_0.4.2            
[178] laeken_0.5.3              mgcv_1.9-1                xfun_0.45                
[181] e1071_1.7-14              TH.data_1.1-2             iterators_1.0.14         
[184] abind_1.4-5               GOSemSim_2.30.0           treeio_1.28.0            
[187] futile.options_1.0.1      bitops_1.0-7              Rdpack_2.6               
[190] promises_1.3.0            scatterpie_0.2.3          RSQLite_2.3.7            
[193] qvalue_2.36.0             sandwich_3.1-0            fgsea_1.30.0             
[196] DelayedArray_0.30.1       proxy_0.4-27              GO.db_3.19.1             
[199] compiler_4.4.0            prettyunits_1.2.0         boot_1.3-30              
[202] beachmat_2.20.0           listenv_0.9.1             Rcpp_1.0.12              
[205] edgeR_4.2.0               workflowr_1.7.1           BiocSingular_1.20.0      
[208] tensor_1.5                MASS_7.3-61               progress_1.2.3           
[211] BiocParallel_1.38.0       babelgene_22.9            spatstat.random_3.2-3    
[214] R6_2.5.1                  fastmap_1.2.0             multcomp_1.4-25          
[217] fastmatch_1.1-4           rstatix_0.7.2             vipor_0.4.7              
[220] TTR_0.24.4                ROCR_1.0-11               TFisher_0.2.0            
[223] rsvd_1.0.5                vcd_1.4-12                nnet_7.3-19              
[226] gtable_0.3.5              KernSmooth_2.23-24        miniUI_0.1.1.1           
[229] deldir_2.0-4              htmltools_0.5.8.1         ggthemes_5.1.0           
[232] bit64_4.0.5               spatstat.explore_3.2-7    lifecycle_1.0.4          
[235] blme_1.0-5                nloptr_2.1.1              sass_0.4.9               
[238] vctrs_0.6.5               robustbase_0.99-3         spatstat.geom_3.2-9      
[241] sn_2.1.1                  ggfun_0.1.5               future.apply_1.11.2      
[244] bslib_0.7.0               pillar_1.9.0              gplots_3.1.3.1           
[247] pcaMethods_1.96.0         locfit_1.5-9.10           jsonlite_1.8.8           
[250] GetoptLong_1.0.5         </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
